{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Filtering Papers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "\n",
    "uid_list = []\n",
    "with open('uid_list3.txt', 'r') as f:\n",
    "    line = f.read()\n",
    "    uid_list = line.split('\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "306.57s - pydevd: Sending message related to process being replaced timed-out after 5 seconds\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Defaulting to user installation because normal site-packages is not writeable\n",
      "Collecting tiktoken\n",
      "  Downloading tiktoken-0.8.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.2 MB)\n",
      "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m1.2/1.2 MB\u001b[0m \u001b[31m9.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m:00:01\u001b[0m0:01\u001b[0m\n",
      "\u001b[?25hRequirement already satisfied: requests>=2.26.0 in /home/abrehamtadesse/.local/lib/python3.10/site-packages (from tiktoken) (2.32.3)\n",
      "Requirement already satisfied: regex>=2022.1.18 in /home/abrehamtadesse/.local/lib/python3.10/site-packages (from tiktoken) (2024.5.15)\n",
      "Requirement already satisfied: certifi>=2017.4.17 in /usr/lib/python3/dist-packages (from requests>=2.26.0->tiktoken) (2020.6.20)\n",
      "Requirement already satisfied: idna<4,>=2.5 in /usr/lib/python3/dist-packages (from requests>=2.26.0->tiktoken) (3.3)\n",
      "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/lib/python3/dist-packages (from requests>=2.26.0->tiktoken) (1.26.5)\n",
      "Requirement already satisfied: charset-normalizer<4,>=2 in /home/abrehamtadesse/.local/lib/python3.10/site-packages (from requests>=2.26.0->tiktoken) (3.3.2)\n",
      "Installing collected packages: tiktoken\n",
      "Successfully installed tiktoken-0.8.0\n"
     ]
    }
   ],
   "source": [
    "!pip install tiktoken"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Using tiktoken from OpenAI"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "import tiktoken\n",
    "\n",
    "# Getting the tokenizer for gpt-40\n",
    "\n",
    "# tokenizer = tiktoken.get_encoding(\"cl100k_base\")\n",
    "\n",
    "# or using tiktoken.encoding_for_model\n",
    "\n",
    "tokenizer = tiktoken.encoding_for_model('gpt-4o')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No record found for article 6687896.\\Didn't find 1 articles\n",
      "No record found for article 6246379.\\Didn't find 2 articles\n",
      "No record found for article 4330560.\\Didn't find 3 articles\n",
      "No record found for article 9891329.\\Didn't find 4 articles\n",
      "No record found for article 6538741.\\Didn't find 5 articles\n",
      "No record found for article 6691994.\\Didn't find 6 articles\n",
      "No record found for article 4150914.\\Didn't find 7 articles\n",
      "No record found for article 2902235.\\Didn't find 8 articles\n",
      "No record found for article 10870342.\\Didn't find 9 articles\n",
      "No record found for article 3838734.\\Didn't find 10 articles\n",
      "No record found for article 2773894.\\Didn't find 11 articles\n",
      "258269   10698634  ---  1\n",
      "No record found for article 6072511.\\Didn't find 12 articles\n",
      "No record found for article 7492017.\\Didn't find 13 articles\n",
      "No record found for article 8015904.\\Didn't find 14 articles\n",
      "No record found for article 10063398.\\Didn't find 15 articles\n",
      "No record found for article 7713748.\\Didn't find 16 articles\n",
      "No record found for article 10240291.\\Didn't find 17 articles\n",
      "No record found for article 8327121.\\Didn't find 18 articles\n",
      "No record found for article 10267632.\\Didn't find 19 articles\n",
      "No record found for article 6758409.\\Didn't find 20 articles\n",
      "No record found for article 5008427.\\Didn't find 21 articles\n",
      "No record found for article 10544255.\\Didn't find 22 articles\n",
      "Json error for string:  [{\"source\": \"PMC\", \"date\": \"20240622\", \"key\": \"pmc.key\", \"infons\": {}, \"documents\": [{\"id\": \"10460506\", \"infons\": {\"license\": \"author_manuscript\"}, \"passages\": [{\"offset\": 0, \"infons\": {\"article-id_doi\": \"10.1016/j.bbi.2023.04.008\", \"article-id_manuscript\": \"NIHMS1900921\", \"article-id_pmc\": \"10460506\", \"article-id_pmid\": \"37100211\", \"fpage\": \"277\", \"license\": \"\\n          This file is available for text mining. It may also be used consistent with the principles of fair use under the copyright law.\\n        \", \"lpage\": \"291\", \"name_0\": \"surname:Hobson;given-names:Benjamin D.\", \"name_1\": \"surname:Stanley;given-names:Adrien T.\", \"name_2\": \"surname:De Los Santos;given-names:Mark B.\", \"name_3\": \"surname:Culbertson;given-names:Bruce\", \"name_4\": \"surname:Mosharov;given-names:Eugene V.\", \"name_5\": \"surname:Sims;given-names:Peter A.\", \"name_6\": \"surname:Sulzer;given-names:David\", \"section_type\": \"TITLE\", \"type\": \"front\", \"volume\": \"111\", \"year\": \"2023\"}, \"text\": \"Conserved and cell type-specific transcriptional responses to IFN-\\u03b3 in the ventral midbrain\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 96, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"Dysregulated inflammation within the central nervous system (CNS) contributes to neuropathology in infectious, autoimmune, and neurodegenerative disease. With the exception of microglia, major histocompatibility complex (MHC) proteins are virtually undetectable in the mature, healthy central nervous system (CNS). Neurons have generally been considered incapable of antigen presentation, and although interferon gamma (IFN-\\u03b3) can elicit neuronal MHC class I (MHC-I) expression and antigen presentation in vitro, it has been unclear whether similar responses occur in vivo. Here we directly injected IFN-\\u03b3 into the ventral midbrain of mature mice and analyzed gene expression profiles of specific CNS cell types. We found that IFN-\\u03b3 upregulated MHC-I and associated mRNAs in ventral midbrain microglia, astrocytes, oligodendrocytes, and GABAergic, glutamatergic, and dopaminergic neurons. The core set of IFN-\\u03b3-induced genes and their response kinetics were similar in neurons and glia, but with a lower amplitude of expression in neurons. A diverse repertoire of genes was upregulated in glia, particularly microglia, which were the only cells to undergo cellular proliferation and express MHC classII (MHC-II) and associated genes. To determine if neurons respond directly via cell-autonomous IFN-\\u03b3 receptor (IFNGR) signaling, we produced mutant mice with a deletion of the IFN-\\u03b3-binding domain of IFNGR1 in dopaminergic neurons, which resulted in a complete loss of dopaminergic neuronal responses to IFN-\\u03b3. Our results demonstrate that IFN-\\u03b3 induces neuronal IFNGR signaling and upregulation of MHC-I and related genes in vivo, although the expression level is low compared to oligodendrocytes, astrocytes, and microglia.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 1854, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"title_1\"}, \"text\": \"Introduction\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 1867, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Neuroinflammation is associated with neurodegenerative diseases including multiple sclerosis (MS), Alzheimer\\u2019s disease (AD), and Parkinson\\u2019s disease (PD), while clonally expanded T cells accumulate in the aged brain. MS is a demyelinating disorder classically characterized by autoimmune responses to oligodendrocyte proteins, and recent evidence suggests that T cell reactivity towards the neuronal protein \\u03b2-synuclein contributes to grey matter pathology in MS. There is considerable neuropathological evidence of inflammation in the substantia nigra pars compacta (SNc) of PD patients, including reactive, HLA-DR+ microglia and the presence of CD8 + and CD4 + T cells near the pigmented SNc dopaminergic (DA) neurons and noradrenergic neurons of the locus coeruleus, neurons that are killed in PD. We have demonstrated T cell autoreactivity towards \\u03b1-synuclein peptides in PD patients and recently showed that peripheral \\u03b1-synuclein-reactive T cells are most abundant in early stage PD patients. Further evidence suggests that T cell infiltration of the midbrain parenchyma is more common in early stage PD, but the precise role of T cells in PD pathophysiology remains unknown.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 3060, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Interferon gamma (IFN-\\u03b3) is a proinflammatory cytokine released primarily from CD4 + T helper type 1 (Th1), CD8 + T cells, and natural killer (NK) cells, cell types with low abundance in the brain parenchyma. IFN-\\u03b3 elicits cellular responses by increasing transcription of interferon-stimulated genes (ISGs), many of which encode transcription factors, antigen processing and presentation machinery, and innate immune effector proteins. Although IFN-\\u03b3 was formerly considered to be generally undetectable in the CNS, low levels of IFN-\\u03b3 derived from meningeal T cells can regulate neuronal activity and behavior in healthy mice. CNS cells may be exposed to high levels of IFN-\\u03b3 under infectious and inflammatory conditions that feature blood brain barrier disruption and T cell infiltration. In a mouse model of viral encephalitis, the release of IFN-\\u03b3 from CD8 + T cells has been suggested to drive the stripping of neuronal synapses via actions of phagocytic cells.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 4052, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Several reports indicate that IFN-\\u03b3 may play roles in PD pathogenesis. Elevated plasma levels of IFN-\\u03b3 were found in PD patients, and we recently identified altered transcriptional signatures in peripheral T cells of PD patients that may contribute to Th1/IFN-\\u03b3 bias. Elevated levels of IFN-\\u03b3 were reported in the substantia nigra, caudate, and putamen in PD, regions that contain dopaminergic cell bodies and their axonal projections. Constitutive expression of IFN-\\u03b3 in the lateral ventricles of mice leads to progressive nigrostriatal degeneration. Collectively, these data suggest that infiltrating T cells may contribute to dopaminergic neuron demise in PD (reviewed in), in part by IFN-\\u03b3-driven neuroinflammation.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 4796, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Key questions on the possible role of IFN-\\u03b3 in PD pathogenesis include: how do specific CNS cell types respond to IFN-\\u03b3, and which are capable of antigen presentation, by which pathways, and does cellular proliferation occur? Do these include mature neurons, which are often thought to not present antigen, and if so, does this consist of MHC-I, MHC-II or both? Do midbrain neurons directly respond to IFN-\\u03b3 in vivo in a cell autonomous manner as they appear to in neuronal culture, or are they bystanders to glial reactions, and do they require steps that involve peripheral inflammation? Which IFN- \\u03b3 dependent signaling pathways do neurons and other cells of the brain parenchyma exhibit and are any of these associated with neurodegenerative disorders? In this study, we address these questions by analyzing neuronal and glial gene expression responses to IFN-\\u03b3 injected directly into the ventral midbrain of adult mice, thereby intentionally bypassing peripheral inflammatory signals. We find that MHC-I antigen processing and presentation genes are rapidly induced in all CNS cell types, including all of the major groups of midbrain neurons. Single nucleus RNA-sequencing (snRNA-seq) revealed numerous transcriptional responses that were restricted to glial cells, as well as microglia-specific expression of MHC-II antigen presentation genes. We show that dopamine neuronal responses to IFN-\\u03b3 are abolished by the selective deletion of the IFN-\\u03b3-binding domain of IFNGR1, indicating that neurons are capable of MHC-I upregulation via direct cell-autonomous innate immune signaling.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 6414, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_1\"}, \"text\": \"Results\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 6422, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Kinetics of transcriptional responses to IFN-\\u03b3 by midbrain neurons and glia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 6502, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We chose to study the effects of intraparenchymal injection of IFN-\\u03b3 to exclude effects of systemic inflammation or activation of the peripheral immune system, as previously reported. We first sought to measure the kinetics of IFN-\\u03b3 responses in ventral midbrain neurons and glia. We delivered either 400 nL of 0.9% saline or a high dose of IFN-\\u03b3 (40 ng in 0.9% saline) to the ventral midbrain of WT C57BL/6J mice bilaterally via stereotactic injection and sacrificed mice 6, 24, 48, and 72 h after injection (Fig. 1a). The high dose employed is likely to reach a concentration that is near or greater than the dissociation constant of of IFN-\\u03b3 for its cognate receptor but is lower than used in previous models of PD. After rapid tissue dissection, we prepared nuclei, conducted fluorescence-activated nuclear sorting (FANs) to separate neuronal nuclei (NeuN +) from other cell types (NeuN\\u2212), and analyzed the sorted nuclei by bulk RNA sequencing (RNA-Seq, Supp. Fig. 1a). We found no evidence of dopamine neuronal degeneration or toxicity, as demonstrated by stable expression of dopamine neuronspecific mRNAs Th, Slc6a3 (DAT), and Ddc in NeuN + nuclei up to 72 h post-injection (Supp. Fig. 1e).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 7716, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Binding of IFN-\\u03b3 dimers to the IFN-\\u03b3 receptor (IFNGR) leads to cytoplasmic activation of Janus kinases 1\\u20132, a cascade of tyrosine phosphorylation events, recruitment and phosphorylation of STAT1\\u03b1, and nuclear translocation of phospho-STAT1\\u03b1 dimers (pSTAT1, reviewed in). pSTAT1 binding at \\u03b3-activation site (GAS) elements leads to increased transcription of ISGs involved in antiviral and antibacterial immunity, including other transcription factors. The first wave of rapidly-induced genes includes IFN regulatory factor 1 (IRF-1), a transcription factor and master regulator of IFN-\\u03b3-induced gene expression that drives subsequent waves of transcription via binding to IFN-stimulated response elements (ISRE). Comparing IFN-\\u03b3 to saline at each timepoint, we found the highest number of differentially expressed genes (DEGs; DESeq2, \\u2223log2FC\\u2223 > 1, padj < 0.01) at 6 and 24 h for both neurons and glia (Fig. 1c). At 6 and 24 h, there were more than twice as many upregulated genes in glia compared to neurons, with considerable overlap in the upregulated genes (Fig. 1c). The number of DEGs in neurons and glia at 48 and 72 h was comparatively modest, consistent with the decay of IFN-\\u03b3 and/or negative feedback.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 8961, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Consistent with prior kinetic analyses, the induction of the ER peptide transporter mRNAs Tap1 and Tapbp was rapid, with peak expression at 6 h in both neurons and glia (Fig. 1d). MHC-I heavy chain (H2-K1, H2-D1) and light chain (B2m) genes do not harbor GAS elements, but rather are induced by IRF-1 binding to ISRE sequences in later waves of transcription. Accordingly, levels of H2-K1 and \\u03b22m mRNAs were highest at 24 h and remained ~ 8-fold elevated at 72 h (Fig. 1d). The rise and decay of all four mRNAs was markedly similar across neuronal and glial nuclei, although the amplitude of induction was higher in glia (~2\\u20138-fold higher expression relative to neurons at all timepoints, Fig. 1d).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 9663, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Of the DEGs upregulated by IFN-\\u03b3 in both neuronal and glial nuclei, 28 genes were upregulated at > 3 timepoints (Supp. Fig. 1d), indicating sustained upregulation across multiple cell types and central involvement in the IFN-\\u03b3 host defense program. Indeed, gene ontology (GO) analysis of these 28 genes identified top over-represented ontologies including \\u2018Cellular response to IFN-\\u03b3\\u2019, \\u2018MHC class I protein complex\\u2019, and \\u2018Antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent\\u2019 (Supp. Fig. 1d). The induced genes included canonical IFN-inducible proteins such as MHC class I (H2-K1, H2-D1, B2m), immunoproteasome subunits (Psmb9/LMP2), ER peptide transporters (Tap1), guanylate binding proteins, and key regulators of IFN-\\u03b3-induced transcription (Stat1, Nlrc5) (Fig. 1e). Expression of these core IFN-\\u03b3-responsive genes was virtually absent in neuronal nuclei from saline-treated mice, and relative expression was uniformly higher in glia than in neurons (Fig. 1e). This phenomenon was true for most genes induced by IFN-\\u03b3 at all time points: while the magnitude of upregulation by IFN-\\u03b3 is well-correlated between neurons and glia (similar log2 fold change IFN-\\u03b3/Sal, x vs. y correlation in Fig. 1f), expression in glia was typically 2\\u20138-fold higher than in neurons (colorscale in Fig. 1f). These results demonstrate that the canonical IFN-\\u03b3 transcriptional response occurs in both neurons and glia with similar kinetics, but with lower amplitude in neurons.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 11200, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"IFN-\\u03b3 induces expression of T cell chemoattractant and CCR2 ligand chemokines\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 11282, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Given the role of chemokines in regulating leukocyte migration into the CNS, we analyzed the expression of mRNAs encoding chemokines. Several chemokines were expressed primarily in neurons and were unchanged after treatment with IFN-\\u03b3, including Cx3cl1/Fractalkine, Ccl25/TECK, and Ccl27a/CTACK (Supp. Fig. 2a, b). Neuronally derived CX3CL1/Fractalkine is known to modulate microglia via CX3CR1, while CCL25/TECK and CCL27/CTACK are primarily known for T cell homing in the gut and skin and to our knowledge have not been described in the CNS. In addition to well-known T cell chemoattractants (Cxcl9/MIG, Cxcl10/IP-10, Cxcl11/I-TAC), IFN-\\u03b3 also induced the expression of CCR2 ligands (Ccl2/MCP1, Ccl12/MCP-5) in both neurons and glia, with levels peaking at 6 h and declining rapidly thereafter (Supp. Fig. 2c). Consistent with prior studies, these results suggest that IFN-\\u03b3-exposed ventral midbrain neurons express chemokines that attract T cells and modulate phagocytic function of microglia/macrophages.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 12303, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Cell type-specific temporal responses to IFN-\\u03b3 in midbrain neurons and glia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 12383, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We next used a generalized linear model in DESeq2 to identify genes that are differentially regulated by IFN-\\u03b3 in neurons vs. glia (NeuN vs. Drug interaction, Supplementary Table 1, see Methods). Forty-two genes were upregulated in both neurons and glia, but the effect of IFN-\\u03b3 was greater in glia; these included canonical IFN-inducible genes such as Tap2 and Gbp2 (Fig. 2a-c, purple module). Most of the remaining DEGs from the interaction analysis were largely unchanged in neurons despite upregulation or downregulation by IFN-\\u03b3 in glia (Fig. 2a). These genes could be further separated into three groups based on the effect of IFN-\\u03b3 in glia and differences in baseline (saline) expression in neurons and glia.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 13115, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Twenty-seven genes were highly expressed in neurons at baseline, but not glia, and were upregulated by IFN-\\u03b3 only in glia (Fig. 2a-c, green module). These included the vesicular secretion regulator Cadps2 (CAPS2) and Socs2 (SOCS2), a negative feedback regulator of JAK-STAT signaling (Fig. 2c). Another 10 genes were also highly expressed in neurons at baseline, but not glia, but were rapidly downregulated by IFN-\\u03b3 only in glia; these included the enzyme dopa decarboxylase (Ddc) that is required for dopamine synthesis, and the endocytic modulator auxilin (Dnajc6) which is associated with rare cases of familial PD (Fig. 2a & 2c-d, orange module). These results suggest that glia, but not neurons, transiently suppress transcription of specific mRNAs involved in vesicular secretion and recycling after exposure to IFN-\\u03b3.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 13953, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Finally, 34 genes were highly expressed in glia at baseline, but not neurons, and were rapidly downregulated by IFN-\\u03b3 (Fig. 2a & 2c-d, yellow module). Notably, these included many microglia-enriched genes such as Csf1r, Siglech, and Dock8 (Fig. 2c). We further found that 23 of the top 50 microglial marker genes (identified via snRNA-seq data, see Fig. 3 below) were significantly downregulated at 6 h after IFN-\\u03b3 (Supp. Fig. 3). Thus, the microglial activation in response to IFN-\\u03b3 likely involves a transient repression of some, but not all, lineage marker genes.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 14532, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Conserved and cell-type-specific single cell responses to IFN-\\u03b3\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 14600, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"To resolve IFN-\\u03b3 responses at the single cell level, we next conducted snRNA-seq on ventral midbrain nuclei. To ensure proper coverage of neuronal subtypes, we used FANS as above to enrich NeuN + nuclei to ~ 70% of each mouse sample (n = 1 saline, n = 2 IFN-\\u03b3) prior to sequencing. Since the core IFN-\\u03b3-induced signature was stably upregulated for at least 48 h (Fig. 1), we selected 48 h for subsequent analysis in order to better approximate chronic in vivo exposure and avoid transient disruption of cell lineage marker expression (Fig. 2c-d). We processed the sequencing data and performed unsupervised clustering on the molecular count matrices as described previously. We identified thirty-eight clusters (Supp. Fig. 4a), which were collapsed to six major cell types with sufficient coverage for downstream analysis based on key marker genes significantly enriched in each cluster (Supp. Fig. 4b). After removal of nuclear doublets (Supp. Fig. 4c-d, see Methods), we retained a total of 4,699 nuclei, including 1,241 oligodendrocytes, 194 astrocytes, 134 microglia, 437 dopamine neurons, 1,257 GABAergic neurons, and 1,436 VGLUT2 neurons for downstream analysis (Fig. 3a), with comparable distribution across saline- and IFN-\\u03b3-treated mice (Supp. Fig. 5a).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 15879, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We previously found that IFN-\\u03b3-treated dopamine neurons express MHC-I in vitro, and baseline MHC-I in ventral midbrain dopamine neurons has been demonstrated in vivo, but dopamine neurons comprise a minority of all ventral midbrain neurons represented in our bulk NeuN + RNA-seq data (Fig. 1). We first analyzed the expression of a gene signature of MHC-I related genes (H2-K1, H2-D1, B2m, Tap1). In the absence of IFN-\\u03b3, only microglia expressed detectable levels of these genes, whereas all cell types could be induced to express the MHC-I gene signature (Fig. 3b). We previously reported that non-catecholaminergic neurons express MHC-I with higher concentrations of IFN-\\u03b3 in vitro, and consistently, we found that GABAergic, glutamatergic, and dopaminergic midbrain neurons all expressed similar levels of MHC-I genes after exposure to high dose IFN-\\u03b3 in vivo (Fig. 3b). Induced levels of MHC-I genes were similar between astrocytes and neurons, all of which were lower than in oligodendrocytes and microglia (Fig. 3b). Thus, all major cell types in the mouse ventral midbrain parenchyma can upregulate MHC-I mRNA upon exposure to IFN-\\u03b3.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 17041, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"In contrast to MHC-I, the expression of MHC-II is typically limited to professional antigen presenting cells. We found that the MHC-II genes (H2-Ab1, H2-Eb1, H2-Aa) and the invariant chain (Cd74) were highly expressed in IFN-\\u03b3-treated microglia, and no other cell types showed detectable MHC-II gene expression (Fig. 3b, Supp. Fig. 5c). Similarly, we found that IFN-\\u03b3 led to marked changes in the morphology of Iba1+ microglia, with associated cellular proliferation as detected by Ki67 protein expression and nuclear localization (Supp. Fig. 6). Ki67 staining was not present within any other cell types.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 17655, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We next conducted differential expression analysis, comparing each IFN-\\u03b3 vs. saline sample within each major cell type. Only genes differentially regulated by IFN-\\u03b3 in both samples were retained (see Methods; \\u2223log2FC\\u2223 > 3, FDR < 0.01), which comprised 108 DEGs across all six major cell types (Fig. 3c). Hierarchical clustering revealed that the most distinct expression profile for these genes belonged to IFN-\\u03b3-activated microglia, followed by oligodendrocytes and astrocytes, while the profiles of saline- and IFN-\\u03b3 -exposed neurons were more similar (Fig. 3c). We found that two genes, Atp9a and Ltbp3, were massively downregulated in all cell types (Supp. Fig. 5b). ATP9A is a type IV P-type ATPase involved in endosome recycling and exosome release, while LTBP-3 is a regulator of TGF-\\u03b2, although to our knowledge neither has been reported to possess an explicit immune function. Meanwhile, the majority of genes were upregulated by IFN-\\u03b3 and were highly expressed in glia, but not neurons (Fig. 3c). One notable exception to this trend was the expression of the non-classical MHC-I (class Ib) genes H2-Q5, H2-Q6, and H2-Q7, which were robustly induced by IFN-\\u03b3 in microglia and all neurons, but not oligodendrocytes and astrocytes (Fig. 3c). Class Ib molecules are structurally similar to class Ia (H2-D1, H2-K1), although class Ib genes tend to exhibit tissue-specific expression and distinct regulatory elements.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 19104, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"While all neurons upregulated classical MHC-I genes, we found that a number of canonical ISGs (e.g., Tap1, Tap2, Gbp2, Cd274/PD-L1) were poorly expressed in IFN-\\u03b3-treated neuronal nuclei at 48 h (Fig. 3c). Notable among these was Nlrc5, which encodes NLRC5, a member of the NOD-like receptor family that critically regulates MHC-I expression analogous to CIITA regulation of MHC-II expression. These results suggest that deficiency in effectors of IFN-\\u03b3-mediated signaling or transcription might lead to the dampened neuronal responses observed in our studies. Indeed, we found that several genes associated with the GO term \\u2018positive regulation of IFN-\\u03b3 mediated signaling pathway\\u2019 were robustly induced in glia, but not in neurons (Fig. 3d). These included poly(ADP-ribose) polymerase 9 (PARP9), which is known to activate STAT1 phosphorylation in activated macrophages, and immunity-related GTPases Irgm1-2 which regulate IFN-\\u03b3-driven effector responses to intracellular pathogens.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 20105, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Within the ventral midbrain of PD patients, dopamine neurons of the ventral tegmental area (VTA) tend to be spared relative to more vulnerable populations in the SNc. Notably, we previously found HLA class I expression in dopamine neurons within the SNc, but not VTA, in postmortem tissue from PD patients. However, recent studies have demonstrated considerable transcriptomic heterogeneity amongst midbrain dopamine neurons. We thus leveraged our snRNA-seq data to study transcriptional responses to IFN-\\u03b3 amongst various subpopulations of dopamine neurons. We further analyzed dopamine neuronal expression profiles using unsupervised clustering as above (see Methods) and identified eight subclusters (Fig. 3e and Supp. Fig. 7). All subclusters expressed moderate to high levels of Slc6a3 (DAT) but were distinguished by expression of marker genes consistent with previous studies (Ndnf, Ntng1, Nefl, C1ql3, Cbln1, Cbln4, Tacr3), including those with ventral SN bias (Sox6, Aldh1a1, Grin2c, and Atp2a3) and those with medial VTA bias (Sorcs3, Nrp2, Grp, Calb1) (Fig. 3e, Supp. Fig. 7, and Supplementary Table 2). Dopamine neurons from saline- and IFN-\\u03b3-treated mice were distributed similarly throughout all subclusters (Fig. 3e), suggesting that IFN-\\u03b3 treatment did not significantly alter the expression of subcluster-defining marker genes. We found no evidence of differential responses to IFN-\\u03b3, with all subclusters upregulating classical (H2-K1, H2-D1, B2m) and non-classical MHC-I genes (H2-Q6, H2-Q7) (Fig. 3e). Thus, at least under conditions of direct exposure to high concentrations of IFN-\\u03b3, an MHC-I antigen presentation program can be induced in all populations of ventral midbrain dopamine neurons, suggesting that the presence of MHC-I alone is not sufficient to determine the subclusters targeted in parkinsonian neurodegeneration and that multiple hits may be required.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 22015, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Dopamine neurons respond to IFN-\\u03b3 via cell autonomous IFNGR signaling\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 22089, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"In addition to IFN-\\u03b3, MHC-I heavy chain transcription can be induced in neurons by TNF-\\u03b1, and it is possible that the effects of IFN-\\u03b3 on neurons may be due to release of other cytokines or interactions with glia, particularly activated microglia. Furthermore, canonical ISGs can be upregulated in dopamine neurons without IFN-\\u03b3 under conditions of oxidative stress, such as high dose L-DOPA, 6-OHDA exposure, or a model of \\u03b1-synuclein aggregation. Thus, we first confirmed that dopamine neurons in the ventral tegmental area (VTA) and SNc express IFN-\\u03b3 receptor subunit 1 (Ifngr1) mRNA using fluorescence in situ hybridization (FISH) mRNA (Supp. Fig. 8). To test whether cell autonomous IFN-\\u03b3 signaling is required for their transcriptional responses, we selectively modified the Ifngr1 locus in dopamine neurons by crossing Ifngr1fl/fl mice to dopamine transporter (DAT)-cre mice. As previously described, Cre-mediated recombination in Ifngr1fl/fl mice leads to deletion of the IFN-\\u03b3-binding extracellular domain of IFNGR1 and conditional loss of IFN-\\u03b3 signaling.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 23191, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We first confirmed that DAT-Cre:Ifngr1fl/fl mice showed no gross alteration in midbrain or striatal TH and DAT immunolabel (Supp. Fig. 9). We then used immunohistochemistry to analyze phosphorylation of STAT1 Tyr701 (pSTAT1-Y701), the STAT1 residue critical for IFN-\\u03b3-mediated nuclear translocation and transcriptional activation, in the ventral midbrain at 6 h after injection of IFN-\\u03b3 or saline. IFN-\\u03b3-induced nuclear pSTAT1-Y701 was observed in TH+ dopamine neurons and TH\\u2212 cells in WT:Ifngr1fl/fl mice, but only in TH\\u2212 cells in DAT-Cre:Ifngr1fl/fl mice (Fig. 4a). Triple labeling with NeuN enabled analysis of nuclei from dopamine neurons (TH+/NeuN+), non-dopamine neurons (TH\\u2212/NeuN+), and glia (TH\\u2212/NeuN\\u2212) (Fig. 4b). Quantification of pSTAT1-Y701 induction in these populations confirmed a selective deficiency of IFN-\\u03b3 response in dopamine neurons (Fig. 4c). Notably, the average pSTAT1-Y701 intensity in glia was ~ 3-fold higher than in neurons, with no appreciable differences between dopamine neurons and non-dopamine neurons (Fig. 4b-c). Consistent with the pSTAT1-Y701 results, RNA-FISH confirmed that dopamine neurons in IFN-\\u03b3-treated DAT-Cre:Ifngr1fl/fl mice express significantly less Tap1 and MHC-I heavy chain mRNAs compared to WT:Ifngr1fl/fl mice (Fig. 4e-f). Collectively, these results confirm that cell autonomous IFNGR signaling and nuclear pSTAT1 translocation, not bystander responses to reactive glia, are responsible for transcription of antigen presentation genes within dopamine neurons following ventral midbrain exposure to IFN-\\u03b3.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 24775, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_1\"}, \"text\": \"Discussion\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 24786, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Interferons were discovered as factors with anti-viral activity and research over the following decades identified a large set of interferon-regulated pathways, including about 10% of the human genome. Due to reports of CD8 + and Th1 CD4 + T cells infiltrating the brain in aging and neurological disease, we characterized transcriptional changes caused by acute local exposure to the proinflammatory cytokine IFN-\\u03b3 in the ventral midbrain. Inflammation associated with neurodegenerative disorders may include cellular responses in the meninges and elsewhere in the periphery (Hobson et al., 2022c), which are likely to produce interacting effects. To avoid the confounding influence of other inflammatory cytokines, we designed our current approach to provide a molecular analysis of acute cellular changes in the absence of peripheral inflammation. To avoid the confounding influence of other inflammatory cytokines, we designed our current approach to provide a molecular analysis of acute cellular changes in the absence of peripheral inflammation.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 25843, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_2\"}, \"text\": \"Conserved responses to IFN-\\u03b3 in brain cells\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 25891, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Here, we focused on the brain parenchymal response to IFN-\\u03b3 and note that rare cell types in or at the border the CNS, including dendritic cells, monocytes, natural killer cells, and border associated macrophages were either too uncommon for detection or lost during brain dissection due to meningeal removal: these cell types may nevertheless play important roles in adaptive immune responses in the brain. Although MHC-I is expressed by most nucleated cells in the periphery, our results confirm previous reports that MHC-I mRNA is expressed at negligible levels in the healthy rodent CNS, with the exception of microglia (Fig. 3b). However, we find that for all major classes of CNS cells, IFN-\\u03b3 rapidly induces expression of MHC-I and other ISGs involved in antigen presentation, with similar kinetics to non-brain cells. Transcriptional kinetics were similar across CNS cell types, but ISG expression amplitude was highest in microglia, lower in astrocytes and oligodendrocytes, and lowest in neurons (Fig. 3). However, the abundance of neurons compared to microglia suggests that neuronal expression may still be important for the overall brain response to IFN-\\u03b3. For example, the IFN-\\u03b3-inducible T cell chemoattractants Cxcl9, Cxcl10, and Cxcl11 were rapidly upregulated in both neurons and glia (Supp. Fig. 2). Although our data suggest that microglia induce these genes at > 8-fold higher levels than neurons, neurons are typically > 5-fold more abundant than microglia in the human brain. Thus, it is likely that neurons, astrocytes, oligodendrocytes, and microglia each contribute to conserved IFN-\\u03b3 responses in the brain.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 27546, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_2\"}, \"text\": \"Dampened neuronal responses to IFN-\\u03b3\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 27587, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Why are neurons less responsive to IFN-\\u03b3 than glia? It has been suggested that the absence of MHC-I expression and lack of antigen presentation capacity may be protective for post-mitotic neurons. Our results underscore the longstanding view that MHC-I is not readily detectable in neurons, and highlight a growing appreciation that detection of physiologically relevant neuronal MHC-I requires highly sensitive techniques.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 28015, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Multiple mechanisms could underlie the dampened transcriptional response to IFN-\\u03b3 in neurons. First, we observed that the levels of nuclear pSTAT1 at 6 h post-IFN-\\u03b3 are significantly lower in neurons compared to glia (Fig. 4). Decreased abundance of IFNGR, STAT1, and/or negative feedback signaling that regulates STAT1-Y701 phosphorylation could all contribute to decreased nuclear pSTAT1 accumulation. Given the dramatic response in microglia compared to other glia, differential regulation of pSTAT1 is unlikely to be the sole mediator of cell type-specific IFN-\\u03b3 responses in the CNS. Other minor differences in ISG induction between cell types suggest subtle differences in transcriptional dynamics downstream of initial pSTAT1 binding: for example, the upregulation of non-classical MHC-I genes H2-Q5, H2-Q6, and H2-Q7 was specific to microglia and neurons (Fig. 3c). Thus, the overall lower amplitude of IFN-\\u03b3-induced transcription in neurons could be due in part to altered basal chromatin accessibility and/or inefficient expression of key transcription factors that drive downstream expression programs.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 29145, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_2\"}, \"text\": \"Microglia-specific responses to IFN-\\u03b3\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 29187, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Unexpectedly, we found an early, transient downregulation of some but not all microglial lineage-identity genes in NeuN\\u2212 nuclei at 6 h post-exposure to IFN-\\u03b3 (Supp. Fig. 3). These include Dock8, a protein implicated in controlling microglial function in neurodegeneration, the cytokine receptors Csf1r and Csf3r, and Siglech, a sialic acid-binding receptor that inhibits microglial activation. These responses may lead to reduced microglial sensitivity to anti-inflammatory and homeostatic mediators, as has been observed in IFN-\\u03b3-polarized macrophages. However, further studies are required to characterize the role of this transient transcriptional suppression within the microglial activation program induced by IFN-\\u03b3.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 29921, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Within the CNS, it is well-established that activated microglia express MHC-II, but prior studies of MHC-II expression in mouse oligodendrocytes and astrocytes have yielded conflicting results. In the context of MS, single cell RNA-seq in mouse models and human brain tissue identified subsets of mature oligodendrocytes that express IFN-inducible genes, including MHC-I and MHC-II genes, although only MHC-I associated signals were observed in human neuropathology. Our results suggest that acute stimulation with IFN-\\u03b3 elicits the transcriptional upregulation of MHC-II genes only in microglia (Fig. 3b), although more chronic exposure or the presence of other stimulatory factors may lead to expression in astrocytes and/or oligodendrocytes under certain conditions.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 30695, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_2\"}, \"text\": \"Dopamine neurons direcdy respond to IFN-\\u03b3\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 30741, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Several studies indicate that neuronal MHC-I presentation could be a response to intrinsic neuronal stress, even in culture paradigms lacking cell types that release IFN-\\u03b3, TNF-\\u03b1, or other candidate extrinsic triggers. In our hands, cultured postnatal DA neurons can be induced to present an intracellularly processed foreign antigen by MHC-I following high cytosolic oxidative stress caused by high levels of L-DOPA in the absence of microglia, T cells, or extrinsic cytokines. The hypothesis that neuronal stress may be an independent means to drive antigen presentation is consistent with reports of increased MHC-I in mature neurons by kainic acid, which indices excitotoxicity, and increased TAP-1 and immunoproteasome messages following the toxin, 6-hydroxy-dopamine and by aggregated alpha-synuclein. However, we found that dopamine neurons in DAT-Cre:Ifngr1fl/fl mice did not significantly upregulate pSTAT1 or MHC-I in response to IFN-\\u03b3 injection in the ventral midbrain (Fig. 4). These results suggest that the acute stress placed on dopaminergic neurons by 48 h of surrounding glial activation is insufficient to upregulate pSTAT1 via alternate pathways, although these may act at longer time scales.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 31965, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Although IFN-\\u03b3 has been implicated in dopaminergic neuro-degeneration, studies in rodent and culture models of PD have focused on glial activation as the key driver of neuronal cell death. IFNGR1 and STAT1 knockout mice are protected from pathological overexpression of IFN-\\u03b3 in the mouse brain, but it is unknown which cells require IFN-\\u03b3 signaling to drive nigrostriatal degeneration in this model. Consistent with previous studies of cortical and hippocampal neurons, we found that midbrain dopaminergic neurons express Ifngr1 and upregulate MHC-I genes via increased nuclear pSTAT1-Y701 after IFN-\\u03b3 exposure (Fig. 4). Although we directly exposed dopaminergic neurons to IFN-\\u03b3 in the ventral midbrain, previous work found that IFN-\\u03b3 stimulation of axons can induce neuron MHC-I via retrograde signaling. Given that their massive axonal arbors comprise the majority of their surface area and protein mass, dopamine neurons may also be induced to express MHC-I via inflammation within the caudate and putamen in PD.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 33007, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"PD is widely considered to require multiple hits, including aging, and antigen presentation may play a role in some forms of neuron damage and death. Possible deleterious roles for IFN-\\u03b3 signaling and MHC-I expression within dopamine neurons in PD are suggested from mouse models of autoimmune-mediated narcolepsy, paraneoplastic encephalitis, and neurotropic viral infection. and demonstrated that neuronal expression of ectopic \\u201cneo-self-antigen\\u201d led to antigen-specific CD8 + T cell infiltration of the brain parenchyma and neuronal destruction. While found that viral antigen-specific CD8 + T cell-mediated neuronal destruction requires functional IFN-\\u03b3 receptor signaling in the CNS, constitutive neuronal overexpression of the MHC-I molecule H-2Db did not elicit pathology in the absence of CNS IFN-\\u03b3 receptor signaling. These results raised the possibility that neuronal upregulation of MHC-I is not critical for IFN-\\u03b3-mediated neuropathology in this model, but our results suggest that loss of neuronal IFNGR1 would completely prevent upregulation of the machinery necessary to process and present antigen on the constitutively expressed H-2Db molecules (e.g., B2m, Tap1, Psmb8). Nonetheless, subsequent studies found that IFN-\\u03b3-mediated synapse loss in virally infected neurons was driven by neuronal CCL2 expression and phagocyte-mediated synaptic removal. Thus, in addition to potential autoimmune-mediated responses directed at dopamine neurons, future work should investigate the synaptic and electrophysiological consequences of IFN-\\u03b3 signaling in the ventral midbrain.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34615, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_1\"}, \"text\": \"Materials and Methods\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34637, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Animals\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34645, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Mice were housed on a 12-hour light/dark cycle with food and water available ad libitum. Adult male and female mice (6\\u201312 months old) were used in all experiments. DATIRES-Cre mice (JAX #006660, RRID: IMSR_JAX:006660) and Ifngr1fl mice (JAX #025394, RRID: IMSR_JAX:025394) were obtained from Jackson Laboratories (Bar Harbor, ME). DATIRES-Cre mice derive from C57BL/6J background, while Ifngr1fl mice derive from C57BL/6N background. All experiments were conducted according to NIH guidelines and approved by the Institutional Animal Care and Use Committees of Columbia University and the New York State Psychiatric Institute.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 35274, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"DATIRES-Cre:Ifngr1fl experimental litters were bred from a cross of DATIRES-Cre/wt;Ifngr1fl/fl \\u00d7 DATwt/wt;Ifngr1fl/fl mice such that Cre+ mice (DATIRES-Cre/wt;Ifngr1fl/fl) could be compared to Cre\\u2212 littermates with no modification of the Ifngr1 locus (DATwt/wt;Ifngr1fl/fl). To ensure proper allocation of genotypes to experimental treatments (stereotaxic injection of virus and/or IFN-\\u03b3), experimenters were not blinded to the genotype of mice during stereotaxic injection. However, a separate experimenter that was blind to the genotype and treatment condition conducted animal sacrifice and tissue dissection. All experimental procedures were conducted according to NIH guidelines and were approved by the Institutional Animal Care and Use Committees of Columbia University and the New York State Psychiatric Institute.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 36101, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Stereotaxic injections\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 36124, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"All surgical procedures were approved by the Institutional Animal Care and Use Committee and the Department of Comparative Medicine at New York State Psychiatric Institute. Mice were anesthetized with 4% isoflurane. Animals were transferred onto a Kopf Stereotaxic apparatus and maintained under isoflurane anesthesia (1\\u20132%). After hair removal and sterilization of the scalp using chlorhexidine and ethanol, a midline incision was made. Bregma and Lambda coordinates were determined, and minor adjustments in head position were made to match the DV coordinates. Saline or IFN-\\u03b3 was injected at AP \\u22123.2, ML \\u2013 0.9, and DV \\u2013 4.4. A small hole was drilled into the skull and 400 nL of 0.9% saline or 0.9% saline + 100 ng/\\u03bcL recombinant mouse IFN-\\u03b3 (40 ng IFN-\\u03b3 total) was injected through a pulled glass pipet using a Nanoject 2000 (Drummond Scientific; 8 pulses of 50 nL). At 5 min after injection, the glass pipet was slowly withdrawn over 5 min. After closing the skin with vicryl sutures, mice received 0.5 mL of 0.9% saline i.p. and were allowed to recover for > 1 h before being returned to their home cages.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 37249, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Antibodies and Reagents\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 37273, \"infons\": {\"file\": \"T1.xml\", \"id\": \"T1\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"void\\\" rules=\\\"groups\\\">\\n            <colgroup span=\\\"1\\\">\\n              <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n              <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n              <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n              <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n              <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n            </colgroup>\\n            <thead>\\n              <tr>\\n                <th align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Name</th>\\n                <th align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Manufacturer</th>\\n                <th align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Catalog</th>\\n                <th align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID</th>\\n                <th align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Usage</th>\\n              </tr>\\n            </thead>\\n            <tbody>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Chicken anti-TH</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Millipore</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">AB9702</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_570923</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:500</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Mouse anti-NeuN</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Millipore</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">MAB377</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_2298772</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:500</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Rabbit anti-Iba1</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Wako</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">019\\u201319741</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_839504</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:1000</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Rat anti-Ki67</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ThermoFisher</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">14\\u20135698-82</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_10854564</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:1000</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Mouse anti-NeuN, Alexa488</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Millipore</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">MAB377X</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_2149209</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">FANS, 1:500</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Rabbit anti-pStat-Y701</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Cell Signaling</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">9167</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_561284</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:1000</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Rat anti-DAT</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Millipore</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">MAB369</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_2190413</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:500</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Goat anti-Chicken IgY (H + L), Alexa Fluor Plus 488</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ThermoFisher</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">A-32931TR</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_2866499</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:1000</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Goat anti-Mouse IgG (H + L), Alexa Fluor Plus 568</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ThermoFisher</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">A-11004</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_2534072</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:1000</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Goat anti-Rabbit IgG (H + L), Alexa Fluor 647</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ThermoFisher</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">A32733</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_2633282</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:1000</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Goat anti-Rat IgG (H + L), Alexa Fluor 647</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ThermoFisher</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">A-32931TR</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_2866499</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:1000</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Mm-Ifngr1-C2</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ACD Biotechne</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">441491-C2</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NA</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">FISH</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Mm-Tap1-C2</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ACD Biotechne</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">575041-C2</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NA</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">FISH</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Pan-MHC Class 1<xref rid=\\\"TFN2\\\" ref-type=\\\"table-fn\\\">**</xref></td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ACD Biotechne</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NPR-0023100</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NA</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">FISH</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">TSA Cy3</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Perkin Elmer</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NEL744001KT</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NA</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">FISH, 1:1500</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">TSA Cy5</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Perkin Elmer</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NEL745001KT</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NA</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">FISH, 1:1500IHC, 1:7500</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Recombinant Mouse IFN-gamma Protein</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">R&amp;D Systems (Biotechne)</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">485-MI-100/CF</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NA</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">100 ng/\\u03bcL</td>\\n              </tr>\\n            </tbody>\\n          </table>\\n\"}, \"text\": \"Name\\tManufacturer\\tCatalog\\tRRID\\tUsage\\t \\tChicken anti-TH\\tMillipore\\tAB9702\\tRRID: AB_570923\\tIHC, 1:500\\t \\tMouse anti-NeuN\\tMillipore\\tMAB377\\tRRID: AB_2298772\\tIHC, 1:500\\t \\tRabbit anti-Iba1\\tWako\\t019\\u201319741\\tRRID: AB_839504\\tIHC, 1:1000\\t \\tRat anti-Ki67\\tThermoFisher\\t14\\u20135698-82\\tRRID: AB_10854564\\tIHC, 1:1000\\t \\tMouse anti-NeuN, Alexa488\\tMillipore\\tMAB377X\\tRRID: AB_2149209\\tFANS, 1:500\\t \\tRabbit anti-pStat-Y701\\tCell Signaling\\t9167\\tRRID: AB_561284\\tIHC, 1:1000\\t \\tRat anti-DAT\\tMillipore\\tMAB369\\tRRID: AB_2190413\\tIHC, 1:500\\t \\tGoat anti-Chicken IgY (H + L), Alexa Fluor Plus 488\\tThermoFisher\\tA-32931TR\\tRRID: AB_2866499\\tIHC, 1:1000\\t \\tGoat anti-Mouse IgG (H + L), Alexa Fluor Plus 568\\tThermoFisher\\tA-11004\\tRRID: AB_2534072\\tIHC, 1:1000\\t \\tGoat anti-Rabbit IgG (H + L), Alexa Fluor 647\\tThermoFisher\\tA32733\\tRRID: AB_2633282\\tIHC, 1:1000\\t \\tGoat anti-Rat IgG (H + L), Alexa Fluor 647\\tThermoFisher\\tA-32931TR\\tRRID: AB_2866499\\tIHC, 1:1000\\t \\tMm-Ifngr1-C2\\tACD Biotechne\\t441491-C2\\tNA\\tFISH\\t \\tMm-Tap1-C2\\tACD Biotechne\\t575041-C2\\tNA\\tFISH\\t \\tPan-MHC Class 1**\\tACD Biotechne\\tNPR-0023100\\tNA\\tFISH\\t \\tTSA Cy3\\tPerkin Elmer\\tNEL744001KT\\tNA\\tFISH, 1:1500\\t \\tTSA Cy5\\tPerkin Elmer\\tNEL745001KT\\tNA\\tFISH, 1:1500IHC, 1:7500\\t \\tRecombinant Mouse IFN-gamma Protein\\tR&D Systems (Biotechne)\\t485-MI-100/CF\\tNA\\t100 ng/\\u03bcL\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 38533, \"infons\": {\"file\": \"T1.xml\", \"id\": \"T1\", \"section_type\": \"TABLE\", \"type\": \"table_footnote\"}, \"text\": \"FANS: fluorescence-activated nuclear sorting, IHC: immunohistochemical staining of brain sections, FISH: Fluorescence in situ hybridization.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 38674, \"infons\": {\"file\": \"T1.xml\", \"id\": \"T1\", \"section_type\": \"TABLE\", \"type\": \"table_footnote\"}, \"text\": \"Due to extensive homology between MHC Class I gene family members, probes selectively targeting H2-K1 and/or H2-D1 were not feasible. The new probe employed was designed to maximally cross-react with H2-K1 mRNA (NCBI Reference Sequence, NM_001001892.2), although substantial cross-reactivity is predicted for H2-K2, H2-D1, H2-D4, H2-Q7, H2-Q9, H2-Q10, and Gm38417.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 39039, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Immunohistochemistry (IHC)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 39066, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Mice were anesthetized with euthasol and transcardially perfused with ~ 15 mL of 0.9% saline followed by 40\\u201350 mL of ice-cold 4% paraformaldehyde (PFA) in 0.1 M phosphate buffer (PB), pH 7.4. Brains were post-fixed in 4% PFA in 0.1 M PB for 6\\u201312 h at 4 \\u00b0C, washed three times in phosphate buffered saline (PBS), and sectioned at 50 \\u03bcm on a Leica VT1000S vibratome. Sections were placed in cryoprotectant solution (30% ethylene glycol, 30% glycerol, 0.1 M PB, pH 7.4) and stored at \\u221220 \\u00b0C until further use.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 39582, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Sections were removed from cryoprotectant solution and washed three times in tris-buffered saline (TBS) at room temperature. Except for pSTAT1 staining, sections were then permeabilized in TBS + 0.2% Triton-X 100 for one hour at room temperature, followed by blocking in TBS + 10% normal goat serum (NGS) and 0.1% Triton-X 100 for 1.5 h at room temperature. Sections for pSTAT1 staining were slide mounted, submerged in 10 mM sodium citrate (pH 6.0) at 50 \\u00b0C for 15 min, and washed several times in TBS. All sections were then transferred to or submerged in a pre-chilled solution containing primary antibodies in TBS + 2% NGS + 0.1% Triton-X 100 and incubated overnight at 4 \\u00b0C. Sections were washed in TBS + 0.05% Tween 20 (TBS + T) five times over an hour at room temperature. Sections were incubated in a solution containing secondary antibodies in TBS + 2% NGS + 0.1% Triton-X 100 at room temperature for 1.5 h, followed by four washes in TBS + T over 45 min at room temperature. Sections were slide mounted and cover-slipped with Fluoromount G (Southern Biotech). See Antibodies and Reagents for a complete list of antibodies and concentrations used in this study.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 40755, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Fluorescence in situ hybridization (FISH)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 40797, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"FISH was performed using the highly sensitive RNAScope\\u00ae Multiplex Fluorescent v2 assay (ACD Bio). See Antibodies and Reagents for complete list of probes and reagents used in this study. Although most single FISH puncta using this assay are likely single mRNA molecules, this cannot be definitively determined due to the enzymatic signal amplification and non-diffraction-limited size of the mRNA puncta.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 41203, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Mouse brain sections were prepared as above, removed from cryoprotectant solution, and washed three times in tris-buffered saline (TBS) at room temperature. Sections were incubated with hydrogen peroxide (ACD) for 15 min at room temperature, washed several times in TBS, and then mounted to Superfrost slides (Fisher). Sections were allowed to dry for 10 min and a hydrophobic barrier (PAP pen, Vector Labs) was created around the tissue. Tissue was incubated in 50% EtOH, then 70% EtOH, then 100% EtOH for 5 min each. Sections were rehydrated in TBS for several minutes, digested with Protease IV (ACD) for 25 min at room temperature, and rinsed twice with TBS before proceeding to the RNA Scope Multiplex Fluorescent v2 assay (ACD).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 41938, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"The RNA Scope Multiplex Fluorescent v2 assay was conducted according to the manufacturer\\u2019s instructions, with all incubations taking place in a humidified chamber at 40 \\u00b0C. Two 5-minute washes in excess RNA Scope Wash Buffer (ACD) took place between each incubation in sequential order: probes (2-hours), AMP1 (30 min), AMP2 (30 min), AMP3 (15 min), HRP-C1/2/3 (15 min), TSA Cy3 (30 min), HRP blocker (30 min), HRP-C1/2/3 (15 min), and TSA Cy5 (30 min). Samples were washed twice more in RNA Scope Wash Buffer, then twice more in TBS. Samples were then blocked and immunostained for tyrosine hydroxylase as described above. After immunostaining, samples were mounted in Fluoromount G and stored at 4 \\u00b0C for up to 1 week before imaging.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 42678, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Image acquisition and analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 42709, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"All imaging of 50 \\u03bcm sections from perfusion-fixed brain was conducted either on a Nikon Ti2 Eclipse epifluorescence microscope or on a Leica SP8 scanning confocal microscope using a 20x/0.75NA or 60x/1.4NA objective.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 42928, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"For pSTAT1-Y701 analysis, 5 \\u03bcm Z stacks from the SNc were acquired using a 20x/0.75NA objective and collapsed via maximum projection. Three types of DAPI+ nuclei were segmented as ROIs: TH+/NeuN+ (mDA neurons), TH\\u2212/NeuN+ (non mDA neurons), and glia (TH\\u2212/NeuN\\u2212). The mean pSTAT1 intensity for all pixels within each ROI was background subtracted using the median of the entire image.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43318, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"For FISH analysis, 5 \\u03bcm Z stacks from the SNc were acquired using a 20x/0.75NA objective and collapsed via maximum projection. For quantification of mRNA density in neuronal somata, a binary threshold was set for the TH immunofluorescence signal based on two standard deviations above the image background to generate a binary mask of pixels for TH+ neurons. Segmentation of mDA neuronal somata was conducted in ImageJ to generate ROIs for each neuronal soma, after which the total signal in both RNA channels was measured. The mean RNA intensity for all pixels within each soma ROI was background subtracted using the median of the entire image.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43966, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Fluorescence-Activated nuclei sorting (FANS)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44011, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Nuclei were prepared from the ventral midbrain of mice at 48 h after injection of saline or IFN-\\u03b3. Mice were sacrificed by cervical dislocation and brains were rapidly extracted and submerged in ice-cold phosphate-buffered saline (PBS). Brains were placed on an ice-cold brain matrix (Zivic Instruments) and separated into 0.5\\u20131.0 mm sections using ice cold razor blades. Ventral midbrain tissue was dissected from slices between approximately \\u22122.5 mm to \\u22123.75 mm AP to Bregma. First, the cortex, hippocampi, and any hypothalamus or white matter ventral to the midbrain were removed. A single horizontal cut was made just dorsal to the rostral linear nucleus and all dorsal tissue was discarded. The remaining tissue containing the SN/VTA was flash frozen on liquid nitrogen and stored at \\u221280 \\u00b0C.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44818, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Nuclei preparation and sorting were conducted as described by with only minor modifications. Frozen tissue was thawed in nuclear isolation medium 1 (NIM1: 250 mM sucrose, 25 mM KCl, 5 mM MgCl2, 10 mM Tris, pH 8.0) supplemented with 1 mM DTT, 0.2 U/\\u03bcL SUPERaseIN, 1x EDTA-free Protease Inhibitor Cocktail (Roche), and 0.1% Triton X-100. Tissue was homogenized on ice in a glass-glass dounce homogenizer with 30 gentle strokes of loose and tight clearance pestles. All subsequent purification steps were performed on ice or at 4 \\u00b0C unless otherwise specified. Lysates were spun at 500xg for 5 min, resuspended in NIM1 supplemented as above and also with 1% BSA, and passed through a 40 \\u03bcm cell strainer cap. Nuclei were centrifuged again at 500xg for 5 min and resuspended in staining buffer (PBS supplemented with 1% BSA, 5 mM MgCl2, 0.2 U/\\u03bcL SUPERaseIN, 1 \\u03bcg/mL propidium iodide, and 2 \\u03bcg/mL mouse anti-NeuN AlexaFluor488). After 30 min incubation on ice, nuclei were centrifuged at 500xg for 5 min and resuspended in 0.5 mL of FACS buffer (PBS + 0.04% BSA, 0.2 U/\\u03bcL SUPERaseIN, and 1 \\u03bcg/mL propidium iodide) for sorting.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45948, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Nuclei sorting was conducted on a BD Influx (BD Biosciences) using a 100 \\u03bcm nozzle at 11.1 psi. Single nuclei were gated first using FSC-H vs. FSC-A and then on propidium iodide fluorescence. NeuN/AlexaFluor488 fluorescence was used to establish NeuN\\u2212 and NeuN+ gates on the single nuclei population. Nuclei were sorted directly into 96 well plates (for bulk nuclei RNA-seq with 96-well plate, pooled library construction) or into 1.5 mL Eppendorf tubes that had been previously coated with 0.1% BSA in PBS overnight at 37 \\u00b0C (for snRNA-seq).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46495, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Low input RNA sequencing with 96-well plate, pooled library construction\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46568, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"The protocol for plate based, 3\\u2032 end unique molecular indicator (UMI)-based RNA sequencing of single cells has been described previously as well as modifications to accommodate ultra-low input samples. See Supplementary File 1 for sequences of all custom primers and oligonucleotides used in this protocol. For neuronal and glial nuclei analysis in Figs. 1 and 2, 100 NeuN\\u2212 or NeuN+ nuclei were sorted directly into the wells of a 96 well plate containing 3 \\u03bcL of nuclease-free water containing 1 U/\\u03bcL SUPERaseIN (sort volume ~ 3 \\u03bcL for total volume of ~ 6 \\u03bcL). After adding 1.5 \\u03bcL of 10 \\u03bcM barcoded RT primer (Integrated DNA Technologies), primer annealing was performed at 72 \\u00b0C for 3 min. Reverse transcription was performed by adding 7.5 \\u03bcL RT mix to each well (2.81 \\u03bcL of 40% polyethylene glycol 8000, 0.15 \\u03bcL of 100 mM dNTPs, 3 \\u03bcL of 5X Maxima H RT Buffer, 0.2 \\u03bcL of 200 U/\\u03bcL Maxima H Reverse Transcriptase (ThermoFisher), 0.2 \\u03bcL of 20 U/\\u03bcL SUPERaseIN, and 0.15 \\u03bcL of 100 \\u03bcM Template Switching Oligo (Integrated DNA Technologies), and 1 \\u03bcL of nuclease free water). Reverse transcription was performed at 42 \\u00b0C for 90 min, followed by 10 cycles of 50 \\u00b0C for 2 min, 42 \\u00b0C for 2 min, 75 \\u00b0C for 10 min, followed by a 4 \\u00b0C hold. Excess primers were removed by adding 2 \\u03bcL of Exonuclease I mix (1.875U ExoI in water) to each well and incubating at 37 \\u00b0C for 30 min, 85 \\u00b0C for 15 min, 75 \\u00b0C for 30 s, 4 \\u00b0C hold.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 48011, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"All wells were pooled into a single 15-ml falcon tubes and cDNA was purified and concentrated using Dynabeads\\u2122 MyOne\\u2122 Silane beads (ThermoFisher) according to the manufacturer\\u2019s instructions. The cDNA was split into duplicate reactions containing 25 \\u03bcL cDNA, 25 \\u03bcL 2x HIFI HotStart Ready Mix (Kapa Biosystems), and 0.2 M SMART PCR Primer. PCR was run as follows: 37 \\u00b0C for 30 min, 85 \\u00b0C for 15 min, 75 \\u00b0C for 30 s, 4 \\u00b0C hold. Duplicate reactions were combined and purified using 0.7 volumes AMPure XP beads (Beckman Coulter). The amplified cDNA was visualized on an Agilent 2100 Bioanalyzer and quantified using a Qubit II fluorometer (ThermoFisher).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 48674, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Sequencing libraries were constructed using Nextera XT (Illumina) with modifications. A custom i5 primer was used (NexteraPCR) with 0.6 ng input cDNA and 10 cycles of amplification was performed. Unique i7 indexes were used for each plate. After amplification, the library was purified with two rounds of AMPure XP beads, visualized on the Agilent 2100 Bioanalyzer and quantified using the Qubit II fluorometer. Libraries were sequenced on an Illumina NextSeq 500 using the 75 cycle High Output kit (read lengths 26(R1) \\u00d7 8(i) \\u00d7 58(R2)). Custom sequencing primers were used for Read 1 (SMRT_R1seq and ILMN_R1-seq, see Antibodies and Reagents). With each plate we targeted ~ 400 M reads. Library pools were loaded at 1.8 pM with 20% PhiX (Illumina).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 49425, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Reads were aligned to the mouse reference genome GRCm38 and transcriptome annotation (Gencode vM10) using the STAR aligner with parameters \\u2013sjdbOverhang 65 \\u2013twopassMode Basic after trimming poly (A)-tails from the 3\\u2032-ends. The aligned reads were demultiplexed using the well-identifying barcodes, correcting all single-nucleotide errors. All reads with the same well-identifying barcode, UMI, and gene mapping were collapsed to represent an individual transcript. To correct for sequencing errors in UMIs, we further collapsed UMIs that were within Hamming distance one of another UMI with the same well-identifying barcode and gene. For each 96-well plate, after generating a final list of individual transcripts with unique combinations of well-identifying barcodes, UMIs, and gene mapping, we produced a molecular count matrix for downstream analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 50286, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Bulk nuclei RNA-Seq differential expression analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 50339, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Analysis of bulk nuclei RNA-Seq UMI count matrices as shown in Figs. 1 and 2 (from Low input RNA Sequencing with 96-well plate, pooled library construction) was conducted using a generalized linear model (GLM) in DESeq2. For analysis of IFN-\\u03b3 vs. saline analyses, a single contrast was made for each subset of NeuN\\u2212 or NeuN + nuclei at each timepoint (DESeq2 formula: ~Treatment). For interaction analysis at each timepoint (DESeq2 formula: ~NeuN + Treatment + NeuN:Treatment), the contrast specifies the difference in log2 Fold Change for the effect of Treatment (IFN-\\u03b3 vs. saline) between the levels of NeuN (i.e., for NeuPos.IFN, the difference in IFN-\\u03b3 vs. saline between NeuN\\u2212 and NeuN + samples). Global analysis of NeuN\\u2212 vs. NeuN + nuclei was conducted across all samples while controlling for the effect of IFN-\\u03b3 (DESeq2 formula: ~Treatment + NeuN). Complete DESeq2 output for all analyses related to Fig. 1 and Fig. 2 is found in Supplementary Table 1.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51317, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Single nucleus RNA-sequencing (snRNA-seq)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51359, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"NeuN nuclear sorting was conducted as described above. Approximately 70% NeuN+ and 30% NeuN\\u2212 single nuclei were sorted per sample. Sorted nuclei were placed on ice and immediately submitted to the Single Cell Analysis Core (Columbia Genome Center) for 10X Genomics Chromium Single Cell 3\\u2032 v3 library preparation. The resulting libraries were sequenced on an Illumina NovaSeq 6000 with a targeted depth of > 100,000 2x100 bp paired-end reads per nucleus.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51817, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"snRNA-seq data Processing, Clustering and doublet filtering\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51877, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"snRNA-seq data were processed as described previously (code available at: https://github.com/simslab/DropSeqPipeline8). Reads were aligned to the mouse reference genome GRCm38 and transcriptome annotation (Gencode vM10) using the STAR aligner with parameters \\u2013sjdbOverhang 65 \\u2013twopassMode Basic after trimming poly(A)-tails from the 3\\u2032-ends. All reads that uniquely aligned to a given gene body (including introns) were kept for downstream filtering analysis, because a large fraction of nuclear transcripts are unspliced. The aligned reads were demultiplexed using the cell-identifying barcodes, correcting all single-nucleotide errors. All reads with the same cell-identifying barcode, UMI, and gene mapping were collapsed to represent an individual transcript. To correct for sequencing errors in UMIs, we further collapsed UMIs that were within Hamming distance one of another UMI with the same cell-identifying barcode and gene. For each sample, after generating a final list of individual transcripts with unique combinations of cell-identifying barcodes, UMIs, and gene mapping, we produced a molecular count matrix for downstream analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 53031, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We first performed unsupervised clustering on the count matrices using the PhenoGraph implementation of Louvain community detection after selection of highly variable genes and construction of a k-nearest neighbors graph as described previously. We identified 38 clusters (Supp. Fig. 4a), two of which showed statistically significant co-enrichment of dopamine neuron markers such as Th and Slc6a3 based on the binomial test for expression specificity. We identified six major cell types with sufficient coverage for downstream analysis as shown in the UMAP embedding in Fig. 3a. The assignment of PhenoGraph clusters into major cell type groupings was based on key marker genes significantly enriched in each cluster, as shown in Supp. Fig. 4b: Oligodendrocytes (Mal, Mag, Mog), Astrocytes (Gja1, Fgfr3, Atp1a2), Microglia (C1qb, Siglech, Csf1r), VGLUT2 neurons (Slc17a6, Cacna2d1, Ntng1), GABA neurons (Slc32a1, Gad1, Gad2), and DA neurons (Th, Slc6a3, En1, Slc10a4). Rare populations with insufficient cell numbers (e.g., mural cells, T cells) or ambiguous neuronal clusters possibly containing multiplets (e.g., Gad1+/Gad2+/Slc17a7+/Th+) were excluded from downstream analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54213, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"While examining the neuronal profiles, we identified small subpopulations of nuclei with co-enrichment of neuronal and oligodendrocyte markers such as Mbp, Plp1, Mog, and Mag. To filter out oligodendrocyte doublets, we fit Gaussian mixture models to the sum of expression for oligodendrocyte marker genes within each major neuronal class (see Supp. Fig. 4c). The top 99 oligodendrocyte marker genes were used, as determined by the binomial test (log2 specificity > 3, FDR < 10e-50). For all non-oligodendrocyte clusters, we then removed nuclei with expression > 8 standard deviations above the mean of the first Gaussian component corresponding to single neurons (Supp. Fig. 4d). This process was repeated twice more using the top 99 astrocyte and top 99 microglial marker genes to filter out potential doublets.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55026, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"snRNA-Seq differential expression analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55069, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"For differential expression analysis shown in Fig. 3, we identified genes upregulated or downregulated by IFN-\\u03b3 within each cell type after filtering by conducting differential expression analysis as described in. Briefly, to perform differential expression analysis between two groups of cells, we randomly sub-sampled the data so that both groups are represented by the same number of cells. Next, we randomly sub-sampled the detected transcripts so that both groups have the same average number of transcripts per cell. Finally, we normalized the two sub-sampled count matrices using scran and analyzed differential expression for each gene using the SciPy implementation of the Mann-Whitney U test. We corrected the resulting p-values for false discovery using the Benjamini-Hochberg procedure as implemented in the statsmodels package in Python. We compared each IFN-\\u03b3 sample to the saline control and retained only genes with > 8-fold differential expression \\u2223log2FC > 3\\u2223 and FDR < 0.01 in both IFN-\\u03b3 samples. Complete differential expression output for all analyses related to Fig. 3 is reported in Supplementary Table 2.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56210, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Gene ontology (GO) analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56238, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"A single list of unique genes was used (i.e., differentially expressed genes from DESeq2 analysis) to conduct GO Analysis using web-based Enrichr with 2018 GO Terms for Cellular Component, Biological Process, and Molecular Function.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56471, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Visualization and statistical analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56510, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Graphics were created in Adobe Illustrator 24.3 (Adobe, Inc.). Unless otherwise noted, all statistical analysis and data visualization was conducted in Python using SciPy, Matplotlib, and Seaborn packages. Statistical comparisons were conducted using the Mann-Whitney U test with number of replicates and other statistical testing information indicated in the figure captions. For visualization of RNA-seq abundance data, mRNA UMI counts were scaled up by 106 and normalized to total counts (CPM), and log2 transformed after adding 1. Normalized mRNA abundance is referred to as \\u2018log2(CPM + 1)\\u2019 as specified in figure captions. For clustered heatmaps, Z-scores of log2(CPM + 1) were first calculated using the zscore function within the SciPy Stats module, after which the row and column clustering was calculated using the linkage function (metric = \\u2018Euclidean\\u2019, method = \\u2018average\\u2019) within fastcluster 1.2.3 and passed to Seaborn clustermap.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57465, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title_1\"}, \"text\": \"Supplementary Material\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57488, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"Data and materials availability\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57520, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"There are restrictions to the availability of custom-order RNA Scope probes generated for this study due to limited production size. The exact new probe design IDs are listed in Antibodies and Reagents and are available for further production from ACD Biotechne.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57783, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"All bulk nuclear RNA-seq and snRNA-seq data from this study are publicly available on the Gene Expression Omnibus as GSE218132. Count matrices used in this study are in Supplementary File 3 and Supplementary File 5.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57999, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"Python and Shell code used for processing of RNA-seq data is accessible at: https://github.com/simslab/DropSeqPipeline8, and Python code used for clustering and visualization of scRNA-seq data can be found at https://www.github.com/simslab/cluster_diffex2018.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58259, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"Appendices A and B. Supplementary data\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58298, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"Supplementary data to this article can be found online at https://doi.org/10.1016/j.bbi.2023.04.008.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58399, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"Declaration of Competing Interest\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58433, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58604, \"infons\": {\"section_type\": \"REF\", \"type\": \"title\"}, \"text\": \"References\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58615, \"infons\": {\"fpage\": \"151\", \"lpage\": \"166\", \"name_0\": \"surname:Amor;given-names:S\", \"name_1\": \"surname:Peferoen;given-names:LAN\", \"name_2\": \"surname:Vogel;given-names:DYS\", \"name_3\": \"surname:Breur;given-names:M\", \"name_4\": \"surname:van der Valk;given-names:P\", \"name_5\": \"surname:Baker;given-names:D\", \"name_6\": \"surname:van Noort;given-names:JM\", \"pub-id_doi\": \"10.1111/imm.12233\", \"pub-id_pmid\": \"24329535\", \"section_type\": \"REF\", \"source\": \"Immunology\", \"type\": \"ref\", \"volume\": \"142\", \"year\": \"2014\"}, \"text\": \"Inflammation in neurodegenerative diseases\\u2013an update\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58670, \"infons\": {\"fpage\": \"25\", \"lpage\": \"29\", \"name_0\": \"surname:Ashburner;given-names:M\", \"name_1\": \"surname:Ball;given-names:CA\", \"name_10\": \"surname:Harris;given-names:MA\", \"name_11\": \"surname:Hill;given-names:DP\", \"name_12\": \"surname:Issel-Tarver;given-names:L\", \"name_13\": \"surname:Kasarskis;given-names:A\", \"name_14\": \"surname:Lewis;given-names:S\", \"name_15\": \"surname:Matese;given-names:JC\", \"name_16\": \"surname:Richardson;given-names:JE\", \"name_17\": \"surname:Ringwald;given-names:M\", \"name_18\": \"surname:Rubin;given-names:GM\", \"name_19\": \"surname:Sherlock;given-names:G\", \"name_2\": \"surname:Blake;given-names:JA\", \"name_3\": \"surname:Botstein;given-names:D\", \"name_4\": \"surname:Butler;given-names:H\", \"name_5\": \"surname:Cherry;given-names:JM\", \"name_6\": \"surname:Davis;given-names:AP\", \"name_7\": \"surname:Dolinski;given-names:K\", \"name_8\": \"surname:Dwight;given-names:SS\", \"name_9\": \"surname:Eppig;given-names:JT\", \"pub-id_doi\": \"10.1038/75556\", \"pub-id_pmid\": \"10802651\", \"section_type\": \"REF\", \"source\": \"Nat. Genet\", \"type\": \"ref\", \"volume\": \"25\", \"year\": \"2000\"}, \"text\": \"Gene Ontology: tool for the unification of biology\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58721, \"infons\": {\"fpage\": \"7922\", \"issue\": \"21\", \"lpage\": \"7930\", \"name_0\": \"surname:Babcock;given-names:AA\", \"name_1\": \"surname:Kuziel;given-names:WA\", \"name_2\": \"surname:Rivest;given-names:S\", \"name_3\": \"surname:Owens;given-names:T\", \"pub-id_pmid\": \"12944523\", \"section_type\": \"REF\", \"source\": \"J. Neurosci\", \"type\": \"ref\", \"volume\": \"23\", \"year\": \"2003\"}, \"text\": \"Chemokine expression by glial cells directs leukocytes to sites of axonal injury in the CNS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58813, \"infons\": {\"fpage\": \"383\", \"lpage\": \"390\", \"name_0\": \"surname:B\\u00e4ckman;given-names:CM\", \"name_1\": \"surname:Malik;given-names:N\", \"name_2\": \"surname:Zhang;given-names:Y\", \"name_3\": \"surname:Shan;given-names:L\", \"name_4\": \"surname:Grinberg;given-names:A\", \"name_5\": \"surname:Hoffer;given-names:BJ\", \"name_6\": \"surname:Westphal;given-names:H\", \"name_7\": \"surname:Tomac;given-names:AC\", \"pub-id_doi\": \"10.1002/dvg.20228\", \"pub-id_pmid\": \"16865686\", \"section_type\": \"REF\", \"source\": \"Genesis\", \"type\": \"ref\", \"volume\": \"44\", \"year\": \"2006\"}, \"text\": \"Characterization of a mouse strain expressing Cre recombinase from the 3\\u2019 untranslated region of the dopamine transporter locus\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58943, \"infons\": {\"fpage\": \"e142\", \"name_0\": \"surname:Barcia;given-names:C\", \"name_1\": \"surname:Ros;given-names:CM\", \"name_2\": \"surname:Annese;given-names:V\", \"name_3\": \"surname:G\\u00f3mez;given-names:A\", \"name_4\": \"surname:Ros-Bernal;given-names:F\", \"name_5\": \"surname:Aguado-Yera;given-names:D\", \"name_6\": \"surname:Mart\\u00ednez-Pag\\u00e1n;given-names:ME\", \"name_7\": \"surname:de Pablos;given-names:V\", \"name_8\": \"surname:Fernandez-Villalba;given-names:E\", \"name_9\": \"surname:Herrero;given-names:MT\", \"pub-id_doi\": \"10.1038/cddis.2011.17\", \"pub-id_pmid\": \"21472005\", \"section_type\": \"REF\", \"source\": \"Cell Death Dis\", \"type\": \"ref\", \"volume\": \"2\", \"year\": \"2011\"}, \"text\": \"IFN-\\u03b3 signaling, with the synergistic contribution of TNF-\\u03b1, mediates cell specific microglial and astroglial activation in experimental models of Parkinson\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59118, \"infons\": {\"fpage\": \"38\", \"issue\": \"1\", \"lpage\": \"44\", \"name_0\": \"surname:Becht;given-names:E\", \"name_1\": \"surname:McInnes;given-names:L\", \"name_2\": \"surname:Healy;given-names:J\", \"name_3\": \"surname:Dutertre;given-names:C-A\", \"name_4\": \"surname:Kwok;given-names:IWH\", \"name_5\": \"surname:Ng;given-names:LG\", \"name_6\": \"surname:Ginhoux;given-names:F\", \"name_7\": \"surname:Newell;given-names:EW\", \"section_type\": \"REF\", \"source\": \"Nat. Biotechnol\", \"type\": \"ref\", \"volume\": \"37\", \"year\": \"2019\"}, \"text\": \"Dimensionality reduction for visualizing single-cell data using UMAP\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59187, \"infons\": {\"fpage\": \"10956\", \"lpage\": \"10961\", \"name_0\": \"surname:Bernard-Valnet;given-names:R\", \"name_1\": \"surname:Yshii;given-names:L\", \"name_10\": \"surname:Pignolet;given-names:B\", \"name_11\": \"surname:Dauvilliers;given-names:Y\", \"name_12\": \"surname:Peyron;given-names:C\", \"name_13\": \"surname:Liblau;given-names:RS\", \"name_2\": \"surname:Qu\\u00e9riauit;given-names:C\", \"name_3\": \"surname:Nguyen;given-names:X-H\", \"name_4\": \"surname:Arthaud;given-names:S\", \"name_5\": \"surname:Rodrigues;given-names:M\", \"name_6\": \"surname:Canivet;given-names:A\", \"name_7\": \"surname:Morel;given-names:A-L\", \"name_8\": \"surname:Matthys;given-names:A\", \"name_9\": \"surname:Bauer;given-names:J\", \"pub-id_doi\": \"10.1073/pnas.1603325113\", \"pub-id_pmid\": \"27621438\", \"section_type\": \"REF\", \"source\": \"PNAS\", \"type\": \"ref\", \"volume\": \"113\", \"year\": \"2016\"}, \"text\": \"CD8 T cell-mediated killing of orexinergic neurons induces a narcolepsy-like phenotype in mice\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59282, \"infons\": {\"fpage\": \"182\", \"lpage\": \"192\", \"name_0\": \"surname:Brochard;given-names:V\", \"name_1\": \"surname:Combadi\\u00e8re;given-names:B\", \"name_10\": \"surname:Duyckaerts;given-names:C\", \"name_11\": \"surname:Flavell;given-names:RA\", \"name_12\": \"surname:Hirsch;given-names:EC\", \"name_13\": \"surname:Hunot;given-names:S\", \"name_2\": \"surname:Prigent;given-names:A\", \"name_3\": \"surname:Laouar;given-names:Y\", \"name_4\": \"surname:Perrin;given-names:A\", \"name_5\": \"surname:Beray-Berthat;given-names:V\", \"name_6\": \"surname:Bonduelle;given-names:O\", \"name_7\": \"surname:Alvarez-Fischer;given-names:D\", \"name_8\": \"surname:Callebert;given-names:J\", \"name_9\": \"surname:Launay;given-names:J-M\", \"pub-id_doi\": \"10.1172/JCI36470\", \"pub-id_pmid\": \"19104149\", \"section_type\": \"REF\", \"source\": \"J. clin. Invest\", \"type\": \"ref\", \"volume\": \"119\", \"year\": \"2009\"}, \"text\": \"Infiltration of CD4+ lymphocytes into the brain contributes to neurodegeneration in a mouse model of Parkinson disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59401, \"infons\": {\"fpage\": \"150\", \"lpage\": \"157\", \"name_0\": \"surname:Casta\\u00f1o;given-names:A\", \"name_1\": \"surname:Herrera;given-names:AJ\", \"name_2\": \"surname:Cano;given-names:J\", \"name_3\": \"surname:Machado;given-names:A\", \"pub-id_doi\": \"10.1046/j.1471-4159.2002.00799.x\", \"pub-id_pmid\": \"12067227\", \"section_type\": \"REF\", \"source\": \"J. Neurochem\", \"type\": \"ref\", \"volume\": \"81\", \"year\": \"2002\"}, \"text\": \"The degenerative effect of a single intranigral injection of LPS on the dopaminergic system is prevented by dexamethasone, and not mimicked by rh-TNF-alpha, IL-1beta and IFN-gamma\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59581, \"infons\": {\"fpage\": \"3633\", \"name_0\": \"surname:Cebri\\u00e1n;given-names:C\", \"name_1\": \"surname:Zucca;given-names:FA\", \"name_10\": \"surname:Zecca;given-names:L\", \"name_11\": \"surname:Loike;given-names:JD\", \"name_12\": \"surname:Sulzer;given-names:D\", \"name_2\": \"surname:Mauri;given-names:P\", \"name_3\": \"surname:Steinbeck;given-names:JA\", \"name_4\": \"surname:Studer;given-names:L\", \"name_5\": \"surname:Scherzer;given-names:CR\", \"name_6\": \"surname:Kanter;given-names:E\", \"name_7\": \"surname:Budhu;given-names:S\", \"name_8\": \"surname:Mandelbaum;given-names:J\", \"name_9\": \"surname:Vonsattel;given-names:JP\", \"pub-id_doi\": \"10.1038/ncomms4633\", \"pub-id_pmid\": \"24736453\", \"section_type\": \"REF\", \"source\": \"Nat. Commun\", \"type\": \"ref\", \"volume\": \"5\", \"year\": \"2014\"}, \"text\": \"MHC-I expression renders catecholaminergic neurons susceptible to T-cell-mediated degeneration\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59676, \"infons\": {\"fpage\": \"694\", \"lpage\": \"696\", \"name_0\": \"surname:Chakrabarty;given-names:P\", \"name_1\": \"surname:Ceballos-Diaz;given-names:C\", \"name_2\": \"surname:Lin;given-names:W-L\", \"name_3\": \"surname:Beccard;given-names:A\", \"name_4\": \"surname:Jansen-West;given-names:K\", \"name_5\": \"surname:McFarland;given-names:NR\", \"name_6\": \"surname:Janus;given-names:C\", \"name_7\": \"surname:Dickson;given-names:D\", \"name_8\": \"surname:Das;given-names:P\", \"name_9\": \"surname:Golde;given-names:TE\", \"pub-id_doi\": \"10.1038/nn.2829\", \"pub-id_pmid\": \"21572432\", \"section_type\": \"REF\", \"source\": \"Nat. Neurosci\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2011\"}, \"text\": \"Interferon-\\u03b3 induces progressive nigrostriatal degeneration and basal ganglia calcification\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59772, \"infons\": {\"fpage\": \"e0151569\", \"issue\": \"3\", \"name_0\": \"surname:Chien;given-names:C-H\", \"name_1\": \"surname:Lee;given-names:M-J\", \"name_2\": \"surname:Liou;given-names:H-C\", \"name_3\": \"surname:Liou;given-names:H-H\", \"name_4\": \"surname:Fu;given-names:W-M\", \"name_5\": \"surname:Block;given-names:ML\", \"pub-id_pmid\": \"26982707\", \"section_type\": \"REF\", \"source\": \"PLoS One\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2016\"}, \"text\": \"Microglia-derived cytokines/chemokines are involved in the enhancement of LPS-induced loss of nigrostriatal dopaminergic neurons in DJ-1 knockout mice\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59923, \"infons\": {\"fpage\": \"1709\", \"lpage\": \"1725\", \"name_0\": \"surname:Chung;given-names:CY\", \"name_1\": \"surname:Seo;given-names:H\", \"name_2\": \"surname:Sonntag;given-names:KC\", \"name_3\": \"surname:Brooks;given-names:A\", \"name_4\": \"surname:Lin;given-names:L\", \"name_5\": \"surname:Isacson;given-names:O\", \"pub-id_doi\": \"10.1093/hmg/ddi178\", \"pub-id_pmid\": \"15888489\", \"section_type\": \"REF\", \"source\": \"Hum. Mol. Genet\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2005\"}, \"text\": \"Cell type-specific gene expression of midbrain dopaminergic neurons reveals molecules involved in their vulnerability and protection\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60056, \"infons\": {\"fpage\": \"172\", \"lpage\": \"185\", \"name_0\": \"surname:Clarkson;given-names:BDS\", \"name_1\": \"surname:Patel;given-names:MS\", \"name_2\": \"surname:LaFrance-Corey;given-names:RG\", \"name_3\": \"surname:Howe;given-names:CL\", \"pub-id_doi\": \"10.1002/acn3.516\", \"pub-id_pmid\": \"29468178\", \"section_type\": \"REF\", \"source\": \"Ann. Clin. Transl. Neurol\", \"type\": \"ref\", \"volume\": \"5\", \"year\": \"2018\"}, \"text\": \"Retrograde interferon-gamma signaling induces major histocompatibility class I expression in human-induced pluripotent stem cell-derived neurons\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60201, \"infons\": {\"fpage\": \"505\", \"lpage\": \"520\", \"name_0\": \"surname:Corriveau;given-names:RA\", \"name_1\": \"surname:Huh;given-names:GS\", \"name_2\": \"surname:Shatz;given-names:CJ\", \"pub-id_doi\": \"10.1016/S0896-6273(00)80562-0\", \"pub-id_pmid\": \"9768838\", \"section_type\": \"REF\", \"source\": \"Neuron\", \"type\": \"ref\", \"volume\": \"21\", \"year\": \"1998\"}, \"text\": \"Regulation of class I MHC gene expression in the developing and mature CNS by neural activity\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60295, \"infons\": {\"fpage\": \"1\", \"lpage\": \"10\", \"name_0\": \"surname:Dhanwani;given-names:R\", \"name_1\": \"surname:Lima-Junior;given-names:JR\", \"name_10\": \"surname:Litvan;given-names:I\", \"name_11\": \"surname:Alcalay;given-names:RN\", \"name_12\": \"surname:Peters;given-names:B\", \"name_13\": \"surname:Sulzer;given-names:D\", \"name_14\": \"surname:Arlehamn;given-names:CSL\", \"name_15\": \"surname:Sette;given-names:A\", \"name_2\": \"surname:Sethi;given-names:A\", \"name_3\": \"surname:Pham;given-names:J\", \"name_4\": \"surname:Williams;given-names:G\", \"name_5\": \"surname:Frazier;given-names:A\", \"name_6\": \"surname:Xu;given-names:Y\", \"name_7\": \"surname:Amara;given-names:AW\", \"name_8\": \"surname:Standaert;given-names:DG\", \"name_9\": \"surname:Goldman;given-names:JG\", \"pub-id_doi\": \"10.1038/s41531-022-00282-2\", \"pub-id_pmid\": \"35013321\", \"section_type\": \"REF\", \"source\": \"NPJ Parkinsons Dis\", \"type\": \"ref\", \"volume\": \"8\", \"year\": \"2022\"}, \"text\": \"Transcriptional analysis of peripheral memory T cells reveals Parkinson\\u2019s disease-specific gene signatures\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60404, \"infons\": {\"fpage\": \"458\", \"lpage\": \"471.e19\", \"name_0\": \"surname:Di Liberto;given-names:G\", \"name_1\": \"surname:Pantelyushin;given-names:S\", \"name_10\": \"surname:Salinas;given-names:G\", \"name_11\": \"surname:Lin-Marq;given-names:N\", \"name_12\": \"surname:Staszewski;given-names:O\", \"name_13\": \"surname:Costa Jord\\u00e3o;given-names:MJ\", \"name_14\": \"surname:Wagner;given-names:I\", \"name_15\": \"surname:Egervari;given-names:K\", \"name_16\": \"surname:Mack;given-names:M\", \"name_17\": \"surname:Bellone;given-names:C\", \"name_18\": \"surname:Bl\\u00fcmcke;given-names:I\", \"name_19\": \"surname:Prinz;given-names:M\", \"name_2\": \"surname:Kreutzfeldt;given-names:M\", \"name_20\": \"surname:Pinschewer;given-names:DD\", \"name_21\": \"surname:Merkler;given-names:D\", \"name_3\": \"surname:Page;given-names:N\", \"name_4\": \"surname:Musardo;given-names:S\", \"name_5\": \"surname:Coras;given-names:R\", \"name_6\": \"surname:Steinbach;given-names:K\", \"name_7\": \"surname:Vincenti;given-names:I\", \"name_8\": \"surname:Klimek;given-names:B\", \"name_9\": \"surname:Lingner;given-names:T\", \"pub-id_doi\": \"10.1016/j.cell.2018.07.049\", \"pub-id_pmid\": \"30173917\", \"section_type\": \"REF\", \"source\": \"Cell\", \"type\": \"ref\", \"volume\": \"175\", \"year\": \"2018\"}, \"text\": \"Neurons under T cell attack coordinate phagocyte-mediated synaptic stripping\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60481, \"infons\": {\"fpage\": \"205\", \"lpage\": \"210\", \"name_0\": \"surname:Dulken;given-names:BW\", \"name_1\": \"surname:Buckley;given-names:MT\", \"name_10\": \"surname:Wyss-Coray;given-names:T\", \"name_11\": \"surname:Weissman;given-names:IL\", \"name_12\": \"surname:Vogel;given-names:H\", \"name_13\": \"surname:Davis;given-names:MM\", \"name_14\": \"surname:Brunet;given-names:A\", \"name_2\": \"surname:Navarro Negredo;given-names:P\", \"name_3\": \"surname:Saligrama;given-names:N\", \"name_4\": \"surname:Cayrol;given-names:R\", \"name_5\": \"surname:Leeman;given-names:DS\", \"name_6\": \"surname:George;given-names:BM\", \"name_7\": \"surname:Boutet;given-names:SC\", \"name_8\": \"surname:Hebestreit;given-names:K\", \"name_9\": \"surname:Pluvinage;given-names:JV\", \"pub-id_doi\": \"10.1038/S41586-019-1362-5\", \"pub-id_pmid\": \"31270459\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"571\", \"year\": \"2019\"}, \"text\": \"Single-cell analysis reveals T cell infiltration in old neurogenic niches\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60555, \"infons\": {\"fpage\": \"1837\", \"issue\": \"12\", \"lpage\": \"1844\", \"name_0\": \"surname:Falc\\u00e3o;given-names:AM\", \"name_1\": \"surname:van Bruggen;given-names:D\", \"name_10\": \"surname:Williams;given-names:A\", \"name_11\": \"surname:Guerreiro-Cacais;given-names:AO\", \"name_12\": \"surname:Castelo-Branco;given-names:G\", \"name_2\": \"surname:Marques;given-names:S\", \"name_3\": \"surname:Meijer;given-names:M\", \"name_4\": \"surname:J\\u00e4kel;given-names:S\", \"name_5\": \"surname:Agirre;given-names:E\", \"name_6\": \"surname:Samudyata\", \"name_7\": \"surname:Floriddia;given-names:EM\", \"name_8\": \"surname:Vanichkina;given-names:DP\", \"name_9\": \"surname:ffrench-Constant;given-names:C\", \"pub-id_pmid\": \"30420755\", \"section_type\": \"REF\", \"source\": \"Nat. Med\", \"type\": \"ref\", \"volume\": \"24\", \"year\": \"2018\"}, \"text\": \"Disease-specific oligodendrocyte lineage cells arise in multiple sclerosis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60630, \"infons\": {\"fpage\": \"705\", \"lpage\": \"719\", \"name_0\": \"surname:Ferrari;given-names:DP\", \"name_1\": \"surname:Bortolanza;given-names:M\", \"name_2\": \"surname:Del Bel;given-names:EA\", \"pub-id_doi\": \"10.1007/s12640-021-00345-x\", \"pub-id_pmid\": \"33687725\", \"section_type\": \"REF\", \"source\": \"Neurotox. Res\", \"type\": \"ref\", \"volume\": \"39\", \"year\": \"2021\"}, \"text\": \"Interferon-\\u03b3 involvement in the neuroinflammation associated with Parkinson\\u2019s disease and L-DOPA-induced dyskinesia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60750, \"infons\": {\"fpage\": \"425\", \"lpage\": \"429\", \"name_0\": \"surname:Filiano;given-names:AJ\", \"name_1\": \"surname:Xu;given-names:Y\", \"name_10\": \"surname:Peerzade;given-names:SN\", \"name_11\": \"surname:Chen;given-names:H\", \"name_12\": \"surname:Lee;given-names:KS\", \"name_13\": \"surname:Scott;given-names:MM\", \"name_14\": \"surname:Beenhakker;given-names:MP\", \"name_15\": \"surname:Litvak;given-names:V\", \"name_16\": \"surname:Kipnis;given-names:J\", \"name_2\": \"surname:Tustison;given-names:NJ\", \"name_3\": \"surname:Marsh;given-names:RL\", \"name_4\": \"surname:Baker;given-names:W\", \"name_5\": \"surname:Smirnov;given-names:I\", \"name_6\": \"surname:Overall;given-names:CC\", \"name_7\": \"surname:Gadani;given-names:SP\", \"name_8\": \"surname:Turner;given-names:SD\", \"name_9\": \"surname:Weng;given-names:Z\", \"pub-id_doi\": \"10.1038/nature18626\", \"pub-id_pmid\": \"27409813\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"535\", \"year\": \"2016\"}, \"text\": \"Unexpected role of interferon-\\u03b3 in regulating neuronal connectivity and social behavior\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60842, \"infons\": {\"fpage\": \"577050\", \"name_0\": \"surname:Flood;given-names:L\", \"name_1\": \"surname:Korol;given-names:SV\", \"name_2\": \"surname:Ekselius;given-names:L\", \"name_3\": \"surname:Birnir;given-names:B\", \"name_4\": \"surname:Jin;given-names:Z\", \"pub-id_doi\": \"10.1016/j.jneuroim.2019.577050\", \"pub-id_pmid\": \"31505410\", \"section_type\": \"REF\", \"source\": \"J. Neuroimmunol\", \"type\": \"ref\", \"volume\": \"337\", \"year\": \"2019\"}, \"text\": \"Interferon-\\u03b3 potentiates GABAA receptor-mediated inhibitory currents in rat hippocampal CA1 pyramidal neurons\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60956, \"infons\": {\"fpage\": \"3717\", \"lpage\": \"3733\", \"name_0\": \"surname:Galiano-Landeira;given-names:J\", \"name_1\": \"surname:Torra;given-names:A\", \"name_2\": \"surname:Vila;given-names:M\", \"name_3\": \"surname:Bov\\u00e9;given-names:J\", \"pub-id_doi\": \"10.1093/brain/awaa269\", \"pub-id_pmid\": \"33118032\", \"section_type\": \"REF\", \"source\": \"Brain\", \"type\": \"ref\", \"volume\": \"143\", \"year\": \"2020\"}, \"text\": \"CD8 T cell nigral infiltration precedes synucleinopathy in early stages of Parkinson\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61053, \"infons\": {\"fpage\": \"439\", \"lpage\": \"455\", \"name_0\": \"surname:Garretti;given-names:F\", \"name_1\": \"surname:Monahan;given-names:C\", \"name_2\": \"surname:Sette;given-names:A\", \"name_3\": \"surname:Agalliu;given-names:D\", \"name_4\": \"surname:Sulzer;given-names:D\", \"pub-id_doi\": \"10.1016/B978-0-12-819410-2.00023-0\", \"pub-id_pmid\": \"35034753\", \"section_type\": \"REF\", \"source\": \"Handb. Clin. Neurol\", \"type\": \"ref\", \"volume\": \"184\", \"year\": \"2022\"}, \"text\": \"T cells, \\u03b1-synuclein and Parkinson disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61100, \"infons\": {\"fpage\": \"399\", \"lpage\": \"404\", \"name_0\": \"surname:Gate;given-names:D\", \"name_1\": \"surname:Saligrama;given-names:N\", \"name_10\": \"surname:McBride;given-names:A\", \"name_11\": \"surname:Pluvinage;given-names:J\", \"name_12\": \"surname:Elahi;given-names:F\", \"name_13\": \"surname:Tam;given-names:G-K-Y\", \"name_14\": \"surname:Kim;given-names:Y\", \"name_15\": \"surname:Greicius;given-names:M\", \"name_16\": \"surname:Wagner;given-names:AD\", \"name_17\": \"surname:Aigner;given-names:L\", \"name_18\": \"surname:Galasko;given-names:DR\", \"name_19\": \"surname:Davis;given-names:MM\", \"name_2\": \"surname:Leventhal;given-names:O\", \"name_20\": \"surname:Wyss-Coray;given-names:T\", \"name_3\": \"surname:Yang;given-names:AC\", \"name_4\": \"surname:Unger;given-names:MS\", \"name_5\": \"surname:Middeldorp;given-names:J\", \"name_6\": \"surname:Chen;given-names:K\", \"name_7\": \"surname:Lehallier;given-names:B\", \"name_8\": \"surname:Channappa;given-names:D\", \"name_9\": \"surname:De Los Santos;given-names:MB\", \"pub-id_doi\": \"10.1038/S41586-019-1895-7\", \"pub-id_pmid\": \"31915375\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"577\", \"year\": \"2020\"}, \"text\": \"Clonally expanded CD8 T cells patrol the cerebrospinal fluid in Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61186, \"infons\": {\"fpage\": \"868\", \"lpage\": \"874\", \"name_0\": \"surname:Gate;given-names:D\", \"name_1\": \"surname:Tapp;given-names:E\", \"name_10\": \"surname:Channappa;given-names:D\", \"name_11\": \"surname:Henderson;given-names:VW\", \"name_12\": \"surname:Keller;given-names:A\", \"name_13\": \"surname:Aigner;given-names:L\", \"name_14\": \"surname:Galasko;given-names:DR\", \"name_15\": \"surname:Davis;given-names:MM\", \"name_16\": \"surname:Poston;given-names:KL\", \"name_17\": \"surname:Wyss-Coray;given-names:T\", \"name_2\": \"surname:Leventhal;given-names:O\", \"name_3\": \"surname:Shahid;given-names:M\", \"name_4\": \"surname:Nonninger;given-names:TJ\", \"name_5\": \"surname:Yang;given-names:AC\", \"name_6\": \"surname:Strempfl;given-names:K\", \"name_7\": \"surname:Unger;given-names:MS\", \"name_8\": \"surname:Fehlmann;given-names:T\", \"name_9\": \"surname:Oh;given-names:H\", \"pub-id_doi\": \"10.1126/science.abf7266\", \"pub-id_pmid\": \"34648304\", \"section_type\": \"REF\", \"source\": \"Science\", \"type\": \"ref\", \"volume\": \"374\", \"year\": \"2021\"}, \"text\": \"CD4+ T cells contribute to neurodegeneration in Lewy body dementia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61253, \"infons\": {\"fpage\": \"D325\", \"lpage\": \"D334\", \"pub-id_doi\": \"10.1093/nar/gkaa1113\", \"pub-id_pmid\": \"33290552\", \"section_type\": \"REF\", \"source\": \"Nucleic Acids Res\", \"type\": \"ref\", \"volume\": \"49\", \"year\": \"2021\"}, \"text\": \"The Gene Ontology resource: enriching a GOld mine\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61303, \"infons\": {\"fpage\": \"20918\", \"lpage\": \"20923\", \"name_0\": \"surname:Gottfried-Blackmore;given-names:A\", \"name_1\": \"surname:Kaunzner;given-names:UW\", \"name_2\": \"surname:Idoyaga;given-names:J\", \"name_3\": \"surname:Felger;given-names:JC\", \"name_4\": \"surname:McEwen;given-names:BS\", \"name_5\": \"surname:Bulloch;given-names:K\", \"pub-id_doi\": \"10.1073/pnas.0911509106\", \"pub-id_pmid\": \"19906988\", \"section_type\": \"REF\", \"source\": \"PNAS\", \"type\": \"ref\", \"volume\": \"106\", \"year\": \"2009\"}, \"text\": \"Acute in vivo exposure to interferon-gamma enables resident brain dendritic cells to become effective antigen presenting cells\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61430, \"infons\": {\"fpage\": \"82\", \"lpage\": \"90\", \"name_0\": \"surname:Gunsolly;given-names:C\", \"name_1\": \"surname:Nicholson;given-names:JD\", \"name_2\": \"surname:Listwak;given-names:SJ\", \"name_3\": \"surname:Ledee;given-names:D\", \"name_4\": \"surname:Zelenka;given-names:P\", \"name_5\": \"surname:Verthelyi;given-names:D\", \"name_6\": \"surname:Chapoval;given-names:S\", \"name_7\": \"surname:Keegan;given-names:A\", \"name_8\": \"surname:Tonelli;given-names:LH\", \"pub-id_doi\": \"10.1016/j.jneuroim.2010.04.019\", \"pub-id_pmid\": \"20605223\", \"section_type\": \"REF\", \"source\": \"J. Neuroimmunol\", \"type\": \"ref\", \"volume\": \"225\", \"year\": \"2010\"}, \"text\": \"Expression and regulation in the brain of the chemokine CCL27 gene locus\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61503, \"infons\": {\"fpage\": \"10896\", \"lpage\": \"10901\", \"name_0\": \"surname:Harrison;given-names:JK\", \"name_1\": \"surname:Jiang;given-names:Y\", \"name_10\": \"surname:Botti;given-names:P\", \"name_11\": \"surname:Bacon;given-names:KB\", \"name_12\": \"surname:Feng;given-names:L\", \"name_2\": \"surname:Chen;given-names:S\", \"name_3\": \"surname:Xia;given-names:Y\", \"name_4\": \"surname:Maciejewski;given-names:D\", \"name_5\": \"surname:McNamara;given-names:RK\", \"name_6\": \"surname:Streit;given-names:WJ\", \"name_7\": \"surname:Salafranca;given-names:MN\", \"name_8\": \"surname:Adhikari;given-names:S\", \"name_9\": \"surname:Thompson;given-names:DA\", \"pub-id_doi\": \"10.1073/pnas.95.18.10896\", \"pub-id_pmid\": \"9724801\", \"section_type\": \"REF\", \"source\": \"PNAS\", \"type\": \"ref\", \"volume\": \"95\", \"year\": \"1998\"}, \"text\": \"Role for neuronally derived fractalkine in mediating interactions between neurons and CX3CR1-expressing microglia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61617, \"infons\": {\"fpage\": \"345\", \"lpage\": \"348\", \"name_0\": \"surname:Hirsch;given-names:E\", \"name_1\": \"surname:Graybiel;given-names:AM\", \"name_2\": \"surname:Agid;given-names:YA\", \"pub-id_doi\": \"10.1038/334345a0\", \"pub-id_pmid\": \"2899295\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"334\", \"year\": \"1988\"}, \"text\": \"Melanized dopaminergic neurons are differentially susceptible to degeneration in Parkinson\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61720, \"infons\": {\"fpage\": \"e70921\", \"name_0\": \"surname:Hobson;given-names:BD\", \"name_1\": \"surname:Choi;given-names:SJ\", \"name_2\": \"surname:Mosharov;given-names:EV\", \"name_3\": \"surname:Soni;given-names:RK\", \"name_4\": \"surname:Sulzer;given-names:D\", \"name_5\": \"surname:Sims;given-names:P\", \"pub-id_doi\": \"10.7554/eLife.70921\", \"pub-id_pmid\": \"35098924\", \"section_type\": \"REF\", \"source\": \"Elife\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2022a\"}, \"text\": \"Subcellular proteomics of dopamine neurons in the mouse brain\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61782, \"infons\": {\"name_0\": \"surname:Hobson;given-names:BD\", \"name_1\": \"surname:Sulzer;given-names:D\", \"pub-id_doi\": \"10.3233/JPD-223153\", \"section_type\": \"REF\", \"source\": \"J Parkinsons Dis\", \"type\": \"ref\", \"year\": \"2022\"}, \"text\": \"Neuronal Presentation of Antigen and Its Possible Role in Parkinson\\u2019s Disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61862, \"infons\": {\"fpage\": \"110208\", \"issue\": \"2\", \"name_0\": \"surname:Hobson;given-names:BD\", \"name_1\": \"surname:Kong;given-names:L\", \"name_2\": \"surname:Angelo;given-names:MF\", \"name_3\": \"surname:Lieberman;given-names:OJ\", \"name_4\": \"surname:Mosharov;given-names:EV\", \"name_5\": \"surname:Herzog;given-names:E\", \"name_6\": \"surname:Sulzer;given-names:D\", \"name_7\": \"surname:Sims;given-names:PA\", \"pub-id_pmid\": \"35021090\", \"section_type\": \"REF\", \"source\": \"Cell Rep\", \"type\": \"ref\", \"volume\": \"38\", \"year\": \"2022b\"}, \"text\": \"Subcellular and regional localization of mRNA translation in midbrain dopamine neurons\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61949, \"infons\": {\"fpage\": \"43\", \"lpage\": \"50\", \"name_0\": \"surname:H\\u00f6ftberger;given-names:R\", \"name_1\": \"surname:Aboul-Enein;given-names:F\", \"name_2\": \"surname:Brueck;given-names:W\", \"name_3\": \"surname:Lucchinetti;given-names:C\", \"name_4\": \"surname:Rodriguez;given-names:M\", \"name_5\": \"surname:Schmidbauer;given-names:M\", \"name_6\": \"surname:Jellinger;given-names:K\", \"name_7\": \"surname:Lassmann;given-names:H\", \"pub-id_doi\": \"10.1111/j.1750-3639.2004.tb00496.x\", \"pub-id_pmid\": \"14997936\", \"section_type\": \"REF\", \"source\": \"Brain Pathol\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2004\"}, \"text\": \"Expression of major histocompatibility complex class I molecules on the different cell types in multiple sclerosis lesions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62072, \"infons\": {\"fpage\": \"1\", \"lpage\": \"30\", \"name_0\": \"surname:Howcroft;given-names:TK\", \"name_1\": \"surname:Singer;given-names:DS\", \"pub-id_doi\": \"10.1385/IR:27:1:1\", \"pub-id_pmid\": \"12637766\", \"section_type\": \"REF\", \"source\": \"Immunol. Res\", \"type\": \"ref\", \"volume\": \"27\", \"year\": \"2003\"}, \"text\": \"Expression of nonclassical MHC class Ib genes: comparison of regulatory elements\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62153, \"infons\": {\"fpage\": \"238\", \"name_0\": \"surname:Howe;given-names:CL\", \"name_1\": \"surname:LaFrance-Corey;given-names:RG\", \"name_2\": \"surname:Goddery;given-names:EN\", \"name_3\": \"surname:Johnson;given-names:RK\", \"name_4\": \"surname:Mirchia;given-names:K\", \"pub-id_doi\": \"10.1186/s12974-017-1015-2\", \"pub-id_pmid\": \"29202854\", \"section_type\": \"REF\", \"source\": \"J. Neuroinflammation\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2017\"}, \"text\": \"Neuronal CCL2 expression drives inflammatory monocyte infiltration into the brain during acute virus infection\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62264, \"infons\": {\"fpage\": \"258\", \"lpage\": \"267\", \"name_0\": \"surname:Isaacs;given-names:A\", \"name_1\": \"surname:Lindenmann;given-names:J\", \"name_2\": \"surname:Andrewes;given-names:CH\", \"pub-id_doi\": \"10.1098/rspb.1957.0048\", \"pub-id_pmid\": \"13465720\", \"section_type\": \"REF\", \"source\": \"Proceedings of the Royal Society of London. Series B - Biological Sciences\", \"type\": \"ref\", \"volume\": \"147\", \"year\": \"1957\"}, \"text\": \"Virus interference. I. The interferon\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62302, \"infons\": {\"fpage\": \"545\", \"lpage\": \"558\", \"name_0\": \"surname:Ivashkiv;given-names:LB\", \"pub-id_doi\": \"10.1038/S41577-018-0029-z\", \"pub-id_pmid\": \"29921905\", \"section_type\": \"REF\", \"source\": \"Nat. Rev. Immunol\", \"type\": \"ref\", \"volume\": \"18\", \"year\": \"2018\"}, \"text\": \"IFN\\u03b3: signalling, epigenetics and roles in immunity, metabolism, disease and cancer immunotherapy\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62404, \"infons\": {\"fpage\": \"12849\", \"name_0\": \"surname:Iwata;given-names:H\", \"name_1\": \"surname:Goettsch;given-names:C\", \"name_10\": \"surname:Inoue;given-names:N\", \"name_11\": \"surname:Fukuda;given-names:D\", \"name_12\": \"surname:Mojcher;given-names:A\", \"name_13\": \"surname:Mattson;given-names:PC\", \"name_14\": \"surname:Barab\\u00e1si;given-names:A-L\", \"name_15\": \"surname:Boothby;given-names:M\", \"name_16\": \"surname:Aikawa;given-names:E\", \"name_17\": \"surname:Singh;given-names:SA\", \"name_18\": \"surname:Aikawa;given-names:M\", \"name_2\": \"surname:Sharma;given-names:A\", \"name_3\": \"surname:Ricchiuto;given-names:P\", \"name_4\": \"surname:Goh;given-names:WWB\", \"name_5\": \"surname:Halu;given-names:A\", \"name_6\": \"surname:Yamada;given-names:I\", \"name_7\": \"surname:Yoshida;given-names:H\", \"name_8\": \"surname:Hara;given-names:T\", \"name_9\": \"surname:Wei;given-names:M\", \"pub-id_doi\": \"10.1038/ncomms12849\", \"pub-id_pmid\": \"27796300\", \"section_type\": \"REF\", \"source\": \"Nat. Commun\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2016\"}, \"text\": \"PARP9 and PARP14 cross-regulate macrophage activation via STAT1 ADP-ribosylation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62485, \"infons\": {\"fpage\": \"543\", \"issue\": \"7745\", \"lpage\": \"547\", \"name_0\": \"surname:J\\u00e4kel;given-names:S\", \"name_1\": \"surname:Agirre;given-names:E\", \"name_2\": \"surname:Mendanha Falc\\u00e3o;given-names:A\", \"name_3\": \"surname:van Bruggen;given-names:D\", \"name_4\": \"surname:Lee;given-names:KW\", \"name_5\": \"surname:Knuesel;given-names:I\", \"name_6\": \"surname:Malhotra;given-names:D\", \"name_7\": \"surname:ffrench-Constant;given-names:C\", \"name_8\": \"surname:Williams;given-names:A\", \"name_9\": \"surname:Castelo-Branco;given-names:G\", \"pub-id_pmid\": \"30747918\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"566\", \"year\": \"2019\"}, \"text\": \"Altered human oligodendrocyte heterogeneity in multiple sclerosis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62551, \"infons\": {\"fpage\": \"16\", \"lpage\": \"21\", \"name_0\": \"surname:Jongsma;given-names:MLM\", \"name_1\": \"surname:Guarda;given-names:G\", \"name_2\": \"surname:Spaapen;given-names:RM\", \"pub-id_doi\": \"10.1016/j.molimm.2017.12.005\", \"pub-id_pmid\": \"29224918\", \"section_type\": \"REF\", \"source\": \"Mol. Immunol\", \"type\": \"ref\", \"volume\": \"113\", \"year\": \"2019\"}, \"text\": \"The regulatory network behind MHC class I expression\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62604, \"infons\": {\"fpage\": \"1446\", \"issue\": \"11\", \"lpage\": \"1457\", \"name_0\": \"surname:Kaya;given-names:T\", \"name_1\": \"surname:Mattugini;given-names:N\", \"name_10\": \"surname:Kaji;given-names:S\", \"name_11\": \"surname:Liesz;given-names:A\", \"name_12\": \"surname:Gokce;given-names:O\", \"name_13\": \"surname:Simons;given-names:M\", \"name_2\": \"surname:Liu;given-names:L.u.\", \"name_3\": \"surname:Ji;given-names:H\", \"name_4\": \"surname:Cantuti-Castelvetri;given-names:L\", \"name_5\": \"surname:Wu;given-names:J\", \"name_6\": \"surname:Schifferer;given-names:M\", \"name_7\": \"surname:Groh;given-names:J\", \"name_8\": \"surname:Martini;given-names:R\", \"name_9\": \"surname:Besson-Girard;given-names:S\", \"pub-id_pmid\": \"36280798\", \"section_type\": \"REF\", \"source\": \"Nat. Neurosci\", \"type\": \"ref\", \"volume\": \"25\", \"year\": \"2022\"}, \"text\": \"CD8+ T cells induce interferon-responsive oligodendrocytes and microglia in white matter aging\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62699, \"infons\": {\"fpage\": \"1619\", \"lpage\": \"1636\", \"name_0\": \"surname:Klaus;given-names:C\", \"name_1\": \"surname:Liao;given-names:H\", \"name_2\": \"surname:Allendorf;given-names:DH\", \"name_3\": \"surname:Brown;given-names:GC\", \"name_4\": \"surname:Neumann;given-names:H\", \"pub-id_doi\": \"10.1002/glia.23945\", \"pub-id_pmid\": \"33340149\", \"section_type\": \"REF\", \"source\": \"Glia\", \"type\": \"ref\", \"volume\": \"69\", \"year\": \"2021\"}, \"text\": \"Sialylation acts as a checkpoint for innate immune responses in the central nervous system\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62790, \"infons\": {\"fpage\": \"2087\", \"lpage\": \"2103\", \"name_0\": \"surname:Kreutzfeldt;given-names:M\", \"name_1\": \"surname:Bergthaler;given-names:A\", \"name_10\": \"surname:Forman;given-names:R\", \"name_11\": \"surname:M\\u00fcller;given-names:W\", \"name_12\": \"surname:Becher;given-names:B\", \"name_13\": \"surname:Misgeld;given-names:T\", \"name_14\": \"surname:Kerschensteiner;given-names:M\", \"name_15\": \"surname:Pinschewer;given-names:DD\", \"name_16\": \"surname:Merkler;given-names:D\", \"name_2\": \"surname:Fernandez;given-names:M\", \"name_3\": \"surname:Br\\u00fcck;given-names:W\", \"name_4\": \"surname:Steinbach;given-names:K\", \"name_5\": \"surname:Vorm;given-names:M\", \"name_6\": \"surname:Coras;given-names:R\", \"name_7\": \"surname:Bl\\u00fcmcke;given-names:I\", \"name_8\": \"surname:Bonilla;given-names:WV\", \"name_9\": \"surname:Fleige;given-names:A\", \"pub-id_doi\": \"10.1084/jem.20122143\", \"pub-id_pmid\": \"23999498\", \"section_type\": \"REF\", \"source\": \"J. Exp. Med\", \"type\": \"ref\", \"volume\": \"210\", \"year\": \"2013\"}, \"text\": \"Neuroprotective intervention by interferon-\\u03b3 blockade prevents CD8+ T cell\\u2013mediated dendrite and synapse loss\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62904, \"infons\": {\"fpage\": \"499\", \"lpage\": \"524\", \"name_0\": \"surname:Krishnaswami;given-names:SR\", \"name_1\": \"surname:Grindberg;given-names:RV\", \"name_10\": \"surname:Hodge;given-names:R\", \"name_11\": \"surname:McCarthy;given-names:JK\", \"name_12\": \"surname:Kelder;given-names:M\", \"name_13\": \"surname:McCorrison;given-names:J\", \"name_14\": \"surname:Aevermann;given-names:BD\", \"name_15\": \"surname:Fuertes;given-names:FD\", \"name_16\": \"surname:Scheuermann;given-names:RH\", \"name_17\": \"surname:Lee;given-names:J\", \"name_18\": \"surname:Lein;given-names:ES\", \"name_19\": \"surname:Schork;given-names:N\", \"name_2\": \"surname:Novotny;given-names:M\", \"name_20\": \"surname:McConnell;given-names:MJ\", \"name_21\": \"surname:Gage;given-names:FH\", \"name_22\": \"surname:Lasken;given-names:RS\", \"name_3\": \"surname:Venepally;given-names:P\", \"name_4\": \"surname:Lacar;given-names:B\", \"name_5\": \"surname:Bhutani;given-names:K\", \"name_6\": \"surname:Linker;given-names:SB\", \"name_7\": \"surname:Pham;given-names:S\", \"name_8\": \"surname:Erwin;given-names:JA\", \"name_9\": \"surname:Miller;given-names:JA\", \"pub-id_doi\": \"10.1038/nprot.2016.015\", \"pub-id_pmid\": \"26890679\", \"section_type\": \"REF\", \"source\": \"Nat. Protoc\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2016\"}, \"text\": \"Using single nuclei for RNA-seq to capture the transcriptome of postmortem neurons\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62987, \"infons\": {\"fpage\": \"205\", \"name_0\": \"surname:Kustrimovic;given-names:N\", \"name_1\": \"surname:Comi;given-names:C\", \"name_10\": \"surname:Sturchio;given-names:A\", \"name_11\": \"surname:Mauri;given-names:M\", \"name_12\": \"surname:Bono;given-names:G\", \"name_13\": \"surname:Marino;given-names:F\", \"name_14\": \"surname:Cosentino;given-names:M\", \"name_2\": \"surname:Magistrelli;given-names:L\", \"name_3\": \"surname:Rasini;given-names:E\", \"name_4\": \"surname:Legnaro;given-names:M\", \"name_5\": \"surname:Bombelli;given-names:R\", \"name_6\": \"surname:Aleksic;given-names:I\", \"name_7\": \"surname:Blandini;given-names:F\", \"name_8\": \"surname:Minafra;given-names:B\", \"name_9\": \"surname:Riboldazzi;given-names:G\", \"pub-id_doi\": \"10.1186/s12974-018-1248-8\", \"section_type\": \"REF\", \"source\": \"J. Neuroinflamm\", \"type\": \"ref\", \"volume\": \"15\", \"year\": \"2018\"}, \"text\": \"Parkinson\\u2019s disease patients have a complex phenotypic and functional Th1 bias: cross-sectional studies of CD4+ Th1/Th2/T17 and Treg in drug-na\\u00efve and drug-treated patients\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63163, \"infons\": {\"fpage\": \"4223\", \"lpage\": \"4234\", \"name_0\": \"surname:Lee;given-names:SH\", \"name_1\": \"surname:Carrero;given-names:JA\", \"name_2\": \"surname:Uppaluri;given-names:R\", \"name_3\": \"surname:White;given-names:JM\", \"name_4\": \"surname:Archambault;given-names:JM\", \"name_5\": \"surname:Lai;given-names:KS\", \"name_6\": \"surname:Chan;given-names:SR\", \"name_7\": \"surname:Sheehan;given-names:KCF\", \"name_8\": \"surname:Unanue;given-names:ER\", \"name_9\": \"surname:Schreiber;given-names:RD\", \"pub-id_doi\": \"10.4049/jimmunol.1300910\", \"pub-id_pmid\": \"24048899\", \"section_type\": \"REF\", \"source\": \"J. Immunol\", \"type\": \"ref\", \"volume\": \"191\", \"year\": \"2013\"}, \"text\": \"Identifying the initiating events of anti-Listeria responses using mice with conditional loss of IFN-\\u03b3 receptor subunit 1 (IFNGR1)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63298, \"infons\": {\"fpage\": \"184\", \"lpage\": \"197\", \"name_0\": \"surname:Levine;given-names:JH\", \"name_1\": \"surname:Simonds;given-names:EF\", \"name_10\": \"surname:Finck;given-names:R\", \"name_11\": \"surname:Gedman;given-names:AL\", \"name_12\": \"surname:Radtke;given-names:I\", \"name_13\": \"surname:Downing;given-names:JR\", \"name_14\": \"surname:Pe\\u2019er;given-names:D\", \"name_15\": \"surname:Nolan;given-names:GP\", \"name_2\": \"surname:Bendall;given-names:SC\", \"name_3\": \"surname:Davis;given-names:KL\", \"name_4\": \"surname:Amir;given-names:ED\", \"name_5\": \"surname:Tadmor;given-names:MD\", \"name_6\": \"surname:Litvin;given-names:O\", \"name_7\": \"surname:Fienberg;given-names:HG\", \"name_8\": \"surname:Jager;given-names:A\", \"name_9\": \"surname:Zunder;given-names:ER\", \"pub-id_doi\": \"10.1016/j.cell.2015.05.047\", \"pub-id_pmid\": \"26095251\", \"section_type\": \"REF\", \"source\": \"Cell\", \"type\": \"ref\", \"volume\": \"162\", \"year\": \"2015\"}, \"text\": \"Data-Driven Phenotypic Dissection of AML Reveals Progenitor-like Cells that Correlate with Prognosis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63399, \"infons\": {\"fpage\": \"e8557\", \"name_0\": \"surname:Levitin;given-names:HM\", \"name_1\": \"surname:Yuan;given-names:J\", \"name_10\": \"surname:Sims;given-names:PA\", \"name_2\": \"surname:Cheng;given-names:YL\", \"name_3\": \"surname:Ruiz;given-names:FJ\", \"name_4\": \"surname:Bush;given-names:EC\", \"name_5\": \"surname:Bruce;given-names:JN\", \"name_6\": \"surname:Canoll;given-names:P\", \"name_7\": \"surname:Iavarone;given-names:A\", \"name_8\": \"surname:Lasorella;given-names:A\", \"name_9\": \"surname:Blei;given-names:DM\", \"pub-id_pmid\": \"30796088\", \"section_type\": \"REF\", \"source\": \"Mol. Syst. Biol\", \"type\": \"ref\", \"volume\": \"15\", \"year\": \"2019\"}, \"text\": \"De novo gene signature identification from single-cell RNA-seq with hierarchical Poisson factorization\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63502, \"infons\": {\"fpage\": \"1875\", \"name_0\": \"surname:Lindestam Arlehamn;given-names:CS\", \"name_1\": \"surname:Dhanwani;given-names:R\", \"name_10\": \"surname:Amara;given-names:AW\", \"name_11\": \"surname:Standaert;given-names:DG\", \"name_12\": \"surname:Goldman;given-names:JG\", \"name_13\": \"surname:Litvan;given-names:I\", \"name_14\": \"surname:Peters;given-names:B\", \"name_15\": \"surname:Sulzer;given-names:D\", \"name_16\": \"surname:Sette;given-names:A\", \"name_2\": \"surname:Pham;given-names:J\", \"name_3\": \"surname:Kuan;given-names:R\", \"name_4\": \"surname:Frazier;given-names:A\", \"name_5\": \"surname:Rezende Dutra;given-names:J\", \"name_6\": \"surname:Phillips;given-names:E\", \"name_7\": \"surname:Mallal;given-names:S\", \"name_8\": \"surname:Roederer;given-names:M\", \"name_9\": \"surname:Marder;given-names:KS\", \"pub-id_doi\": \"10.1038/s41467-020-15626-w\", \"pub-id_pmid\": \"32313102\", \"section_type\": \"REF\", \"source\": \"Nat. Commun\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2020\"}, \"text\": \"\\u03b1-Synuclein-specific T cell reactivity is associated with preclinical and early Parkinson\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63606, \"infons\": {\"fpage\": \"1191\", \"lpage\": \"1195\", \"name_0\": \"surname:Littman;given-names:SJ\", \"name_1\": \"surname:Faltynek;given-names:CR\", \"name_2\": \"surname:Baglioni;given-names:C\", \"pub-id_doi\": \"10.1016/S0021-9258(20)71226-3\", \"pub-id_pmid\": \"3155725\", \"section_type\": \"REF\", \"source\": \"J. Biol. Chem\", \"type\": \"ref\", \"volume\": \"260\", \"year\": \"1985\"}, \"text\": \"Binding of human recombinant 125I-interferon gamma to receptors on human cells\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63685, \"infons\": {\"fpage\": \"503\", \"lpage\": \"508\", \"name_0\": \"surname:Lodygin;given-names:D\", \"name_1\": \"surname:Hermann;given-names:M\", \"name_10\": \"surname:Reichardt;given-names:HM\", \"name_11\": \"surname:Zagrebelsky;given-names:M\", \"name_12\": \"surname:Mollenhauer;given-names:B\", \"name_13\": \"surname:K\\u00fcgler;given-names:S\", \"name_14\": \"surname:Fitzner;given-names:D\", \"name_15\": \"surname:Frahm;given-names:J\", \"name_16\": \"surname:Stadelmann;given-names:C\", \"name_17\": \"surname:Haberl;given-names:M\", \"name_18\": \"surname:Odoardi;given-names:F\", \"name_19\": \"surname:Fl\\u00fcgel;given-names:A\", \"name_2\": \"surname:Schweingruber;given-names:N\", \"name_3\": \"surname:Fl\\u00fcgel-Koch;given-names:C\", \"name_4\": \"surname:Watanabe;given-names:T\", \"name_5\": \"surname:Schlosser;given-names:C\", \"name_6\": \"surname:Merlini;given-names:A\", \"name_7\": \"surname:K\\u00f6rner;given-names:H\", \"name_8\": \"surname:Chang;given-names:H-F\", \"name_9\": \"surname:Fischer;given-names:HJ\", \"pub-id_doi\": \"10.1038/S41586-019-0964-2\", \"pub-id_pmid\": \"30787438\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"566\", \"year\": \"2019\"}, \"text\": \"\\u03b2-synuclein-reactive T cells induce autoimmune CNS grey matter degeneration\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63764, \"infons\": {\"fpage\": \"550\", \"name_0\": \"surname:Love;given-names:MI\", \"name_1\": \"surname:Huber;given-names:W\", \"name_2\": \"surname:Anders;given-names:S\", \"pub-id_doi\": \"10.1186/s13059-014-0550-8\", \"pub-id_pmid\": \"25516281\", \"section_type\": \"REF\", \"source\": \"Genome Biol\", \"type\": \"ref\", \"volume\": \"15\", \"year\": \"2014\"}, \"text\": \"Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63844, \"infons\": {\"fpage\": \"75\", \"name_0\": \"surname:Lun;given-names:ATL\", \"name_1\": \"surname:Bach;given-names:K\", \"name_2\": \"surname:Marioni;given-names:JC\", \"pub-id_doi\": \"10.1186/s13059-016-0947-7\", \"pub-id_pmid\": \"27122128\", \"section_type\": \"REF\", \"source\": \"Genome Biol\", \"type\": \"ref\", \"volume\": \"17\", \"year\": \"2016\"}, \"text\": \"Pooling across cells to normalize single-cell RNA sequencing data with many zero counts\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63932, \"infons\": {\"fpage\": \"444\", \"lpage\": \"453\", \"name_0\": \"surname:Matsuda;given-names:W\", \"name_1\": \"surname:Furuta;given-names:T\", \"name_2\": \"surname:Nakamura;given-names:KC\", \"name_3\": \"surname:Hioki;given-names:H\", \"name_4\": \"surname:Fujiyama;given-names:F\", \"name_5\": \"surname:Arai;given-names:R\", \"name_6\": \"surname:Kaneko;given-names:T\", \"pub-id_doi\": \"10.1523/JNEUROSCI.4029-08.2009\", \"pub-id_pmid\": \"19144844\", \"section_type\": \"REF\", \"source\": \"J. Neurosci\", \"type\": \"ref\", \"volume\": \"29\", \"year\": \"2009\"}, \"text\": \"Single nigrostriatal dopaminergic neurons form widely spread and highly dense axonal arborizations in the neostriatum\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64050, \"infons\": {\"fpage\": \"1823\", \"lpage\": \"1833\", \"name_0\": \"surname:McDole;given-names:JR\", \"name_1\": \"surname:Danzer;given-names:SC\", \"name_2\": \"surname:Pun;given-names:RYK\", \"name_3\": \"surname:Chen;given-names:Y\", \"name_4\": \"surname:Johnson;given-names:HL\", \"name_5\": \"surname:Pirko;given-names:I\", \"name_6\": \"surname:Johnson;given-names:AJ\", \"pub-id_doi\": \"10.2353/ajpath.2010.100231\", \"pub-id_pmid\": \"20813972\", \"section_type\": \"REF\", \"source\": \"Am. J. Pathol\", \"type\": \"ref\", \"volume\": \"177\", \"year\": \"2010\"}, \"text\": \"Rapid formation of extended processes and engagement of Theiler\\u2019s virus-infected neurons by CNS-infiltrating CD8 T cells\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64173, \"infons\": {\"fpage\": \"1285\", \"lpage\": \"1291\", \"name_0\": \"surname:McGeer;given-names:PL\", \"name_1\": \"surname:Itagaki;given-names:S\", \"name_2\": \"surname:Boyes;given-names:BE\", \"name_3\": \"surname:McGeer;given-names:EG\", \"pub-id_doi\": \"10.1212/wnl.38.8.1285\", \"pub-id_pmid\": \"3399080\", \"section_type\": \"REF\", \"source\": \"Neurology\", \"type\": \"ref\", \"volume\": \"38\", \"year\": \"1988\"}, \"text\": \"Reactive microglia are positive for HLA-DR in the substantia nigra of Parkinson\\u2019s and Alzheimer\\u2019s disease brains\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64290, \"infons\": {\"fpage\": \"3174\", \"lpage\": \"3183\", \"name_0\": \"surname:Min;given-names:W\", \"name_1\": \"surname:Pober;given-names:JS\", \"name_2\": \"surname:Johnson;given-names:DR\", \"pub-id_pmid\": \"8617938\", \"section_type\": \"REF\", \"source\": \"J Immunol\", \"type\": \"ref\", \"volume\": \"156\", \"year\": \"1996\"}, \"text\": \"Kinetically coordinated induction of TAP1 and HLA class I by IFN-gamma: the rapid induction of TAP1 by IFN-gamma is mediated by Stat1 alpha\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64430, \"infons\": {\"fpage\": \"19\", \"name_0\": \"surname:Mo;given-names:M-S\", \"name_1\": \"surname:Li;given-names:G-H\", \"name_10\": \"surname:Chen;given-names:C-J\", \"name_11\": \"surname:Qu;given-names:S-G\", \"name_12\": \"surname:He;given-names:J-X\", \"name_13\": \"surname:Xu;given-names:P-Y\", \"name_2\": \"surname:Sun;given-names:C-C\", \"name_3\": \"surname:Huang;given-names:S-X\", \"name_4\": \"surname:Wei;given-names:L\", \"name_5\": \"surname:Zhang;given-names:L-M\", \"name_6\": \"surname:Zhou;given-names:M-M\", \"name_7\": \"surname:Wu;given-names:Z-H\", \"name_8\": \"surname:Guo;given-names:W-Y\", \"name_9\": \"surname:Yang;given-names:X-L\", \"pub-id_doi\": \"10.1186/s40035-018-0125-9\", \"section_type\": \"REF\", \"source\": \"Transl. Neurodegen\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2018\"}, \"text\": \"Dopaminergic neurons show increased low-molecular-mass protein 7 activity induced by 6-hydroxydopamine in vitro and in vivo\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64554, \"infons\": {\"fpage\": \"94\", \"lpage\": \"97\", \"name_0\": \"surname:Mogi;given-names:M\", \"name_1\": \"surname:Kondo;given-names:T\", \"name_2\": \"surname:Mizuno;given-names:Y\", \"name_3\": \"surname:Nagatsu;given-names:T\", \"pub-id_doi\": \"10.1016/j.neulet.2006.12.003\", \"pub-id_pmid\": \"17196747\", \"section_type\": \"REF\", \"source\": \"Neurosci. Lett\", \"type\": \"ref\", \"volume\": \"414\", \"year\": \"2007\"}, \"text\": \"p53 protein, interferon-gamma, and NF-kappaB levels are elevated in the parkinsonian brain\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64645, \"infons\": {\"fpage\": \"14470\", \"lpage\": \"14475\", \"name_0\": \"surname:Morales;given-names:J\", \"name_1\": \"surname:Homey;given-names:B\", \"name_10\": \"surname:Zlotnik;given-names:A\", \"name_2\": \"surname:Vicari;given-names:AP\", \"name_3\": \"surname:Hudak;given-names:S\", \"name_4\": \"surname:Oldham;given-names:E\", \"name_5\": \"surname:Hedrick;given-names:J\", \"name_6\": \"surname:Orozco;given-names:R\", \"name_7\": \"surname:Copeland;given-names:NG\", \"name_8\": \"surname:Jenkins;given-names:NA\", \"name_9\": \"surname:McEvoy;given-names:LM\", \"pub-id_doi\": \"10.1073/pnas.96.25.14470\", \"pub-id_pmid\": \"10588729\", \"section_type\": \"REF\", \"source\": \"PNAS\", \"type\": \"ref\", \"volume\": \"96\", \"year\": \"1999\"}, \"text\": \"CTACK, a skin-associated chemokine that preferentially attracts skin-homing memory T cells\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64736, \"infons\": {\"fpage\": \"3328\", \"lpage\": \"3337\", \"name_0\": \"surname:Mount;given-names:MP\", \"name_1\": \"surname:Lira;given-names:A\", \"name_2\": \"surname:Grimes;given-names:D\", \"name_3\": \"surname:Smith;given-names:PD\", \"name_4\": \"surname:Faucher;given-names:S\", \"name_5\": \"surname:Slack;given-names:R\", \"name_6\": \"surname:Anisman;given-names:H\", \"name_7\": \"surname:Hayley;given-names:S\", \"name_8\": \"surname:Park;given-names:DS\", \"pub-id_doi\": \"10.1523/JNEUROSCI.5321-06.2007\", \"pub-id_pmid\": \"17376993\", \"section_type\": \"REF\", \"source\": \"J. Neurosci\", \"type\": \"ref\", \"volume\": \"27\", \"year\": \"2007\"}, \"text\": \"Involvement of interferon-gamma in microglial-mediated loss of dopaminergic neurons\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64820, \"infons\": {\"fpage\": \"1\", \"lpage\": \"18\", \"name_0\": \"surname:M\\u00fcllner;given-names:D\", \"pub-id_doi\": \"10.18637/jss.v053.i09\", \"section_type\": \"REF\", \"source\": \"J. Stat. Softw\", \"type\": \"ref\", \"volume\": \"53\", \"year\": \"2013\"}, \"text\": \"fastcluster: Fast hierarchical, agglomerative clustering routines for R and python\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64903, \"infons\": {\"fpage\": \"eaap7388\", \"name_0\": \"surname:Murakami;given-names:G\", \"name_1\": \"surname:Edamura;given-names:M\", \"name_2\": \"surname:Furukawa;given-names:T\", \"name_3\": \"surname:Kawasaki;given-names:H\", \"name_4\": \"surname:Kosugi;given-names:I\", \"name_5\": \"surname:Fukuda;given-names:A\", \"name_6\": \"surname:Iwashita;given-names:T\", \"name_7\": \"surname:Nakahara;given-names:D\", \"pub-id_doi\": \"10.1126/sciadv.aap7388\", \"pub-id_pmid\": \"29546241\", \"section_type\": \"REF\", \"source\": \"Sci. Adv\", \"type\": \"ref\", \"volume\": \"4\", \"year\": \"2018\"}, \"text\": \"MHC class I in dopaminergic neurons suppresses relapse to reward seeking\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64976, \"infons\": {\"fpage\": \"e0213069\", \"issue\": \"4\", \"name_0\": \"surname:Naik;given-names:J\", \"name_1\": \"surname:Hau;given-names:CM\", \"name_10\": \"surname:Nieuwland;given-names:R\", \"name_11\": \"surname:Paulusma;given-names:CC\", \"name_12\": \"surname:Bosma;given-names:PJ\", \"name_13\": \"surname:Hakami;given-names:RM\", \"name_2\": \"surname:ten Bloemendaal;given-names:L\", \"name_3\": \"surname:Mok;given-names:KS\", \"name_4\": \"surname:Hajji;given-names:N\", \"name_5\": \"surname:Wehman;given-names:AM\", \"name_6\": \"surname:Meisner;given-names:S\", \"name_7\": \"surname:Muncan;given-names:V\", \"name_8\": \"surname:Paauw;given-names:NJ\", \"name_9\": \"surname:de Vries;given-names:HE\", \"pub-id_pmid\": \"30947313\", \"section_type\": \"REF\", \"source\": \"PLoS One\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2019\"}, \"text\": \"The P4-ATPase ATP9A is a novel determinant of exosome release\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65038, \"infons\": {\"fpage\": \"2781\", \"lpage\": \"2787\", \"name_0\": \"surname:Namiki;given-names:S\", \"name_1\": \"surname:Nakamura;given-names:T\", \"name_2\": \"surname:Oshima;given-names:S\", \"name_3\": \"surname:Yamazaki;given-names:M\", \"name_4\": \"surname:Sekine;given-names:Y\", \"name_5\": \"surname:Tsuchiya;given-names:K\", \"name_6\": \"surname:Okamoto;given-names:R\", \"name_7\": \"surname:Kanai;given-names:T\", \"name_8\": \"surname:Watanabe;given-names:M\", \"pub-id_doi\": \"10.1016/j.febslet.2005.04.012\", \"pub-id_pmid\": \"15907481\", \"section_type\": \"REF\", \"source\": \"FEBS Lett\", \"type\": \"ref\", \"volume\": \"579\", \"year\": \"2005\"}, \"text\": \"IRF-1 mediates upregulation of LMP7 by IFN-gamma and concerted expression of immunosubunits of the proteasome\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65148, \"infons\": {\"fpage\": \"305\", \"lpage\": \"316\", \"name_0\": \"surname:Neumann;given-names:H\", \"name_1\": \"surname:Schmidt;given-names:H\", \"name_2\": \"surname:Cavali\\u00e9;given-names:A\", \"name_3\": \"surname:Jenne;given-names:D\", \"name_4\": \"surname:Wekerle;given-names:H\", \"pub-id_doi\": \"10.1084/jem.185.2.305\", \"pub-id_pmid\": \"9016879\", \"section_type\": \"REF\", \"source\": \"J. Exp. Med\", \"type\": \"ref\", \"volume\": \"185\", \"year\": \"1997\"}, \"text\": \"Major histocompatibility complex (MHC) class I gene expression in single neurons of the central nervous system: differential regulation by interferon (IFN)-gamma and tumor necrosis factor (TNF)-alpha\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65348, \"infons\": {\"fpage\": \"19\", \"lpage\": \"35\", \"name_0\": \"surname:Popko;given-names:B\", \"name_1\": \"surname:Corbin;given-names:JG\", \"name_2\": \"surname:Baerwald;given-names:KD\", \"name_3\": \"surname:Dupree;given-names:J\", \"name_4\": \"surname:Garcia;given-names:AM\", \"pub-id_doi\": \"10.1007/BF02740619\", \"pub-id_pmid\": \"9170099\", \"section_type\": \"REF\", \"source\": \"Mol. Neurobiol\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"1997\"}, \"text\": \"The effects of interferon-gamma on the central nervous system\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65410, \"infons\": {\"fpage\": \"930\", \"lpage\": \"943\", \"name_0\": \"surname:Poulin;given-names:J-F\", \"name_1\": \"surname:Zou;given-names:J\", \"name_2\": \"surname:Drouin-Ouellet;given-names:J\", \"name_3\": \"surname:Kim;given-names:K-Y-A\", \"name_4\": \"surname:Cicchetti;given-names:F\", \"name_5\": \"surname:Awatramani;given-names:RB\", \"pub-id_doi\": \"10.1016/j.celrep.2014.10.008\", \"pub-id_pmid\": \"25437550\", \"section_type\": \"REF\", \"source\": \"Cell Rep\", \"type\": \"ref\", \"volume\": \"9\", \"year\": \"2014\"}, \"text\": \"Defining midbrain dopaminergic neuron diversity by single-cell gene expression profiling\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65499, \"infons\": {\"fpage\": \"115\", \"lpage\": \"134\", \"name_0\": \"surname:Rall;given-names:GF\", \"name_1\": \"surname:Whitton;given-names:JL\", \"pub-id_doi\": \"10.1007/978-3-642-72045-1_6\", \"section_type\": \"REF\", \"source\": \"Antigen Presentation, Current Topics in Microbiology and Immunology\", \"type\": \"ref\", \"year\": \"1998\"}, \"text\": \"\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65500, \"infons\": {\"fpage\": \"431\", \"lpage\": \"439\", \"name_0\": \"surname:Rico-Bautista;given-names:E\", \"name_1\": \"surname:Flores-Morales;given-names:A\", \"name_2\": \"surname:Fern\\u00e1ndez-P\\u00e9rez;given-names:L\", \"pub-id_doi\": \"10.1016/j.cytogfr.2006.09.008\", \"pub-id_pmid\": \"17070092\", \"section_type\": \"REF\", \"source\": \"Cytokine Growth Factor Rev\", \"type\": \"ref\", \"volume\": \"17\", \"year\": \"2006\"}, \"text\": \"Suppressor of cytokine signaling (SOCS) 2, a protein with multiple functions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65577, \"infons\": {\"fpage\": \"44\", \"lpage\": \"53\", \"name_0\": \"surname:Robertson;given-names:IB\", \"name_1\": \"surname:Horiguchi;given-names:M\", \"name_2\": \"surname:Zilberberg;given-names:L\", \"name_3\": \"surname:Dabovic;given-names:B\", \"name_4\": \"surname:Hadjiolova;given-names:K\", \"name_5\": \"surname:Rifkin;given-names:DB\", \"pub-id_doi\": \"10.1016/j.matbio.2015.05.005\", \"pub-id_pmid\": \"25960419\", \"section_type\": \"REF\", \"source\": \"Matrix Biol\", \"type\": \"ref\", \"volume\": \"47\", \"year\": \"2015\"}, \"text\": \"Latent TGF-\\u03b2-binding proteins\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65610, \"infons\": {\"fpage\": \"145\", \"lpage\": \"156\", \"name_0\": \"surname:Rose;given-names:RW\", \"name_1\": \"surname:Vorobyeva;given-names:AG\", \"name_2\": \"surname:Skipworth;given-names:JD\", \"name_3\": \"surname:Nicolas;given-names:E\", \"name_4\": \"surname:Rall;given-names:GF\", \"pub-id_doi\": \"10.1016/j.jneuroim.2007.10.007\", \"pub-id_pmid\": \"18006082\", \"section_type\": \"REF\", \"source\": \"J. Neuroimmunol\", \"type\": \"ref\", \"volume\": \"192\", \"year\": \"2007\"}, \"text\": \"Altered Levels of STAT1 and STAT3 influence the neuronal response to interferon Gamma\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65696, \"infons\": {\"fpage\": \"119\", \"name_0\": \"surname:Rostami;given-names:J\", \"name_1\": \"surname:Fotaki;given-names:G\", \"name_2\": \"surname:Sirois;given-names:J\", \"name_3\": \"surname:Mzezewa;given-names:R\", \"name_4\": \"surname:Bergstr\\u00f6m;given-names:J\", \"name_5\": \"surname:Essand;given-names:M\", \"name_6\": \"surname:Healy;given-names:L\", \"name_7\": \"surname:Erlandsson;given-names:A\", \"pub-id_doi\": \"10.1186/s12974-020-01776-7\", \"section_type\": \"REF\", \"source\": \"J. Neuroinflamm\", \"type\": \"ref\", \"volume\": \"17\", \"year\": \"2020\"}, \"text\": \"Astrocytes have the capacity to act as antigen-presenting cells in the Parkinson\\u2019s disease brain\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65795, \"infons\": {\"fpage\": \"43\", \"lpage\": \"52\", \"name_0\": \"surname:Sadakata;given-names:T\", \"name_1\": \"surname:Mizoguchi;given-names:A\", \"name_2\": \"surname:Sato;given-names:Y\", \"name_3\": \"surname:Katoh-Semba;given-names:R\", \"name_4\": \"surname:Fukuda;given-names:M\", \"name_5\": \"surname:Mikoshiba;given-names:K\", \"name_6\": \"surname:Furuichi;given-names:T\", \"pub-id_doi\": \"10.1523/JNEUROSCI.2528-03.2004\", \"pub-id_pmid\": \"14715936\", \"section_type\": \"REF\", \"source\": \"J. Neurosci\", \"type\": \"ref\", \"volume\": \"24\", \"year\": \"2004\"}, \"text\": \"The secretory granule-associated protein CAPS2 regulates neurotrophin release and cell survival\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65891, \"infons\": {\"fpage\": \"D852\", \"lpage\": \"857\", \"name_0\": \"surname:Samarajiwa;given-names:SA\", \"name_1\": \"surname:Forster;given-names:S\", \"name_2\": \"surname:Auchettl;given-names:K\", \"name_3\": \"surname:Hertzog;given-names:PJ\", \"pub-id_doi\": \"10.1093/nar/gkn732\", \"pub-id_pmid\": \"18996892\", \"section_type\": \"REF\", \"source\": \"Nucleic Acids Res\", \"type\": \"ref\", \"volume\": \"37\", \"year\": \"2009\"}, \"text\": \"INTERFEROME: the database of interferon regulated genes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65947, \"infons\": {\"fpage\": \"4622\", \"lpage\": \"4643\", \"name_0\": \"surname:Santos;given-names:SED\", \"name_1\": \"surname:Medeiros;given-names:M\", \"name_10\": \"surname:Noctor;given-names:SC\", \"name_11\": \"surname:Sherwood;given-names:CC\", \"name_12\": \"surname:Kaas;given-names:JH\", \"name_13\": \"surname:Manger;given-names:PR\", \"name_14\": \"surname:Herculano-Houzel;given-names:S\", \"name_2\": \"surname:Porfirio;given-names:J\", \"name_3\": \"surname:Tavares;given-names:W\", \"name_4\": \"surname:Pess\\u00f4a;given-names:L\", \"name_5\": \"surname:Grinberg;given-names:L\", \"name_6\": \"surname:Leite;given-names:REP\", \"name_7\": \"surname:Ferretti-Rebustini;given-names:REL\", \"name_8\": \"surname:Suemoto;given-names:CK\", \"name_9\": \"surname:Filho;given-names:WJ\", \"pub-id_doi\": \"10.1523/JNEUROSCI.2339-19.2020\", \"pub-id_pmid\": \"32253358\", \"section_type\": \"REF\", \"source\": \"J. Neurosci\", \"type\": \"ref\", \"volume\": \"40\", \"year\": \"2020\"}, \"text\": \"Similar microglial cell densities across brain structures and mammalian species: implications for brain tissue function\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66067, \"infons\": {\"fpage\": \"1015\", \"issue\": \"4\", \"lpage\": \"1030. e16\", \"name_0\": \"surname:Saunders;given-names:A\", \"name_1\": \"surname:Macosko;given-names:EZ\", \"name_10\": \"surname:Goeva;given-names:A\", \"name_11\": \"surname:Nemesh;given-names:J\", \"name_12\": \"surname:Kamitaki;given-names:N\", \"name_13\": \"surname:Brumbaugh;given-names:S\", \"name_14\": \"surname:Kulp;given-names:D\", \"name_15\": \"surname:McCarroll;given-names:SA\", \"name_2\": \"surname:Wysoker;given-names:A\", \"name_3\": \"surname:Goldman;given-names:M\", \"name_4\": \"surname:Krienen;given-names:FM\", \"name_5\": \"surname:de Rivera;given-names:H\", \"name_6\": \"surname:Bien;given-names:E\", \"name_7\": \"surname:Baum;given-names:M\", \"name_8\": \"surname:Bortolin;given-names:L\", \"name_9\": \"surname:Wang;given-names:S\", \"pub-id_pmid\": \"30096299\", \"section_type\": \"REF\", \"source\": \"Cell\", \"type\": \"ref\", \"volume\": \"174\", \"year\": \"2018\"}, \"text\": \"Molecular diversity and specializations among the cells of the adult mouse brain\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66148, \"infons\": {\"fpage\": \"163\", \"lpage\": \"189\", \"name_0\": \"surname:Schroder;given-names:K\", \"name_1\": \"surname:Hertzog;given-names:PJ\", \"name_2\": \"surname:Ravasi;given-names:T\", \"name_3\": \"surname:Hume;given-names:DA\", \"pub-id_doi\": \"10.1189/jlb.0603252\", \"pub-id_pmid\": \"14525967\", \"section_type\": \"REF\", \"source\": \"Journal of Leukocyte Biology\", \"type\": \"ref\", \"volume\": \"75\", \"year\": \"2004\"}, \"text\": \"Interferon-\\u03b3: an overview of signals, mechanisms and functions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66215, \"infons\": {\"fpage\": \"1308\", \"lpage\": \"1323.e30\", \"name_0\": \"surname:Shekhar;given-names:K\", \"name_1\": \"surname:Lapan;given-names:SW\", \"name_10\": \"surname:McCarroll;given-names:SA\", \"name_11\": \"surname:Cepko;given-names:CL\", \"name_12\": \"surname:Regev;given-names:A\", \"name_13\": \"surname:Sanes;given-names:JR\", \"name_2\": \"surname:Whitney;given-names:IE\", \"name_3\": \"surname:Tran;given-names:NM\", \"name_4\": \"surname:Macosko;given-names:EZ\", \"name_5\": \"surname:Kowalczyk;given-names:M\", \"name_6\": \"surname:Adiconis;given-names:X\", \"name_7\": \"surname:Levin;given-names:JZ\", \"name_8\": \"surname:Nemesh;given-names:J\", \"name_9\": \"surname:Goldman;given-names:M\", \"pub-id_doi\": \"10.1016/j.cell.2016.07.054\", \"pub-id_pmid\": \"27565351\", \"section_type\": \"REF\", \"source\": \"Cell\", \"type\": \"ref\", \"volume\": \"166\", \"year\": \"2016\"}, \"text\": \"Comprehensive classification of retinal bipolar neurons by single-cell transcriptomics\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66302, \"infons\": {\"fpage\": \"1744\", \"lpage\": \"1746\", \"name_0\": \"surname:Shuai;given-names:K\", \"name_1\": \"surname:Stark;given-names:GR\", \"name_2\": \"surname:Kerr;given-names:IM\", \"name_3\": \"surname:Darnell;given-names:JE\", \"pub-id_doi\": \"10.1126/science.7690989\", \"pub-id_pmid\": \"7690989\", \"section_type\": \"REF\", \"source\": \"Science\", \"type\": \"ref\", \"volume\": \"261\", \"year\": \"1993\"}, \"text\": \"A single phosphotyrosine residue of Stat91 required for gene activation by interferon-gamma\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66394, \"infons\": {\"name_0\": \"surname:Snyder;given-names:ME\", \"name_1\": \"surname:Finlayson;given-names:MO\", \"name_10\": \"surname:Robbins;given-names:H\", \"name_11\": \"surname:Hook;given-names:JL\", \"name_12\": \"surname:Sykes;given-names:M\", \"name_13\": \"surname:D\\u2019Ovidio;given-names:F\", \"name_14\": \"surname:Bacchetta;given-names:M\", \"name_15\": \"surname:Sonett;given-names:JR\", \"name_16\": \"surname:Lederer;given-names:DJ\", \"name_17\": \"surname:Arcasoy;given-names:S\", \"name_18\": \"surname:Sims;given-names:PA\", \"name_19\": \"surname:Farber;given-names:DL\", \"name_2\": \"surname:Connors;given-names:TJ\", \"name_3\": \"surname:Dogra;given-names:P\", \"name_4\": \"surname:Senda;given-names:T\", \"name_5\": \"surname:Bush;given-names:E\", \"name_6\": \"surname:Carpenter;given-names:D\", \"name_7\": \"surname:Marboe;given-names:C\", \"name_8\": \"surname:Benvenuto;given-names:L\", \"name_9\": \"surname:Shah;given-names:L\", \"pub-id_doi\": \"10.1126/sciimmunol.aav5581\", \"section_type\": \"REF\", \"source\": \"Sci. Immunol\", \"type\": \"ref\", \"volume\": \"4\", \"year\": \"2019\"}, \"text\": \"Generation and persistence of human tissue-resident memory T cells in lung transplantation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66485, \"infons\": {\"fpage\": \"123\", \"lpage\": \"131.e6\", \"name_0\": \"surname:Sommer;given-names:A\", \"name_1\": \"surname:Marxreiter;given-names:F\", \"name_10\": \"surname:Kohl;given-names:Z\", \"name_11\": \"surname:Xiang;given-names:W\", \"name_12\": \"surname:Gage;given-names:FH\", \"name_13\": \"surname:Winkler;given-names:J\", \"name_14\": \"surname:Prots;given-names:I\", \"name_15\": \"surname:Winner;given-names:B\", \"name_2\": \"surname:Krach;given-names:F\", \"name_3\": \"surname:Fadler;given-names:T\", \"name_4\": \"surname:Grosch;given-names:J\", \"name_5\": \"surname:Maroni;given-names:M\", \"name_6\": \"surname:Graef;given-names:D\", \"name_7\": \"surname:Eberhardt;given-names:E\", \"name_8\": \"surname:Riemenschneider;given-names:MJ\", \"name_9\": \"surname:Yeo;given-names:GW\", \"pub-id_doi\": \"10.1016/j.stem.2018.06.015\", \"pub-id_pmid\": \"29979986\", \"section_type\": \"REF\", \"source\": \"Cell Stem Cell\", \"type\": \"ref\", \"volume\": \"23\", \"year\": \"2018\"}, \"text\": \"Th17 lymphocytes induce neuronal cell death in a human iPSC-based model of parkinson\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66582, \"infons\": {\"fpage\": \"133\", \"lpage\": \"141\", \"name_0\": \"surname:Strickland;given-names:MR\", \"name_1\": \"surname:Koller;given-names:EJ\", \"name_2\": \"surname:Deng;given-names:DZ\", \"name_3\": \"surname:Ceballos-Diaz;given-names:C\", \"name_4\": \"surname:Golde;given-names:TE\", \"name_5\": \"surname:Chakrabarty;given-names:P\", \"pub-id_doi\": \"10.1016/j.nbd.2017.11.007\", \"pub-id_pmid\": \"29196213\", \"section_type\": \"REF\", \"source\": \"Neurobiol. Dis\", \"type\": \"ref\", \"volume\": \"110\", \"year\": \"2018\"}, \"text\": \"Ifngr1 and Stat1 mediated canonical Ifn-\\u03b3 signaling drives nigrostriatal degeneration\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66672, \"infons\": {\"fpage\": \"6720\", \"lpage\": \"6732\", \"name_0\": \"surname:St\\u00fcve;given-names:O\", \"name_1\": \"surname:Youssef;given-names:S\", \"name_10\": \"surname:Zamvil;given-names:SS\", \"name_2\": \"surname:Slavin;given-names:AJ\", \"name_3\": \"surname:King;given-names:CL\", \"name_4\": \"surname:Patarroyo;given-names:JC\", \"name_5\": \"surname:Hirschberg;given-names:DL\", \"name_6\": \"surname:Brickey;given-names:WJ\", \"name_7\": \"surname:Soos;given-names:JM\", \"name_8\": \"surname:Piskurich;given-names:JF\", \"name_9\": \"surname:Chapman;given-names:HA\", \"pub-id_doi\": \"10.4049/jimmunol.169.12.6720\", \"pub-id_pmid\": \"12471103\", \"section_type\": \"REF\", \"source\": \"The Journal of Immunology\", \"type\": \"ref\", \"volume\": \"169\", \"year\": \"2002\"}, \"text\": \"The Role of the MHC Class II Transactivator in Class II Expression and Antigen Presentation by Astrocytes and in Susceptibility to Central Nervous System Autoimmune Disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66845, \"infons\": {\"fpage\": \"656\", \"lpage\": \"661\", \"name_0\": \"surname:Sulzer;given-names:D\", \"name_1\": \"surname:Alcalay;given-names:RN\", \"name_10\": \"surname:Dawson;given-names:VL\", \"name_11\": \"surname:Dawson;given-names:TM\", \"name_12\": \"surname:Oseroff;given-names:C\", \"name_13\": \"surname:Pham;given-names:J\", \"name_14\": \"surname:Sidney;given-names:J\", \"name_15\": \"surname:Dillon;given-names:MB\", \"name_16\": \"surname:Carpenter;given-names:C\", \"name_17\": \"surname:Weiskopf;given-names:D\", \"name_18\": \"surname:Phillips;given-names:E\", \"name_19\": \"surname:Mallal;given-names:S\", \"name_2\": \"surname:Garretti;given-names:F\", \"name_20\": \"surname:Peters;given-names:B\", \"name_21\": \"surname:Frazier;given-names:A\", \"name_22\": \"surname:Lindestam Arlehamn;given-names:CS\", \"name_23\": \"surname:Sette;given-names:A\", \"name_3\": \"surname:Cote;given-names:L\", \"name_4\": \"surname:Kanter;given-names:E\", \"name_5\": \"surname:Agin-Liebes;given-names:J\", \"name_6\": \"surname:Liong;given-names:C\", \"name_7\": \"surname:McMurtrey;given-names:C\", \"name_8\": \"surname:Hildebrand;given-names:WH\", \"name_9\": \"surname:Mao;given-names:X\", \"pub-id_doi\": \"10.1038/nature22815\", \"pub-id_pmid\": \"28636593\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"546\", \"year\": \"2017\"}, \"text\": \"T cells from patients with Parkinson\\u2019s disease recognize \\u03b1-synuclein peptides\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66927, \"infons\": {\"fpage\": \"1113\", \"lpage\": \"1121\", \"name_0\": \"surname:Svensson;given-names:M\", \"name_1\": \"surname:Marsal;given-names:J\", \"name_2\": \"surname:Ericsson;given-names:A\", \"name_3\": \"surname:Carramolino;given-names:L\", \"name_4\": \"surname:Brod\\u00e9n;given-names:T\", \"name_5\": \"surname:M\\u00e1rquez;given-names:G\", \"name_6\": \"surname:Agace;given-names:WW\", \"pub-id_doi\": \"10.1172/JCI15988\", \"pub-id_pmid\": \"12393847\", \"section_type\": \"REF\", \"source\": \"J. Clin. Invest\", \"type\": \"ref\", \"volume\": \"110\", \"year\": \"2002\"}, \"text\": \"CCL25 mediates the localization of recently activated CD8alphabeta(+) lymphocytes to the small-intestinal mucosa\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67040, \"infons\": {\"fpage\": \"3883\", \"issue\": \"24\", \"lpage\": \"3893\", \"name_0\": \"surname:Tanaka;given-names:Y\", \"name_1\": \"surname:Ono;given-names:N\", \"name_2\": \"surname:Shima;given-names:T\", \"name_3\": \"surname:Tanaka;given-names:G\", \"name_4\": \"surname:Katoh;given-names:Y\", \"name_5\": \"surname:Nakayama;given-names:K\", \"name_6\": \"surname:Takatsu;given-names:H\", \"name_7\": \"surname:Shin;given-names:H-W\", \"name_8\": \"surname:Nakano;given-names:A\", \"pub-id_pmid\": \"27733620\", \"section_type\": \"REF\", \"source\": \"Mol. Biol. Cell\", \"type\": \"ref\", \"volume\": \"27\", \"year\": \"2016\"}, \"text\": \"The phospholipid flippase ATP9A is required for the recycling pathway from the endosomes to the plasma membrane\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67152, \"infons\": {\"fpage\": \"1099\", \"lpage\": \"1107\", \"name_0\": \"surname:Taylor;given-names:GA\", \"pub-id_doi\": \"10.1111/j.1462-5822.2007.00916.X\", \"pub-id_pmid\": \"17359233\", \"section_type\": \"REF\", \"source\": \"Cell. Microbiol\", \"type\": \"ref\", \"volume\": \"9\", \"year\": \"2007\"}, \"text\": \"IRG proteins: key mediators of interferon-regulated host resistance to intracellular pathogens\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67247, \"infons\": {\"fpage\": \"307\", \"lpage\": \"319\", \"name_0\": \"surname:Ugras;given-names:S\", \"name_1\": \"surname:Daniels;given-names:MJ\", \"name_10\": \"surname:Martinez;given-names:D\", \"name_11\": \"surname:Evans;given-names:P\", \"name_12\": \"surname:Brown;given-names:D\", \"name_13\": \"surname:Duda;given-names:JE\", \"name_14\": \"surname:Ischiropoulos;given-names:H\", \"name_2\": \"surname:Fazelinia;given-names:H\", \"name_3\": \"surname:Gould;given-names:NS\", \"name_4\": \"surname:Yocum;given-names:AK\", \"name_5\": \"surname:Luk;given-names:KC\", \"name_6\": \"surname:Luna;given-names:E\", \"name_7\": \"surname:Ding;given-names:H\", \"name_8\": \"surname:McKennan;given-names:C\", \"name_9\": \"surname:Seeholzer;given-names:S\", \"pub-id_doi\": \"10.1016/j.ebiom.2018.05.007\", \"pub-id_pmid\": \"29759483\", \"section_type\": \"REF\", \"source\": \"EBioMedicine\", \"type\": \"ref\", \"volume\": \"31\", \"year\": \"2018\"}, \"text\": \"Induction of the immunoproteasome subunit Lmp7 links proteostasis and immunity in \\u03b1-synuclein aggregation disorders\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67367, \"infons\": {\"fpage\": \"67\", \"lpage\": \"75\", \"name_0\": \"surname:van den Elsen;given-names:PJ\", \"name_1\": \"surname:Holling;given-names:TM\", \"name_2\": \"surname:Kuipers;given-names:HF\", \"name_3\": \"surname:van der Stoep;given-names:N\", \"pub-id_doi\": \"10.1016/j.coi.2003.11.015\", \"pub-id_pmid\": \"14734112\", \"section_type\": \"REF\", \"source\": \"Curr. Opin. Immunol\", \"type\": \"ref\", \"volume\": \"16\", \"year\": \"2004\"}, \"text\": \"Transcriptional regulation of antigen presentation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67418, \"infons\": {\"fpage\": \"789\", \"lpage\": \"800\", \"name_0\": \"surname:Vass;given-names:K\", \"name_1\": \"surname:Lassmann;given-names:H\", \"pub-id_pmid\": \"2121041\", \"section_type\": \"REF\", \"source\": \"Am. J. Pathol\", \"type\": \"ref\", \"volume\": \"137\", \"year\": \"1990\"}, \"text\": \"Intrathecal application of interferon gamma. Progressive appearance of MHC antigens within the rat nervous system\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67532, \"infons\": {\"fpage\": \"22\", \"lpage\": \"29\", \"name_0\": \"surname:Wang;given-names:F\", \"name_1\": \"surname:Flanagan;given-names:J\", \"name_2\": \"surname:Su;given-names:N\", \"name_3\": \"surname:Wang;given-names:L-C\", \"name_4\": \"surname:Bui;given-names:S\", \"name_5\": \"surname:Nielson;given-names:A\", \"name_6\": \"surname:Wu;given-names:X\", \"name_7\": \"surname:Vo;given-names:H-T\", \"name_8\": \"surname:Ma;given-names:X-J\", \"name_9\": \"surname:Luo;given-names:Y\", \"pub-id_doi\": \"10.1016/j.jmoldx.2011.08.002\", \"pub-id_pmid\": \"22166544\", \"section_type\": \"REF\", \"source\": \"J. Mol. Diagn\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2012\"}, \"text\": \"RNAscope: A novel in Situ RNA analysis platform for formalin-fixed, paraffin-embedded tissues\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67626, \"infons\": {\"fpage\": \"e90\", \"name_0\": \"surname:Xie;given-names:Z\", \"name_1\": \"surname:Bailey;given-names:A\", \"name_10\": \"surname:Jeon;given-names:M\", \"name_11\": \"surname:Ma\\u2019ayan;given-names:A\", \"name_2\": \"surname:Kuleshov;given-names:MV\", \"name_3\": \"surname:Clarke;given-names:DJB\", \"name_4\": \"surname:Evangelista;given-names:JE\", \"name_5\": \"surname:Jenkins;given-names:SL\", \"name_6\": \"surname:Lachmann;given-names:A\", \"name_7\": \"surname:Wojciechowicz;given-names:ML\", \"name_8\": \"surname:Kropiwnicki;given-names:E\", \"name_9\": \"surname:Jagodnik;given-names:KM\", \"pub-id_doi\": \"10.1002/cpz1.90\", \"pub-id_pmid\": \"33780170\", \"section_type\": \"REF\", \"source\": \"Current Protocols\", \"type\": \"ref\", \"volume\": \"1\", \"year\": \"2021\"}, \"text\": \"Gene Set Knowledge Discovery with Enrichr\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67668, \"infons\": {\"fpage\": \"2923\", \"lpage\": \"2934\", \"name_0\": \"surname:Yshii;given-names:LM\", \"name_1\": \"surname:Gebauer;given-names:CM\", \"name_10\": \"surname:Liblau;given-names:R\", \"name_2\": \"surname:Pignolet;given-names:B\", \"name_3\": \"surname:Maur\\u00e9;given-names:E\", \"name_4\": \"surname:Qu\\u00e9riault;given-names:C\", \"name_5\": \"surname:Pierau;given-names:M\", \"name_6\": \"surname:Saito;given-names:H\", \"name_7\": \"surname:Suzuki;given-names:N\", \"name_8\": \"surname:Brunner-Weinzierl;given-names:M\", \"name_9\": \"surname:Bauer;given-names:J\", \"pub-id_doi\": \"10.1093/brain/aww225\", \"pub-id_pmid\": \"27604307\", \"section_type\": \"REF\", \"source\": \"Brain\", \"type\": \"ref\", \"volume\": \"139\", \"year\": \"2016\"}, \"text\": \"CTLA4 blockade elicits paraneoplastic neurological disease in a mouse model\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67744, \"infons\": {\"fpage\": \"57\", \"name_0\": \"surname:Yuan;given-names:J\", \"name_1\": \"surname:Levitin;given-names:HM\", \"name_10\": \"surname:Bruce;given-names:JN\", \"name_11\": \"surname:Lasorella;given-names:A\", \"name_12\": \"surname:Iavarone;given-names:A\", \"name_13\": \"surname:Sims;given-names:PA\", \"name_2\": \"surname:Frattini;given-names:V\", \"name_3\": \"surname:Bush;given-names:EC\", \"name_4\": \"surname:Boyett;given-names:DM\", \"name_5\": \"surname:Samanamud;given-names:J\", \"name_6\": \"surname:Ceccarelli;given-names:M\", \"name_7\": \"surname:Dovas;given-names:A\", \"name_8\": \"surname:Zanazzi;given-names:G\", \"name_9\": \"surname:Canoll;given-names:P\", \"pub-id_doi\": \"10.1186/s13073-018-0567-9\", \"pub-id_pmid\": \"30041684\", \"section_type\": \"REF\", \"source\": \"Genome Med\", \"type\": \"ref\", \"volume\": \"10\", \"year\": \"2018\"}, \"text\": \"Single-cell transcriptome analysis of lineage diversity in high-grade glioma\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67821, \"infons\": {\"fpage\": \"82\", \"name_0\": \"surname:Zhao;given-names:W\", \"name_1\": \"surname:Dovas;given-names:A\", \"name_10\": \"surname:Bruce;given-names:JN\", \"name_11\": \"surname:Canoll;given-names:P\", \"name_12\": \"surname:Sims;given-names:PA\", \"name_2\": \"surname:Spinazzi;given-names:EF\", \"name_3\": \"surname:Levitin;given-names:HM\", \"name_4\": \"surname:Banu;given-names:MA\", \"name_5\": \"surname:Upadhyayula;given-names:P\", \"name_6\": \"surname:Sudhakar;given-names:T\", \"name_7\": \"surname:Marie;given-names:T\", \"name_8\": \"surname:Otten;given-names:ML\", \"name_9\": \"surname:Sisti;given-names:MB\", \"pub-id_doi\": \"10.1186/s13073-021-00894-y\", \"pub-id_pmid\": \"33975634\", \"section_type\": \"REF\", \"source\": \"Genome Med\", \"type\": \"ref\", \"volume\": \"13\", \"year\": \"2021\"}, \"text\": \"Deconvolution of cell type-specific drug responses in human tumor tissue with single-cell RNA-seq\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67919, \"infons\": {\"fpage\": \"239\", \"lpage\": \"260\", \"name_0\": \"surname:Zhou;given-names:F\", \"pub-id_doi\": \"10.1080/08830180902978120\", \"pub-id_pmid\": \"19811323\", \"section_type\": \"REF\", \"source\": \"Int. Rev. Immunol\", \"type\": \"ref\", \"volume\": \"28\", \"year\": \"2009\"}, \"text\": \"Molecular mechanisms of IFN-gamma to up-regulate MHC class I antigen processing and presentation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 68016, \"infons\": {\"file\": \"nihms-1900921-f0001.jpg\", \"id\": \"F1\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Core transcriptional responses to IFN-\\u03b3 are conserved in neurons but are lower in magnitude compared to glia (a) Experimental schematic for NeuN\\u2212/NeuN + RNA-seq after injection of saline or IFN-\\u03b3 in the midbrain of WT C57BL/6J mice. (b) Principal component analysis (PCA) of RNA-seq UMI counts normalized with DESeq2 variance stabilizing transformation. Each point represents a bulk RNA-seq replicate of ~ 1000 sorted nuclei (n = 4\\u20136 replicates each of NeuN\\u2212 or NeuN + nuclei from 2 to 3 mice for each treatment/timepoint). (c) Number of differentially expressed genes (DEG) for IFN-\\u03b3 vs. Sal comparisons in NeuN\\u2212 or NeuN + samples, or both (overlap), at each timepoint (DESeq2, \\u2223log2FC\\u2223 > 1 and pAdj < 0.01). (d) Mean \\u00b1 SEM for mRNA abundance in NeuN\\u2212/NeuN + samples, normalized as log2 (CPM + 1). Samples as in (b), (n = 4\\u20136 replicates each of NeuN\\u2212/+ nuclei from 2 to 3 mice for each treatment/timepoint). Saline samples from all timepoints are together at t = 0. (e) Heatmap of z-scored, normalized mRNA abundance (log2[CPM + 1]) for 28 mRNAs upregulated at \\u2265 3 timepoints in both NeuN\\u2212 and NeuN + samples (see Supp. Fig. 1d). Samples as in (b); each column is a replicate. (f) Scatter plots comparing the effect of IFN-\\u03b3 (log2 fold change) in NeuN + vs. NeuN\\u2212 nuclei for all IFN-\\u03b3-induced genes at each timepoint (DESeq2, log2FC > 1 and pAdj < 0.01). Points are colored by the DESeq2 log2FC comparing IFN-\\u03b3-treated NeuN + vs. NeuN\\u2212 samples: values \\u22121 to \\u22123 correspond correspond to 2- and 8-fold higher relative expression in NeuN\\u2212 nuclei, respectively.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69614, \"infons\": {\"file\": \"nihms-1900921-f0002.jpg\", \"id\": \"F2\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Divergent transcriptional responses to IFN-\\u03b3 in glia (a) Summary of DEGs from interaction analysis (DESeq2: ~NeuN + IFN + NeuN:IFN), broken down by baseline expression and effect of IFN-\\u03b3 in NeuN + and NeuN\\u2212 samples. (b) Heatmap of z-scored mRNA abundances for genes upregulated in NeuN\\u2212 samples and significant in NeuN \\u00d7 IFN interaction analysis in (a), normalized as log2(CPM + 1). Each column represents a bulk RNA-seq replicate of ~ 1000 sorted nuclei (n = 4\\u20136 replicates from 2 to 3 mice for each treatment/timepoint). (c) Mean \\u00b1 SEM for mRNA abundance in NeuN\\u2212/NeuN + samples, normalized as log2(CPM + 1). Samples as in (b), (n = 4\\u20136 replicates each of NeuN\\u2212/+ nuclei from 2 to 3 mice for each treatment/timepoint). All saline samples are together at t = 0. (d) Heatmap of z-scored mRNA abundances for genes downregulated in NeuN\\u2212 samples and significant in NeuN \\u00d7 IFN interaction analysis in (a), normalized as log2(CPM + 1). Each column represents a bulk RNA-seq replicate of ~ 1000 sorted nuclei (n = 4\\u20136 replicates from 2 to 3 mice for each treatment/timepoint).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 70708, \"infons\": {\"file\": \"nihms-1900921-f0003.jpg\", \"id\": \"F3\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Single nucleus RNA-seq analysis of responses to IFN-\\u03b3. (a) UMAP embedding of snRNA-seq profiles from the ventral midbrain used for downstream analysis (4,699 nuclei after collapse to major cell types and filtering, see Methods). Left: major cell types for downstream analysis, see Supp. Fig. 4b for clustering with marker genes. Right: mouse sample origin (n = 1 for saline, n = 2 for IFN-\\u03b3). (b) Box and whiskers plots depicting the sum of mRNA abundance (log2[CPM + 1]) in each major cell group for the indicated sets of MHC-I or MHC-II genes. (c) Clustered heatmap of z-scored mRNA abundance (log2[CPM + 1]) for the union of highly differentially expressed genes across all major cell groups (modified Mann-Whitney U test comparing IFN vs. Sal, \\u2223log2FC\\u2223 > 4 and q < 0.05). (d) Heatmap of average mRNA abundance (log2[CPM + 1]) for genes in MGI GO:0060335: \\u2018positive regulation of interferon-gamma-mediated signaling pathway\\u2019. (e) Upper: UMAP embedding of dopamine neuronal snRNA-seq profiles with subcluster IDs and mouse sample origin. Lower: Heatmap of average mRNA abundance (log2[CPM + 1]) for select IFN response genes within dopamine neuronal subclusters from each mouse sample.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71906, \"infons\": {\"file\": \"nihms-1900921-f0004.jpg\", \"id\": \"F4\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Ifngr1 is required for pSTAT1 induction and MHC-I upregulation in mDA neurons after in vivo exposure to IFN-\\u03b3 (a) TH and pSTAT1-Y701 immunofluorescence in the SNc of WT:Ifngr1fl/fl or DAT-Cre:Ifngr1fl/fl mice after saline or IFN-\\u03b3. White arrowheads indicate TH+ mDA neurons, blue arrows indicate NeuN\\u2212 nuclei, and yellow arrows indicate TH\\u2212 neurons. Scale bar: 50 \\u03bcm. (b) pSTAT1-Y701 in TH+/NeuN+ mDA neurons, TH\\u2212/NeuN+ non-DA neurons, and TH\\u2212/NeuN\\u2212 glial nuclei in the SNc of WT:Ifngr1fl/fl or DAT-Cre:Ifngr1fl/fl mice after exposure to IFN-\\u03b3. Scale bar: 10 \\u03bcm. (c) Average nuclear pSTAT1 intensity (arbitrary units, median background subtracted) in each of the indicated cell types, genotypes, and treatment groups, related to (a-b). 10\\u201320 neurons were quantified per region per mouse; n = 2 saline, n = 3 IFN-\\u03b3. **** p < 0.001, Mann-Whitney U test. (d-e) MHC-I and Tap1 RNA FISH in the SNc of WT:Ifngr1fl/fl or DAT-Cre:Ifngr1fl/fl mice after saline or IFN-\\u03b3. White arrowheads indicate TH+ mDA neurons, blue arrows indicate other cells with intact Ifngr1. Scale bars (d): 20 \\u03bcm, (e): 15 \\u03bcm. (f) Average MHC-I or Tap1 RNA intensity (arbitrary units, median background subtracted) in TH+ mDA neurons of the indicated genotype and treatment groups, related to (d-e). 10\\u201320 neurons were quantified per region per mouse. WT; n = 3 Sal, n = 4 IFN-\\u03b3. DAT-Cre:Ifngr1fl/fl (DA-cKO); n = 3 Sal, n = 4 IFN-\\u03b3. **** p < 0.001, ** p < 0.01, Mann-Whitney U test.\", \"sentences\": [], \"annotations\": [], \"relations\": []}], \"annotations\": [], \"relations\": []}]}, {\"source\": \"PMC\", \"date\": \"20240622\", \"key\": \"pmc.key\", \"infons\": {}, \"documents\": [{\"id\": \"10460506\", \"infons\": {\"license\": \"author_manuscript\"}, \"passages\": [{\"offset\": 0, \"infons\": {\"article-id_doi\": \"10.1016/j.bbi.2023.04.008\", \"article-id_manuscript\": \"NIHMS1900921\", \"article-id_pmc\": \"10460506\", \"article-id_pmid\": \"37100211\", \"fpage\": \"277\", \"license\": \"\\n          This file is available for text mining. It may also be used consistent with the principles of fair use under the copyright law.\\n        \", \"lpage\": \"291\", \"name_0\": \"surname:Hobson;given-names:Benjamin D.\", \"name_1\": \"surname:Stanley;given-names:Adrien T.\", \"name_2\": \"surname:De Los Santos;given-names:Mark B.\", \"name_3\": \"surname:Culbertson;given-names:Bruce\", \"name_4\": \"surname:Mosharov;given-names:Eugene V.\", \"name_5\": \"surname:Sims;given-names:Peter A.\", \"name_6\": \"surname:Sulzer;given-names:David\", \"section_type\": \"TITLE\", \"type\": \"front\", \"volume\": \"111\", \"year\": \"2023\"}, \"text\": \"Conserved and cell type-specific transcriptional responses to IFN-\\u03b3 in the ventral midbrain\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 96, \"infons\": {\"section_type\": \"ABSTRACT\", \"type\": \"abstract\"}, \"text\": \"Dysregulated inflammation within the central nervous system (CNS) contributes to neuropathology in infectious, autoimmune, and neurodegenerative disease. With the exception of microglia, major histocompatibility complex (MHC) proteins are virtually undetectable in the mature, healthy central nervous system (CNS). Neurons have generally been considered incapable of antigen presentation, and although interferon gamma (IFN-\\u03b3) can elicit neuronal MHC class I (MHC-I) expression and antigen presentation in vitro, it has been unclear whether similar responses occur in vivo. Here we directly injected IFN-\\u03b3 into the ventral midbrain of mature mice and analyzed gene expression profiles of specific CNS cell types. We found that IFN-\\u03b3 upregulated MHC-I and associated mRNAs in ventral midbrain microglia, astrocytes, oligodendrocytes, and GABAergic, glutamatergic, and dopaminergic neurons. The core set of IFN-\\u03b3-induced genes and their response kinetics were similar in neurons and glia, but with a lower amplitude of expression in neurons. A diverse repertoire of genes was upregulated in glia, particularly microglia, which were the only cells to undergo cellular proliferation and express MHC classII (MHC-II) and associated genes. To determine if neurons respond directly via cell-autonomous IFN-\\u03b3 receptor (IFNGR) signaling, we produced mutant mice with a deletion of the IFN-\\u03b3-binding domain of IFNGR1 in dopaminergic neurons, which resulted in a complete loss of dopaminergic neuronal responses to IFN-\\u03b3. Our results demonstrate that IFN-\\u03b3 induces neuronal IFNGR signaling and upregulation of MHC-I and related genes in vivo, although the expression level is low compared to oligodendrocytes, astrocytes, and microglia.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 1854, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"title_1\"}, \"text\": \"Introduction\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 1867, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Neuroinflammation is associated with neurodegenerative diseases including multiple sclerosis (MS), Alzheimer\\u2019s disease (AD), and Parkinson\\u2019s disease (PD), while clonally expanded T cells accumulate in the aged brain. MS is a demyelinating disorder classically characterized by autoimmune responses to oligodendrocyte proteins, and recent evidence suggests that T cell reactivity towards the neuronal protein \\u03b2-synuclein contributes to grey matter pathology in MS. There is considerable neuropathological evidence of inflammation in the substantia nigra pars compacta (SNc) of PD patients, including reactive, HLA-DR+ microglia and the presence of CD8 + and CD4 + T cells near the pigmented SNc dopaminergic (DA) neurons and noradrenergic neurons of the locus coeruleus, neurons that are killed in PD. We have demonstrated T cell autoreactivity towards \\u03b1-synuclein peptides in PD patients and recently showed that peripheral \\u03b1-synuclein-reactive T cells are most abundant in early stage PD patients. Further evidence suggests that T cell infiltration of the midbrain parenchyma is more common in early stage PD, but the precise role of T cells in PD pathophysiology remains unknown.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 3060, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Interferon gamma (IFN-\\u03b3) is a proinflammatory cytokine released primarily from CD4 + T helper type 1 (Th1), CD8 + T cells, and natural killer (NK) cells, cell types with low abundance in the brain parenchyma. IFN-\\u03b3 elicits cellular responses by increasing transcription of interferon-stimulated genes (ISGs), many of which encode transcription factors, antigen processing and presentation machinery, and innate immune effector proteins. Although IFN-\\u03b3 was formerly considered to be generally undetectable in the CNS, low levels of IFN-\\u03b3 derived from meningeal T cells can regulate neuronal activity and behavior in healthy mice. CNS cells may be exposed to high levels of IFN-\\u03b3 under infectious and inflammatory conditions that feature blood brain barrier disruption and T cell infiltration. In a mouse model of viral encephalitis, the release of IFN-\\u03b3 from CD8 + T cells has been suggested to drive the stripping of neuronal synapses via actions of phagocytic cells.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 4052, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Several reports indicate that IFN-\\u03b3 may play roles in PD pathogenesis. Elevated plasma levels of IFN-\\u03b3 were found in PD patients, and we recently identified altered transcriptional signatures in peripheral T cells of PD patients that may contribute to Th1/IFN-\\u03b3 bias. Elevated levels of IFN-\\u03b3 were reported in the substantia nigra, caudate, and putamen in PD, regions that contain dopaminergic cell bodies and their axonal projections. Constitutive expression of IFN-\\u03b3 in the lateral ventricles of mice leads to progressive nigrostriatal degeneration. Collectively, these data suggest that infiltrating T cells may contribute to dopaminergic neuron demise in PD (reviewed in), in part by IFN-\\u03b3-driven neuroinflammation.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 4796, \"infons\": {\"section_type\": \"INTRO\", \"type\": \"paragraph\"}, \"text\": \"Key questions on the possible role of IFN-\\u03b3 in PD pathogenesis include: how do specific CNS cell types respond to IFN-\\u03b3, and which are capable of antigen presentation, by which pathways, and does cellular proliferation occur? Do these include mature neurons, which are often thought to not present antigen, and if so, does this consist of MHC-I, MHC-II or both? Do midbrain neurons directly respond to IFN-\\u03b3 in vivo in a cell autonomous manner as they appear to in neuronal culture, or are they bystanders to glial reactions, and do they require steps that involve peripheral inflammation? Which IFN- \\u03b3 dependent signaling pathways do neurons and other cells of the brain parenchyma exhibit and are any of these associated with neurodegenerative disorders? In this study, we address these questions by analyzing neuronal and glial gene expression responses to IFN-\\u03b3 injected directly into the ventral midbrain of adult mice, thereby intentionally bypassing peripheral inflammatory signals. We find that MHC-I antigen processing and presentation genes are rapidly induced in all CNS cell types, including all of the major groups of midbrain neurons. Single nucleus RNA-sequencing (snRNA-seq) revealed numerous transcriptional responses that were restricted to glial cells, as well as microglia-specific expression of MHC-II antigen presentation genes. We show that dopamine neuronal responses to IFN-\\u03b3 are abolished by the selective deletion of the IFN-\\u03b3-binding domain of IFNGR1, indicating that neurons are capable of MHC-I upregulation via direct cell-autonomous innate immune signaling.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 6414, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_1\"}, \"text\": \"Results\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 6422, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Kinetics of transcriptional responses to IFN-\\u03b3 by midbrain neurons and glia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 6502, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We chose to study the effects of intraparenchymal injection of IFN-\\u03b3 to exclude effects of systemic inflammation or activation of the peripheral immune system, as previously reported. We first sought to measure the kinetics of IFN-\\u03b3 responses in ventral midbrain neurons and glia. We delivered either 400 nL of 0.9% saline or a high dose of IFN-\\u03b3 (40 ng in 0.9% saline) to the ventral midbrain of WT C57BL/6J mice bilaterally via stereotactic injection and sacrificed mice 6, 24, 48, and 72 h after injection (Fig. 1a). The high dose employed is likely to reach a concentration that is near or greater than the dissociation constant of of IFN-\\u03b3 for its cognate receptor but is lower than used in previous models of PD. After rapid tissue dissection, we prepared nuclei, conducted fluorescence-activated nuclear sorting (FANs) to separate neuronal nuclei (NeuN +) from other cell types (NeuN\\u2212), and analyzed the sorted nuclei by bulk RNA sequencing (RNA-Seq, Supp. Fig. 1a). We found no evidence of dopamine neuronal degeneration or toxicity, as demonstrated by stable expression of dopamine neuronspecific mRNAs Th, Slc6a3 (DAT), and Ddc in NeuN + nuclei up to 72 h post-injection (Supp. Fig. 1e).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 7716, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Binding of IFN-\\u03b3 dimers to the IFN-\\u03b3 receptor (IFNGR) leads to cytoplasmic activation of Janus kinases 1\\u20132, a cascade of tyrosine phosphorylation events, recruitment and phosphorylation of STAT1\\u03b1, and nuclear translocation of phospho-STAT1\\u03b1 dimers (pSTAT1, reviewed in). pSTAT1 binding at \\u03b3-activation site (GAS) elements leads to increased transcription of ISGs involved in antiviral and antibacterial immunity, including other transcription factors. The first wave of rapidly-induced genes includes IFN regulatory factor 1 (IRF-1), a transcription factor and master regulator of IFN-\\u03b3-induced gene expression that drives subsequent waves of transcription via binding to IFN-stimulated response elements (ISRE). Comparing IFN-\\u03b3 to saline at each timepoint, we found the highest number of differentially expressed genes (DEGs; DESeq2, \\u2223log2FC\\u2223 > 1, padj < 0.01) at 6 and 24 h for both neurons and glia (Fig. 1c). At 6 and 24 h, there were more than twice as many upregulated genes in glia compared to neurons, with considerable overlap in the upregulated genes (Fig. 1c). The number of DEGs in neurons and glia at 48 and 72 h was comparatively modest, consistent with the decay of IFN-\\u03b3 and/or negative feedback.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 8961, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Consistent with prior kinetic analyses, the induction of the ER peptide transporter mRNAs Tap1 and Tapbp was rapid, with peak expression at 6 h in both neurons and glia (Fig. 1d). MHC-I heavy chain (H2-K1, H2-D1) and light chain (B2m) genes do not harbor GAS elements, but rather are induced by IRF-1 binding to ISRE sequences in later waves of transcription. Accordingly, levels of H2-K1 and \\u03b22m mRNAs were highest at 24 h and remained ~ 8-fold elevated at 72 h (Fig. 1d). The rise and decay of all four mRNAs was markedly similar across neuronal and glial nuclei, although the amplitude of induction was higher in glia (~2\\u20138-fold higher expression relative to neurons at all timepoints, Fig. 1d).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 9663, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Of the DEGs upregulated by IFN-\\u03b3 in both neuronal and glial nuclei, 28 genes were upregulated at > 3 timepoints (Supp. Fig. 1d), indicating sustained upregulation across multiple cell types and central involvement in the IFN-\\u03b3 host defense program. Indeed, gene ontology (GO) analysis of these 28 genes identified top over-represented ontologies including \\u2018Cellular response to IFN-\\u03b3\\u2019, \\u2018MHC class I protein complex\\u2019, and \\u2018Antigen processing and presentation of exogenous peptide antigen via MHC class I, TAP-dependent\\u2019 (Supp. Fig. 1d). The induced genes included canonical IFN-inducible proteins such as MHC class I (H2-K1, H2-D1, B2m), immunoproteasome subunits (Psmb9/LMP2), ER peptide transporters (Tap1), guanylate binding proteins, and key regulators of IFN-\\u03b3-induced transcription (Stat1, Nlrc5) (Fig. 1e). Expression of these core IFN-\\u03b3-responsive genes was virtually absent in neuronal nuclei from saline-treated mice, and relative expression was uniformly higher in glia than in neurons (Fig. 1e). This phenomenon was true for most genes induced by IFN-\\u03b3 at all time points: while the magnitude of upregulation by IFN-\\u03b3 is well-correlated between neurons and glia (similar log2 fold change IFN-\\u03b3/Sal, x vs. y correlation in Fig. 1f), expression in glia was typically 2\\u20138-fold higher than in neurons (colorscale in Fig. 1f). These results demonstrate that the canonical IFN-\\u03b3 transcriptional response occurs in both neurons and glia with similar kinetics, but with lower amplitude in neurons.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 11200, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"IFN-\\u03b3 induces expression of T cell chemoattractant and CCR2 ligand chemokines\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 11282, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Given the role of chemokines in regulating leukocyte migration into the CNS, we analyzed the expression of mRNAs encoding chemokines. Several chemokines were expressed primarily in neurons and were unchanged after treatment with IFN-\\u03b3, including Cx3cl1/Fractalkine, Ccl25/TECK, and Ccl27a/CTACK (Supp. Fig. 2a, b). Neuronally derived CX3CL1/Fractalkine is known to modulate microglia via CX3CR1, while CCL25/TECK and CCL27/CTACK are primarily known for T cell homing in the gut and skin and to our knowledge have not been described in the CNS. In addition to well-known T cell chemoattractants (Cxcl9/MIG, Cxcl10/IP-10, Cxcl11/I-TAC), IFN-\\u03b3 also induced the expression of CCR2 ligands (Ccl2/MCP1, Ccl12/MCP-5) in both neurons and glia, with levels peaking at 6 h and declining rapidly thereafter (Supp. Fig. 2c). Consistent with prior studies, these results suggest that IFN-\\u03b3-exposed ventral midbrain neurons express chemokines that attract T cells and modulate phagocytic function of microglia/macrophages.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 12303, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Cell type-specific temporal responses to IFN-\\u03b3 in midbrain neurons and glia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 12383, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We next used a generalized linear model in DESeq2 to identify genes that are differentially regulated by IFN-\\u03b3 in neurons vs. glia (NeuN vs. Drug interaction, Supplementary Table 1, see Methods). Forty-two genes were upregulated in both neurons and glia, but the effect of IFN-\\u03b3 was greater in glia; these included canonical IFN-inducible genes such as Tap2 and Gbp2 (Fig. 2a-c, purple module). Most of the remaining DEGs from the interaction analysis were largely unchanged in neurons despite upregulation or downregulation by IFN-\\u03b3 in glia (Fig. 2a). These genes could be further separated into three groups based on the effect of IFN-\\u03b3 in glia and differences in baseline (saline) expression in neurons and glia.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 13115, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Twenty-seven genes were highly expressed in neurons at baseline, but not glia, and were upregulated by IFN-\\u03b3 only in glia (Fig. 2a-c, green module). These included the vesicular secretion regulator Cadps2 (CAPS2) and Socs2 (SOCS2), a negative feedback regulator of JAK-STAT signaling (Fig. 2c). Another 10 genes were also highly expressed in neurons at baseline, but not glia, but were rapidly downregulated by IFN-\\u03b3 only in glia; these included the enzyme dopa decarboxylase (Ddc) that is required for dopamine synthesis, and the endocytic modulator auxilin (Dnajc6) which is associated with rare cases of familial PD (Fig. 2a & 2c-d, orange module). These results suggest that glia, but not neurons, transiently suppress transcription of specific mRNAs involved in vesicular secretion and recycling after exposure to IFN-\\u03b3.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 13953, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Finally, 34 genes were highly expressed in glia at baseline, but not neurons, and were rapidly downregulated by IFN-\\u03b3 (Fig. 2a & 2c-d, yellow module). Notably, these included many microglia-enriched genes such as Csf1r, Siglech, and Dock8 (Fig. 2c). We further found that 23 of the top 50 microglial marker genes (identified via snRNA-seq data, see Fig. 3 below) were significantly downregulated at 6 h after IFN-\\u03b3 (Supp. Fig. 3). Thus, the microglial activation in response to IFN-\\u03b3 likely involves a transient repression of some, but not all, lineage marker genes.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 14532, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Conserved and cell-type-specific single cell responses to IFN-\\u03b3\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 14600, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"To resolve IFN-\\u03b3 responses at the single cell level, we next conducted snRNA-seq on ventral midbrain nuclei. To ensure proper coverage of neuronal subtypes, we used FANS as above to enrich NeuN + nuclei to ~ 70% of each mouse sample (n = 1 saline, n = 2 IFN-\\u03b3) prior to sequencing. Since the core IFN-\\u03b3-induced signature was stably upregulated for at least 48 h (Fig. 1), we selected 48 h for subsequent analysis in order to better approximate chronic in vivo exposure and avoid transient disruption of cell lineage marker expression (Fig. 2c-d). We processed the sequencing data and performed unsupervised clustering on the molecular count matrices as described previously. We identified thirty-eight clusters (Supp. Fig. 4a), which were collapsed to six major cell types with sufficient coverage for downstream analysis based on key marker genes significantly enriched in each cluster (Supp. Fig. 4b). After removal of nuclear doublets (Supp. Fig. 4c-d, see Methods), we retained a total of 4,699 nuclei, including 1,241 oligodendrocytes, 194 astrocytes, 134 microglia, 437 dopamine neurons, 1,257 GABAergic neurons, and 1,436 VGLUT2 neurons for downstream analysis (Fig. 3a), with comparable distribution across saline- and IFN-\\u03b3-treated mice (Supp. Fig. 5a).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 15879, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We previously found that IFN-\\u03b3-treated dopamine neurons express MHC-I in vitro, and baseline MHC-I in ventral midbrain dopamine neurons has been demonstrated in vivo, but dopamine neurons comprise a minority of all ventral midbrain neurons represented in our bulk NeuN + RNA-seq data (Fig. 1). We first analyzed the expression of a gene signature of MHC-I related genes (H2-K1, H2-D1, B2m, Tap1). In the absence of IFN-\\u03b3, only microglia expressed detectable levels of these genes, whereas all cell types could be induced to express the MHC-I gene signature (Fig. 3b). We previously reported that non-catecholaminergic neurons express MHC-I with higher concentrations of IFN-\\u03b3 in vitro, and consistently, we found that GABAergic, glutamatergic, and dopaminergic midbrain neurons all expressed similar levels of MHC-I genes after exposure to high dose IFN-\\u03b3 in vivo (Fig. 3b). Induced levels of MHC-I genes were similar between astrocytes and neurons, all of which were lower than in oligodendrocytes and microglia (Fig. 3b). Thus, all major cell types in the mouse ventral midbrain parenchyma can upregulate MHC-I mRNA upon exposure to IFN-\\u03b3.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 17041, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"In contrast to MHC-I, the expression of MHC-II is typically limited to professional antigen presenting cells. We found that the MHC-II genes (H2-Ab1, H2-Eb1, H2-Aa) and the invariant chain (Cd74) were highly expressed in IFN-\\u03b3-treated microglia, and no other cell types showed detectable MHC-II gene expression (Fig. 3b, Supp. Fig. 5c). Similarly, we found that IFN-\\u03b3 led to marked changes in the morphology of Iba1+ microglia, with associated cellular proliferation as detected by Ki67 protein expression and nuclear localization (Supp. Fig. 6). Ki67 staining was not present within any other cell types.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 17655, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We next conducted differential expression analysis, comparing each IFN-\\u03b3 vs. saline sample within each major cell type. Only genes differentially regulated by IFN-\\u03b3 in both samples were retained (see Methods; \\u2223log2FC\\u2223 > 3, FDR < 0.01), which comprised 108 DEGs across all six major cell types (Fig. 3c). Hierarchical clustering revealed that the most distinct expression profile for these genes belonged to IFN-\\u03b3-activated microglia, followed by oligodendrocytes and astrocytes, while the profiles of saline- and IFN-\\u03b3 -exposed neurons were more similar (Fig. 3c). We found that two genes, Atp9a and Ltbp3, were massively downregulated in all cell types (Supp. Fig. 5b). ATP9A is a type IV P-type ATPase involved in endosome recycling and exosome release, while LTBP-3 is a regulator of TGF-\\u03b2, although to our knowledge neither has been reported to possess an explicit immune function. Meanwhile, the majority of genes were upregulated by IFN-\\u03b3 and were highly expressed in glia, but not neurons (Fig. 3c). One notable exception to this trend was the expression of the non-classical MHC-I (class Ib) genes H2-Q5, H2-Q6, and H2-Q7, which were robustly induced by IFN-\\u03b3 in microglia and all neurons, but not oligodendrocytes and astrocytes (Fig. 3c). Class Ib molecules are structurally similar to class Ia (H2-D1, H2-K1), although class Ib genes tend to exhibit tissue-specific expression and distinct regulatory elements.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 19104, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"While all neurons upregulated classical MHC-I genes, we found that a number of canonical ISGs (e.g., Tap1, Tap2, Gbp2, Cd274/PD-L1) were poorly expressed in IFN-\\u03b3-treated neuronal nuclei at 48 h (Fig. 3c). Notable among these was Nlrc5, which encodes NLRC5, a member of the NOD-like receptor family that critically regulates MHC-I expression analogous to CIITA regulation of MHC-II expression. These results suggest that deficiency in effectors of IFN-\\u03b3-mediated signaling or transcription might lead to the dampened neuronal responses observed in our studies. Indeed, we found that several genes associated with the GO term \\u2018positive regulation of IFN-\\u03b3 mediated signaling pathway\\u2019 were robustly induced in glia, but not in neurons (Fig. 3d). These included poly(ADP-ribose) polymerase 9 (PARP9), which is known to activate STAT1 phosphorylation in activated macrophages, and immunity-related GTPases Irgm1-2 which regulate IFN-\\u03b3-driven effector responses to intracellular pathogens.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 20105, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"Within the ventral midbrain of PD patients, dopamine neurons of the ventral tegmental area (VTA) tend to be spared relative to more vulnerable populations in the SNc. Notably, we previously found HLA class I expression in dopamine neurons within the SNc, but not VTA, in postmortem tissue from PD patients. However, recent studies have demonstrated considerable transcriptomic heterogeneity amongst midbrain dopamine neurons. We thus leveraged our snRNA-seq data to study transcriptional responses to IFN-\\u03b3 amongst various subpopulations of dopamine neurons. We further analyzed dopamine neuronal expression profiles using unsupervised clustering as above (see Methods) and identified eight subclusters (Fig. 3e and Supp. Fig. 7). All subclusters expressed moderate to high levels of Slc6a3 (DAT) but were distinguished by expression of marker genes consistent with previous studies (Ndnf, Ntng1, Nefl, C1ql3, Cbln1, Cbln4, Tacr3), including those with ventral SN bias (Sox6, Aldh1a1, Grin2c, and Atp2a3) and those with medial VTA bias (Sorcs3, Nrp2, Grp, Calb1) (Fig. 3e, Supp. Fig. 7, and Supplementary Table 2). Dopamine neurons from saline- and IFN-\\u03b3-treated mice were distributed similarly throughout all subclusters (Fig. 3e), suggesting that IFN-\\u03b3 treatment did not significantly alter the expression of subcluster-defining marker genes. We found no evidence of differential responses to IFN-\\u03b3, with all subclusters upregulating classical (H2-K1, H2-D1, B2m) and non-classical MHC-I genes (H2-Q6, H2-Q7) (Fig. 3e). Thus, at least under conditions of direct exposure to high concentrations of IFN-\\u03b3, an MHC-I antigen presentation program can be induced in all populations of ventral midbrain dopamine neurons, suggesting that the presence of MHC-I alone is not sufficient to determine the subclusters targeted in parkinsonian neurodegeneration and that multiple hits may be required.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 22015, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"title_2\"}, \"text\": \"Dopamine neurons respond to IFN-\\u03b3 via cell autonomous IFNGR signaling\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 22089, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"In addition to IFN-\\u03b3, MHC-I heavy chain transcription can be induced in neurons by TNF-\\u03b1, and it is possible that the effects of IFN-\\u03b3 on neurons may be due to release of other cytokines or interactions with glia, particularly activated microglia. Furthermore, canonical ISGs can be upregulated in dopamine neurons without IFN-\\u03b3 under conditions of oxidative stress, such as high dose L-DOPA, 6-OHDA exposure, or a model of \\u03b1-synuclein aggregation. Thus, we first confirmed that dopamine neurons in the ventral tegmental area (VTA) and SNc express IFN-\\u03b3 receptor subunit 1 (Ifngr1) mRNA using fluorescence in situ hybridization (FISH) mRNA (Supp. Fig. 8). To test whether cell autonomous IFN-\\u03b3 signaling is required for their transcriptional responses, we selectively modified the Ifngr1 locus in dopamine neurons by crossing Ifngr1fl/fl mice to dopamine transporter (DAT)-cre mice. As previously described, Cre-mediated recombination in Ifngr1fl/fl mice leads to deletion of the IFN-\\u03b3-binding extracellular domain of IFNGR1 and conditional loss of IFN-\\u03b3 signaling.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 23191, \"infons\": {\"section_type\": \"RESULTS\", \"type\": \"paragraph\"}, \"text\": \"We first confirmed that DAT-Cre:Ifngr1fl/fl mice showed no gross alteration in midbrain or striatal TH and DAT immunolabel (Supp. Fig. 9). We then used immunohistochemistry to analyze phosphorylation of STAT1 Tyr701 (pSTAT1-Y701), the STAT1 residue critical for IFN-\\u03b3-mediated nuclear translocation and transcriptional activation, in the ventral midbrain at 6 h after injection of IFN-\\u03b3 or saline. IFN-\\u03b3-induced nuclear pSTAT1-Y701 was observed in TH+ dopamine neurons and TH\\u2212 cells in WT:Ifngr1fl/fl mice, but only in TH\\u2212 cells in DAT-Cre:Ifngr1fl/fl mice (Fig. 4a). Triple labeling with NeuN enabled analysis of nuclei from dopamine neurons (TH+/NeuN+), non-dopamine neurons (TH\\u2212/NeuN+), and glia (TH\\u2212/NeuN\\u2212) (Fig. 4b). Quantification of pSTAT1-Y701 induction in these populations confirmed a selective deficiency of IFN-\\u03b3 response in dopamine neurons (Fig. 4c). Notably, the average pSTAT1-Y701 intensity in glia was ~ 3-fold higher than in neurons, with no appreciable differences between dopamine neurons and non-dopamine neurons (Fig. 4b-c). Consistent with the pSTAT1-Y701 results, RNA-FISH confirmed that dopamine neurons in IFN-\\u03b3-treated DAT-Cre:Ifngr1fl/fl mice express significantly less Tap1 and MHC-I heavy chain mRNAs compared to WT:Ifngr1fl/fl mice (Fig. 4e-f). Collectively, these results confirm that cell autonomous IFNGR signaling and nuclear pSTAT1 translocation, not bystander responses to reactive glia, are responsible for transcription of antigen presentation genes within dopamine neurons following ventral midbrain exposure to IFN-\\u03b3.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 24775, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_1\"}, \"text\": \"Discussion\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 24786, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Interferons were discovered as factors with anti-viral activity and research over the following decades identified a large set of interferon-regulated pathways, including about 10% of the human genome. Due to reports of CD8 + and Th1 CD4 + T cells infiltrating the brain in aging and neurological disease, we characterized transcriptional changes caused by acute local exposure to the proinflammatory cytokine IFN-\\u03b3 in the ventral midbrain. Inflammation associated with neurodegenerative disorders may include cellular responses in the meninges and elsewhere in the periphery (Hobson et al., 2022c), which are likely to produce interacting effects. To avoid the confounding influence of other inflammatory cytokines, we designed our current approach to provide a molecular analysis of acute cellular changes in the absence of peripheral inflammation. To avoid the confounding influence of other inflammatory cytokines, we designed our current approach to provide a molecular analysis of acute cellular changes in the absence of peripheral inflammation.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 25843, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_2\"}, \"text\": \"Conserved responses to IFN-\\u03b3 in brain cells\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 25891, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Here, we focused on the brain parenchymal response to IFN-\\u03b3 and note that rare cell types in or at the border the CNS, including dendritic cells, monocytes, natural killer cells, and border associated macrophages were either too uncommon for detection or lost during brain dissection due to meningeal removal: these cell types may nevertheless play important roles in adaptive immune responses in the brain. Although MHC-I is expressed by most nucleated cells in the periphery, our results confirm previous reports that MHC-I mRNA is expressed at negligible levels in the healthy rodent CNS, with the exception of microglia (Fig. 3b). However, we find that for all major classes of CNS cells, IFN-\\u03b3 rapidly induces expression of MHC-I and other ISGs involved in antigen presentation, with similar kinetics to non-brain cells. Transcriptional kinetics were similar across CNS cell types, but ISG expression amplitude was highest in microglia, lower in astrocytes and oligodendrocytes, and lowest in neurons (Fig. 3). However, the abundance of neurons compared to microglia suggests that neuronal expression may still be important for the overall brain response to IFN-\\u03b3. For example, the IFN-\\u03b3-inducible T cell chemoattractants Cxcl9, Cxcl10, and Cxcl11 were rapidly upregulated in both neurons and glia (Supp. Fig. 2). Although our data suggest that microglia induce these genes at > 8-fold higher levels than neurons, neurons are typically > 5-fold more abundant than microglia in the human brain. Thus, it is likely that neurons, astrocytes, oligodendrocytes, and microglia each contribute to conserved IFN-\\u03b3 responses in the brain.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 27546, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_2\"}, \"text\": \"Dampened neuronal responses to IFN-\\u03b3\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 27587, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Why are neurons less responsive to IFN-\\u03b3 than glia? It has been suggested that the absence of MHC-I expression and lack of antigen presentation capacity may be protective for post-mitotic neurons. Our results underscore the longstanding view that MHC-I is not readily detectable in neurons, and highlight a growing appreciation that detection of physiologically relevant neuronal MHC-I requires highly sensitive techniques.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 28015, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Multiple mechanisms could underlie the dampened transcriptional response to IFN-\\u03b3 in neurons. First, we observed that the levels of nuclear pSTAT1 at 6 h post-IFN-\\u03b3 are significantly lower in neurons compared to glia (Fig. 4). Decreased abundance of IFNGR, STAT1, and/or negative feedback signaling that regulates STAT1-Y701 phosphorylation could all contribute to decreased nuclear pSTAT1 accumulation. Given the dramatic response in microglia compared to other glia, differential regulation of pSTAT1 is unlikely to be the sole mediator of cell type-specific IFN-\\u03b3 responses in the CNS. Other minor differences in ISG induction between cell types suggest subtle differences in transcriptional dynamics downstream of initial pSTAT1 binding: for example, the upregulation of non-classical MHC-I genes H2-Q5, H2-Q6, and H2-Q7 was specific to microglia and neurons (Fig. 3c). Thus, the overall lower amplitude of IFN-\\u03b3-induced transcription in neurons could be due in part to altered basal chromatin accessibility and/or inefficient expression of key transcription factors that drive downstream expression programs.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 29145, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_2\"}, \"text\": \"Microglia-specific responses to IFN-\\u03b3\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 29187, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Unexpectedly, we found an early, transient downregulation of some but not all microglial lineage-identity genes in NeuN\\u2212 nuclei at 6 h post-exposure to IFN-\\u03b3 (Supp. Fig. 3). These include Dock8, a protein implicated in controlling microglial function in neurodegeneration, the cytokine receptors Csf1r and Csf3r, and Siglech, a sialic acid-binding receptor that inhibits microglial activation. These responses may lead to reduced microglial sensitivity to anti-inflammatory and homeostatic mediators, as has been observed in IFN-\\u03b3-polarized macrophages. However, further studies are required to characterize the role of this transient transcriptional suppression within the microglial activation program induced by IFN-\\u03b3.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 29921, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Within the CNS, it is well-established that activated microglia express MHC-II, but prior studies of MHC-II expression in mouse oligodendrocytes and astrocytes have yielded conflicting results. In the context of MS, single cell RNA-seq in mouse models and human brain tissue identified subsets of mature oligodendrocytes that express IFN-inducible genes, including MHC-I and MHC-II genes, although only MHC-I associated signals were observed in human neuropathology. Our results suggest that acute stimulation with IFN-\\u03b3 elicits the transcriptional upregulation of MHC-II genes only in microglia (Fig. 3b), although more chronic exposure or the presence of other stimulatory factors may lead to expression in astrocytes and/or oligodendrocytes under certain conditions.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 30695, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"title_2\"}, \"text\": \"Dopamine neurons direcdy respond to IFN-\\u03b3\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 30741, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Several studies indicate that neuronal MHC-I presentation could be a response to intrinsic neuronal stress, even in culture paradigms lacking cell types that release IFN-\\u03b3, TNF-\\u03b1, or other candidate extrinsic triggers. In our hands, cultured postnatal DA neurons can be induced to present an intracellularly processed foreign antigen by MHC-I following high cytosolic oxidative stress caused by high levels of L-DOPA in the absence of microglia, T cells, or extrinsic cytokines. The hypothesis that neuronal stress may be an independent means to drive antigen presentation is consistent with reports of increased MHC-I in mature neurons by kainic acid, which indices excitotoxicity, and increased TAP-1 and immunoproteasome messages following the toxin, 6-hydroxy-dopamine and by aggregated alpha-synuclein. However, we found that dopamine neurons in DAT-Cre:Ifngr1fl/fl mice did not significantly upregulate pSTAT1 or MHC-I in response to IFN-\\u03b3 injection in the ventral midbrain (Fig. 4). These results suggest that the acute stress placed on dopaminergic neurons by 48 h of surrounding glial activation is insufficient to upregulate pSTAT1 via alternate pathways, although these may act at longer time scales.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 31965, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"Although IFN-\\u03b3 has been implicated in dopaminergic neuro-degeneration, studies in rodent and culture models of PD have focused on glial activation as the key driver of neuronal cell death. IFNGR1 and STAT1 knockout mice are protected from pathological overexpression of IFN-\\u03b3 in the mouse brain, but it is unknown which cells require IFN-\\u03b3 signaling to drive nigrostriatal degeneration in this model. Consistent with previous studies of cortical and hippocampal neurons, we found that midbrain dopaminergic neurons express Ifngr1 and upregulate MHC-I genes via increased nuclear pSTAT1-Y701 after IFN-\\u03b3 exposure (Fig. 4). Although we directly exposed dopaminergic neurons to IFN-\\u03b3 in the ventral midbrain, previous work found that IFN-\\u03b3 stimulation of axons can induce neuron MHC-I via retrograde signaling. Given that their massive axonal arbors comprise the majority of their surface area and protein mass, dopamine neurons may also be induced to express MHC-I via inflammation within the caudate and putamen in PD.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 33007, \"infons\": {\"section_type\": \"DISCUSS\", \"type\": \"paragraph\"}, \"text\": \"PD is widely considered to require multiple hits, including aging, and antigen presentation may play a role in some forms of neuron damage and death. Possible deleterious roles for IFN-\\u03b3 signaling and MHC-I expression within dopamine neurons in PD are suggested from mouse models of autoimmune-mediated narcolepsy, paraneoplastic encephalitis, and neurotropic viral infection. and demonstrated that neuronal expression of ectopic \\u201cneo-self-antigen\\u201d led to antigen-specific CD8 + T cell infiltration of the brain parenchyma and neuronal destruction. While found that viral antigen-specific CD8 + T cell-mediated neuronal destruction requires functional IFN-\\u03b3 receptor signaling in the CNS, constitutive neuronal overexpression of the MHC-I molecule H-2Db did not elicit pathology in the absence of CNS IFN-\\u03b3 receptor signaling. These results raised the possibility that neuronal upregulation of MHC-I is not critical for IFN-\\u03b3-mediated neuropathology in this model, but our results suggest that loss of neuronal IFNGR1 would completely prevent upregulation of the machinery necessary to process and present antigen on the constitutively expressed H-2Db molecules (e.g., B2m, Tap1, Psmb8). Nonetheless, subsequent studies found that IFN-\\u03b3-mediated synapse loss in virally infected neurons was driven by neuronal CCL2 expression and phagocyte-mediated synaptic removal. Thus, in addition to potential autoimmune-mediated responses directed at dopamine neurons, future work should investigate the synaptic and electrophysiological consequences of IFN-\\u03b3 signaling in the ventral midbrain.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34615, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_1\"}, \"text\": \"Materials and Methods\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34637, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Animals\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 34645, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Mice were housed on a 12-hour light/dark cycle with food and water available ad libitum. Adult male and female mice (6\\u201312 months old) were used in all experiments. DATIRES-Cre mice (JAX #006660, RRID: IMSR_JAX:006660) and Ifngr1fl mice (JAX #025394, RRID: IMSR_JAX:025394) were obtained from Jackson Laboratories (Bar Harbor, ME). DATIRES-Cre mice derive from C57BL/6J background, while Ifngr1fl mice derive from C57BL/6N background. All experiments were conducted according to NIH guidelines and approved by the Institutional Animal Care and Use Committees of Columbia University and the New York State Psychiatric Institute.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 35274, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"DATIRES-Cre:Ifngr1fl experimental litters were bred from a cross of DATIRES-Cre/wt;Ifngr1fl/fl \\u00d7 DATwt/wt;Ifngr1fl/fl mice such that Cre+ mice (DATIRES-Cre/wt;Ifngr1fl/fl) could be compared to Cre\\u2212 littermates with no modification of the Ifngr1 locus (DATwt/wt;Ifngr1fl/fl). To ensure proper allocation of genotypes to experimental treatments (stereotaxic injection of virus and/or IFN-\\u03b3), experimenters were not blinded to the genotype of mice during stereotaxic injection. However, a separate experimenter that was blind to the genotype and treatment condition conducted animal sacrifice and tissue dissection. All experimental procedures were conducted according to NIH guidelines and were approved by the Institutional Animal Care and Use Committees of Columbia University and the New York State Psychiatric Institute.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 36101, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Stereotaxic injections\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 36124, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"All surgical procedures were approved by the Institutional Animal Care and Use Committee and the Department of Comparative Medicine at New York State Psychiatric Institute. Mice were anesthetized with 4% isoflurane. Animals were transferred onto a Kopf Stereotaxic apparatus and maintained under isoflurane anesthesia (1\\u20132%). After hair removal and sterilization of the scalp using chlorhexidine and ethanol, a midline incision was made. Bregma and Lambda coordinates were determined, and minor adjustments in head position were made to match the DV coordinates. Saline or IFN-\\u03b3 was injected at AP \\u22123.2, ML \\u2013 0.9, and DV \\u2013 4.4. A small hole was drilled into the skull and 400 nL of 0.9% saline or 0.9% saline + 100 ng/\\u03bcL recombinant mouse IFN-\\u03b3 (40 ng IFN-\\u03b3 total) was injected through a pulled glass pipet using a Nanoject 2000 (Drummond Scientific; 8 pulses of 50 nL). At 5 min after injection, the glass pipet was slowly withdrawn over 5 min. After closing the skin with vicryl sutures, mice received 0.5 mL of 0.9% saline i.p. and were allowed to recover for > 1 h before being returned to their home cages.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 37249, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Antibodies and Reagents\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 37273, \"infons\": {\"file\": \"T1.xml\", \"id\": \"T1\", \"section_type\": \"TABLE\", \"type\": \"table\", \"xml\": \"<?xml version=\\\"1.0\\\" encoding=\\\"UTF-8\\\"?>\\n<table frame=\\\"void\\\" rules=\\\"groups\\\">\\n            <colgroup span=\\\"1\\\">\\n              <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n              <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n              <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n              <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n              <col align=\\\"left\\\" valign=\\\"middle\\\" span=\\\"1\\\"/>\\n            </colgroup>\\n            <thead>\\n              <tr>\\n                <th align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Name</th>\\n                <th align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Manufacturer</th>\\n                <th align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Catalog</th>\\n                <th align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID</th>\\n                <th align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Usage</th>\\n              </tr>\\n            </thead>\\n            <tbody>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Chicken anti-TH</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Millipore</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">AB9702</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_570923</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:500</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Mouse anti-NeuN</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Millipore</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">MAB377</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_2298772</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:500</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Rabbit anti-Iba1</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Wako</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">019\\u201319741</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_839504</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:1000</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Rat anti-Ki67</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ThermoFisher</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">14\\u20135698-82</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_10854564</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:1000</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Mouse anti-NeuN, Alexa488</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Millipore</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">MAB377X</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_2149209</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">FANS, 1:500</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Rabbit anti-pStat-Y701</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Cell Signaling</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">9167</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_561284</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:1000</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Rat anti-DAT</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Millipore</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">MAB369</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_2190413</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:500</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Goat anti-Chicken IgY (H + L), Alexa Fluor Plus 488</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ThermoFisher</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">A-32931TR</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_2866499</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:1000</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Goat anti-Mouse IgG (H + L), Alexa Fluor Plus 568</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ThermoFisher</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">A-11004</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_2534072</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:1000</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Goat anti-Rabbit IgG (H + L), Alexa Fluor 647</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ThermoFisher</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">A32733</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_2633282</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:1000</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Goat anti-Rat IgG (H + L), Alexa Fluor 647</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ThermoFisher</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">A-32931TR</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">RRID: AB_2866499</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">IHC, 1:1000</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Mm-Ifngr1-C2</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ACD Biotechne</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">441491-C2</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NA</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">FISH</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Mm-Tap1-C2</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ACD Biotechne</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">575041-C2</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NA</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">FISH</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Pan-MHC Class 1<xref rid=\\\"TFN2\\\" ref-type=\\\"table-fn\\\">**</xref></td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">ACD Biotechne</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NPR-0023100</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NA</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">FISH</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">TSA Cy3</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Perkin Elmer</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NEL744001KT</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NA</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">FISH, 1:1500</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">TSA Cy5</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Perkin Elmer</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NEL745001KT</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NA</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">FISH, 1:1500IHC, 1:7500</td>\\n              </tr>\\n              <tr>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">Recombinant Mouse IFN-gamma Protein</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">R&amp;D Systems (Biotechne)</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">485-MI-100/CF</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">NA</td>\\n                <td align=\\\"left\\\" valign=\\\"top\\\" rowspan=\\\"1\\\" colspan=\\\"1\\\">100 ng/\\u03bcL</td>\\n              </tr>\\n            </tbody>\\n          </table>\\n\"}, \"text\": \"Name\\tManufacturer\\tCatalog\\tRRID\\tUsage\\t \\tChicken anti-TH\\tMillipore\\tAB9702\\tRRID: AB_570923\\tIHC, 1:500\\t \\tMouse anti-NeuN\\tMillipore\\tMAB377\\tRRID: AB_2298772\\tIHC, 1:500\\t \\tRabbit anti-Iba1\\tWako\\t019\\u201319741\\tRRID: AB_839504\\tIHC, 1:1000\\t \\tRat anti-Ki67\\tThermoFisher\\t14\\u20135698-82\\tRRID: AB_10854564\\tIHC, 1:1000\\t \\tMouse anti-NeuN, Alexa488\\tMillipore\\tMAB377X\\tRRID: AB_2149209\\tFANS, 1:500\\t \\tRabbit anti-pStat-Y701\\tCell Signaling\\t9167\\tRRID: AB_561284\\tIHC, 1:1000\\t \\tRat anti-DAT\\tMillipore\\tMAB369\\tRRID: AB_2190413\\tIHC, 1:500\\t \\tGoat anti-Chicken IgY (H + L), Alexa Fluor Plus 488\\tThermoFisher\\tA-32931TR\\tRRID: AB_2866499\\tIHC, 1:1000\\t \\tGoat anti-Mouse IgG (H + L), Alexa Fluor Plus 568\\tThermoFisher\\tA-11004\\tRRID: AB_2534072\\tIHC, 1:1000\\t \\tGoat anti-Rabbit IgG (H + L), Alexa Fluor 647\\tThermoFisher\\tA32733\\tRRID: AB_2633282\\tIHC, 1:1000\\t \\tGoat anti-Rat IgG (H + L), Alexa Fluor 647\\tThermoFisher\\tA-32931TR\\tRRID: AB_2866499\\tIHC, 1:1000\\t \\tMm-Ifngr1-C2\\tACD Biotechne\\t441491-C2\\tNA\\tFISH\\t \\tMm-Tap1-C2\\tACD Biotechne\\t575041-C2\\tNA\\tFISH\\t \\tPan-MHC Class 1**\\tACD Biotechne\\tNPR-0023100\\tNA\\tFISH\\t \\tTSA Cy3\\tPerkin Elmer\\tNEL744001KT\\tNA\\tFISH, 1:1500\\t \\tTSA Cy5\\tPerkin Elmer\\tNEL745001KT\\tNA\\tFISH, 1:1500IHC, 1:7500\\t \\tRecombinant Mouse IFN-gamma Protein\\tR&D Systems (Biotechne)\\t485-MI-100/CF\\tNA\\t100 ng/\\u03bcL\\t \\t\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 38533, \"infons\": {\"file\": \"T1.xml\", \"id\": \"T1\", \"section_type\": \"TABLE\", \"type\": \"table_footnote\"}, \"text\": \"FANS: fluorescence-activated nuclear sorting, IHC: immunohistochemical staining of brain sections, FISH: Fluorescence in situ hybridization.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 38674, \"infons\": {\"file\": \"T1.xml\", \"id\": \"T1\", \"section_type\": \"TABLE\", \"type\": \"table_footnote\"}, \"text\": \"Due to extensive homology between MHC Class I gene family members, probes selectively targeting H2-K1 and/or H2-D1 were not feasible. The new probe employed was designed to maximally cross-react with H2-K1 mRNA (NCBI Reference Sequence, NM_001001892.2), although substantial cross-reactivity is predicted for H2-K2, H2-D1, H2-D4, H2-Q7, H2-Q9, H2-Q10, and Gm38417.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 39039, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Immunohistochemistry (IHC)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 39066, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Mice were anesthetized with euthasol and transcardially perfused with ~ 15 mL of 0.9% saline followed by 40\\u201350 mL of ice-cold 4% paraformaldehyde (PFA) in 0.1 M phosphate buffer (PB), pH 7.4. Brains were post-fixed in 4% PFA in 0.1 M PB for 6\\u201312 h at 4 \\u00b0C, washed three times in phosphate buffered saline (PBS), and sectioned at 50 \\u03bcm on a Leica VT1000S vibratome. Sections were placed in cryoprotectant solution (30% ethylene glycol, 30% glycerol, 0.1 M PB, pH 7.4) and stored at \\u221220 \\u00b0C until further use.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 39582, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Sections were removed from cryoprotectant solution and washed three times in tris-buffered saline (TBS) at room temperature. Except for pSTAT1 staining, sections were then permeabilized in TBS + 0.2% Triton-X 100 for one hour at room temperature, followed by blocking in TBS + 10% normal goat serum (NGS) and 0.1% Triton-X 100 for 1.5 h at room temperature. Sections for pSTAT1 staining were slide mounted, submerged in 10 mM sodium citrate (pH 6.0) at 50 \\u00b0C for 15 min, and washed several times in TBS. All sections were then transferred to or submerged in a pre-chilled solution containing primary antibodies in TBS + 2% NGS + 0.1% Triton-X 100 and incubated overnight at 4 \\u00b0C. Sections were washed in TBS + 0.05% Tween 20 (TBS + T) five times over an hour at room temperature. Sections were incubated in a solution containing secondary antibodies in TBS + 2% NGS + 0.1% Triton-X 100 at room temperature for 1.5 h, followed by four washes in TBS + T over 45 min at room temperature. Sections were slide mounted and cover-slipped with Fluoromount G (Southern Biotech). See Antibodies and Reagents for a complete list of antibodies and concentrations used in this study.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 40755, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Fluorescence in situ hybridization (FISH)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 40797, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"FISH was performed using the highly sensitive RNAScope\\u00ae Multiplex Fluorescent v2 assay (ACD Bio). See Antibodies and Reagents for complete list of probes and reagents used in this study. Although most single FISH puncta using this assay are likely single mRNA molecules, this cannot be definitively determined due to the enzymatic signal amplification and non-diffraction-limited size of the mRNA puncta.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 41203, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Mouse brain sections were prepared as above, removed from cryoprotectant solution, and washed three times in tris-buffered saline (TBS) at room temperature. Sections were incubated with hydrogen peroxide (ACD) for 15 min at room temperature, washed several times in TBS, and then mounted to Superfrost slides (Fisher). Sections were allowed to dry for 10 min and a hydrophobic barrier (PAP pen, Vector Labs) was created around the tissue. Tissue was incubated in 50% EtOH, then 70% EtOH, then 100% EtOH for 5 min each. Sections were rehydrated in TBS for several minutes, digested with Protease IV (ACD) for 25 min at room temperature, and rinsed twice with TBS before proceeding to the RNA Scope Multiplex Fluorescent v2 assay (ACD).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 41938, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"The RNA Scope Multiplex Fluorescent v2 assay was conducted according to the manufacturer\\u2019s instructions, with all incubations taking place in a humidified chamber at 40 \\u00b0C. Two 5-minute washes in excess RNA Scope Wash Buffer (ACD) took place between each incubation in sequential order: probes (2-hours), AMP1 (30 min), AMP2 (30 min), AMP3 (15 min), HRP-C1/2/3 (15 min), TSA Cy3 (30 min), HRP blocker (30 min), HRP-C1/2/3 (15 min), and TSA Cy5 (30 min). Samples were washed twice more in RNA Scope Wash Buffer, then twice more in TBS. Samples were then blocked and immunostained for tyrosine hydroxylase as described above. After immunostaining, samples were mounted in Fluoromount G and stored at 4 \\u00b0C for up to 1 week before imaging.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 42678, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Image acquisition and analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 42709, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"All imaging of 50 \\u03bcm sections from perfusion-fixed brain was conducted either on a Nikon Ti2 Eclipse epifluorescence microscope or on a Leica SP8 scanning confocal microscope using a 20x/0.75NA or 60x/1.4NA objective.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 42928, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"For pSTAT1-Y701 analysis, 5 \\u03bcm Z stacks from the SNc were acquired using a 20x/0.75NA objective and collapsed via maximum projection. Three types of DAPI+ nuclei were segmented as ROIs: TH+/NeuN+ (mDA neurons), TH\\u2212/NeuN+ (non mDA neurons), and glia (TH\\u2212/NeuN\\u2212). The mean pSTAT1 intensity for all pixels within each ROI was background subtracted using the median of the entire image.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43318, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"For FISH analysis, 5 \\u03bcm Z stacks from the SNc were acquired using a 20x/0.75NA objective and collapsed via maximum projection. For quantification of mRNA density in neuronal somata, a binary threshold was set for the TH immunofluorescence signal based on two standard deviations above the image background to generate a binary mask of pixels for TH+ neurons. Segmentation of mDA neuronal somata was conducted in ImageJ to generate ROIs for each neuronal soma, after which the total signal in both RNA channels was measured. The mean RNA intensity for all pixels within each soma ROI was background subtracted using the median of the entire image.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 43966, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Fluorescence-Activated nuclei sorting (FANS)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44011, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Nuclei were prepared from the ventral midbrain of mice at 48 h after injection of saline or IFN-\\u03b3. Mice were sacrificed by cervical dislocation and brains were rapidly extracted and submerged in ice-cold phosphate-buffered saline (PBS). Brains were placed on an ice-cold brain matrix (Zivic Instruments) and separated into 0.5\\u20131.0 mm sections using ice cold razor blades. Ventral midbrain tissue was dissected from slices between approximately \\u22122.5 mm to \\u22123.75 mm AP to Bregma. First, the cortex, hippocampi, and any hypothalamus or white matter ventral to the midbrain were removed. A single horizontal cut was made just dorsal to the rostral linear nucleus and all dorsal tissue was discarded. The remaining tissue containing the SN/VTA was flash frozen on liquid nitrogen and stored at \\u221280 \\u00b0C.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 44818, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Nuclei preparation and sorting were conducted as described by with only minor modifications. Frozen tissue was thawed in nuclear isolation medium 1 (NIM1: 250 mM sucrose, 25 mM KCl, 5 mM MgCl2, 10 mM Tris, pH 8.0) supplemented with 1 mM DTT, 0.2 U/\\u03bcL SUPERaseIN, 1x EDTA-free Protease Inhibitor Cocktail (Roche), and 0.1% Triton X-100. Tissue was homogenized on ice in a glass-glass dounce homogenizer with 30 gentle strokes of loose and tight clearance pestles. All subsequent purification steps were performed on ice or at 4 \\u00b0C unless otherwise specified. Lysates were spun at 500xg for 5 min, resuspended in NIM1 supplemented as above and also with 1% BSA, and passed through a 40 \\u03bcm cell strainer cap. Nuclei were centrifuged again at 500xg for 5 min and resuspended in staining buffer (PBS supplemented with 1% BSA, 5 mM MgCl2, 0.2 U/\\u03bcL SUPERaseIN, 1 \\u03bcg/mL propidium iodide, and 2 \\u03bcg/mL mouse anti-NeuN AlexaFluor488). After 30 min incubation on ice, nuclei were centrifuged at 500xg for 5 min and resuspended in 0.5 mL of FACS buffer (PBS + 0.04% BSA, 0.2 U/\\u03bcL SUPERaseIN, and 1 \\u03bcg/mL propidium iodide) for sorting.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 45948, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Nuclei sorting was conducted on a BD Influx (BD Biosciences) using a 100 \\u03bcm nozzle at 11.1 psi. Single nuclei were gated first using FSC-H vs. FSC-A and then on propidium iodide fluorescence. NeuN/AlexaFluor488 fluorescence was used to establish NeuN\\u2212 and NeuN+ gates on the single nuclei population. Nuclei were sorted directly into 96 well plates (for bulk nuclei RNA-seq with 96-well plate, pooled library construction) or into 1.5 mL Eppendorf tubes that had been previously coated with 0.1% BSA in PBS overnight at 37 \\u00b0C (for snRNA-seq).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46495, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Low input RNA sequencing with 96-well plate, pooled library construction\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 46568, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"The protocol for plate based, 3\\u2032 end unique molecular indicator (UMI)-based RNA sequencing of single cells has been described previously as well as modifications to accommodate ultra-low input samples. See Supplementary File 1 for sequences of all custom primers and oligonucleotides used in this protocol. For neuronal and glial nuclei analysis in Figs. 1 and 2, 100 NeuN\\u2212 or NeuN+ nuclei were sorted directly into the wells of a 96 well plate containing 3 \\u03bcL of nuclease-free water containing 1 U/\\u03bcL SUPERaseIN (sort volume ~ 3 \\u03bcL for total volume of ~ 6 \\u03bcL). After adding 1.5 \\u03bcL of 10 \\u03bcM barcoded RT primer (Integrated DNA Technologies), primer annealing was performed at 72 \\u00b0C for 3 min. Reverse transcription was performed by adding 7.5 \\u03bcL RT mix to each well (2.81 \\u03bcL of 40% polyethylene glycol 8000, 0.15 \\u03bcL of 100 mM dNTPs, 3 \\u03bcL of 5X Maxima H RT Buffer, 0.2 \\u03bcL of 200 U/\\u03bcL Maxima H Reverse Transcriptase (ThermoFisher), 0.2 \\u03bcL of 20 U/\\u03bcL SUPERaseIN, and 0.15 \\u03bcL of 100 \\u03bcM Template Switching Oligo (Integrated DNA Technologies), and 1 \\u03bcL of nuclease free water). Reverse transcription was performed at 42 \\u00b0C for 90 min, followed by 10 cycles of 50 \\u00b0C for 2 min, 42 \\u00b0C for 2 min, 75 \\u00b0C for 10 min, followed by a 4 \\u00b0C hold. Excess primers were removed by adding 2 \\u03bcL of Exonuclease I mix (1.875U ExoI in water) to each well and incubating at 37 \\u00b0C for 30 min, 85 \\u00b0C for 15 min, 75 \\u00b0C for 30 s, 4 \\u00b0C hold.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 48011, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"All wells were pooled into a single 15-ml falcon tubes and cDNA was purified and concentrated using Dynabeads\\u2122 MyOne\\u2122 Silane beads (ThermoFisher) according to the manufacturer\\u2019s instructions. The cDNA was split into duplicate reactions containing 25 \\u03bcL cDNA, 25 \\u03bcL 2x HIFI HotStart Ready Mix (Kapa Biosystems), and 0.2 M SMART PCR Primer. PCR was run as follows: 37 \\u00b0C for 30 min, 85 \\u00b0C for 15 min, 75 \\u00b0C for 30 s, 4 \\u00b0C hold. Duplicate reactions were combined and purified using 0.7 volumes AMPure XP beads (Beckman Coulter). The amplified cDNA was visualized on an Agilent 2100 Bioanalyzer and quantified using a Qubit II fluorometer (ThermoFisher).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 48674, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Sequencing libraries were constructed using Nextera XT (Illumina) with modifications. A custom i5 primer was used (NexteraPCR) with 0.6 ng input cDNA and 10 cycles of amplification was performed. Unique i7 indexes were used for each plate. After amplification, the library was purified with two rounds of AMPure XP beads, visualized on the Agilent 2100 Bioanalyzer and quantified using the Qubit II fluorometer. Libraries were sequenced on an Illumina NextSeq 500 using the 75 cycle High Output kit (read lengths 26(R1) \\u00d7 8(i) \\u00d7 58(R2)). Custom sequencing primers were used for Read 1 (SMRT_R1seq and ILMN_R1-seq, see Antibodies and Reagents). With each plate we targeted ~ 400 M reads. Library pools were loaded at 1.8 pM with 20% PhiX (Illumina).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 49425, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Reads were aligned to the mouse reference genome GRCm38 and transcriptome annotation (Gencode vM10) using the STAR aligner with parameters \\u2013sjdbOverhang 65 \\u2013twopassMode Basic after trimming poly (A)-tails from the 3\\u2032-ends. The aligned reads were demultiplexed using the well-identifying barcodes, correcting all single-nucleotide errors. All reads with the same well-identifying barcode, UMI, and gene mapping were collapsed to represent an individual transcript. To correct for sequencing errors in UMIs, we further collapsed UMIs that were within Hamming distance one of another UMI with the same well-identifying barcode and gene. For each 96-well plate, after generating a final list of individual transcripts with unique combinations of well-identifying barcodes, UMIs, and gene mapping, we produced a molecular count matrix for downstream analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 50286, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Bulk nuclei RNA-Seq differential expression analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 50339, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Analysis of bulk nuclei RNA-Seq UMI count matrices as shown in Figs. 1 and 2 (from Low input RNA Sequencing with 96-well plate, pooled library construction) was conducted using a generalized linear model (GLM) in DESeq2. For analysis of IFN-\\u03b3 vs. saline analyses, a single contrast was made for each subset of NeuN\\u2212 or NeuN + nuclei at each timepoint (DESeq2 formula: ~Treatment). For interaction analysis at each timepoint (DESeq2 formula: ~NeuN + Treatment + NeuN:Treatment), the contrast specifies the difference in log2 Fold Change for the effect of Treatment (IFN-\\u03b3 vs. saline) between the levels of NeuN (i.e., for NeuPos.IFN, the difference in IFN-\\u03b3 vs. saline between NeuN\\u2212 and NeuN + samples). Global analysis of NeuN\\u2212 vs. NeuN + nuclei was conducted across all samples while controlling for the effect of IFN-\\u03b3 (DESeq2 formula: ~Treatment + NeuN). Complete DESeq2 output for all analyses related to Fig. 1 and Fig. 2 is found in Supplementary Table 1.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51317, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Single nucleus RNA-sequencing (snRNA-seq)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51359, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"NeuN nuclear sorting was conducted as described above. Approximately 70% NeuN+ and 30% NeuN\\u2212 single nuclei were sorted per sample. Sorted nuclei were placed on ice and immediately submitted to the Single Cell Analysis Core (Columbia Genome Center) for 10X Genomics Chromium Single Cell 3\\u2032 v3 library preparation. The resulting libraries were sequenced on an Illumina NovaSeq 6000 with a targeted depth of > 100,000 2x100 bp paired-end reads per nucleus.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51817, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"snRNA-seq data Processing, Clustering and doublet filtering\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 51877, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"snRNA-seq data were processed as described previously (code available at: https://github.com/simslab/DropSeqPipeline8). Reads were aligned to the mouse reference genome GRCm38 and transcriptome annotation (Gencode vM10) using the STAR aligner with parameters \\u2013sjdbOverhang 65 \\u2013twopassMode Basic after trimming poly(A)-tails from the 3\\u2032-ends. All reads that uniquely aligned to a given gene body (including introns) were kept for downstream filtering analysis, because a large fraction of nuclear transcripts are unspliced. The aligned reads were demultiplexed using the cell-identifying barcodes, correcting all single-nucleotide errors. All reads with the same cell-identifying barcode, UMI, and gene mapping were collapsed to represent an individual transcript. To correct for sequencing errors in UMIs, we further collapsed UMIs that were within Hamming distance one of another UMI with the same cell-identifying barcode and gene. For each sample, after generating a final list of individual transcripts with unique combinations of cell-identifying barcodes, UMIs, and gene mapping, we produced a molecular count matrix for downstream analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 53031, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"We first performed unsupervised clustering on the count matrices using the PhenoGraph implementation of Louvain community detection after selection of highly variable genes and construction of a k-nearest neighbors graph as described previously. We identified 38 clusters (Supp. Fig. 4a), two of which showed statistically significant co-enrichment of dopamine neuron markers such as Th and Slc6a3 based on the binomial test for expression specificity. We identified six major cell types with sufficient coverage for downstream analysis as shown in the UMAP embedding in Fig. 3a. The assignment of PhenoGraph clusters into major cell type groupings was based on key marker genes significantly enriched in each cluster, as shown in Supp. Fig. 4b: Oligodendrocytes (Mal, Mag, Mog), Astrocytes (Gja1, Fgfr3, Atp1a2), Microglia (C1qb, Siglech, Csf1r), VGLUT2 neurons (Slc17a6, Cacna2d1, Ntng1), GABA neurons (Slc32a1, Gad1, Gad2), and DA neurons (Th, Slc6a3, En1, Slc10a4). Rare populations with insufficient cell numbers (e.g., mural cells, T cells) or ambiguous neuronal clusters possibly containing multiplets (e.g., Gad1+/Gad2+/Slc17a7+/Th+) were excluded from downstream analysis.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 54213, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"While examining the neuronal profiles, we identified small subpopulations of nuclei with co-enrichment of neuronal and oligodendrocyte markers such as Mbp, Plp1, Mog, and Mag. To filter out oligodendrocyte doublets, we fit Gaussian mixture models to the sum of expression for oligodendrocyte marker genes within each major neuronal class (see Supp. Fig. 4c). The top 99 oligodendrocyte marker genes were used, as determined by the binomial test (log2 specificity > 3, FDR < 10e-50). For all non-oligodendrocyte clusters, we then removed nuclei with expression > 8 standard deviations above the mean of the first Gaussian component corresponding to single neurons (Supp. Fig. 4d). This process was repeated twice more using the top 99 astrocyte and top 99 microglial marker genes to filter out potential doublets.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55026, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"snRNA-Seq differential expression analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 55069, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"For differential expression analysis shown in Fig. 3, we identified genes upregulated or downregulated by IFN-\\u03b3 within each cell type after filtering by conducting differential expression analysis as described in. Briefly, to perform differential expression analysis between two groups of cells, we randomly sub-sampled the data so that both groups are represented by the same number of cells. Next, we randomly sub-sampled the detected transcripts so that both groups have the same average number of transcripts per cell. Finally, we normalized the two sub-sampled count matrices using scran and analyzed differential expression for each gene using the SciPy implementation of the Mann-Whitney U test. We corrected the resulting p-values for false discovery using the Benjamini-Hochberg procedure as implemented in the statsmodels package in Python. We compared each IFN-\\u03b3 sample to the saline control and retained only genes with > 8-fold differential expression \\u2223log2FC > 3\\u2223 and FDR < 0.01 in both IFN-\\u03b3 samples. Complete differential expression output for all analyses related to Fig. 3 is reported in Supplementary Table 2.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56210, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Gene ontology (GO) analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56238, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"A single list of unique genes was used (i.e., differentially expressed genes from DESeq2 analysis) to conduct GO Analysis using web-based Enrichr with 2018 GO Terms for Cellular Component, Biological Process, and Molecular Function.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56471, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"title_2\"}, \"text\": \"Visualization and statistical analysis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 56510, \"infons\": {\"section_type\": \"METHODS\", \"type\": \"paragraph\"}, \"text\": \"Graphics were created in Adobe Illustrator 24.3 (Adobe, Inc.). Unless otherwise noted, all statistical analysis and data visualization was conducted in Python using SciPy, Matplotlib, and Seaborn packages. Statistical comparisons were conducted using the Mann-Whitney U test with number of replicates and other statistical testing information indicated in the figure captions. For visualization of RNA-seq abundance data, mRNA UMI counts were scaled up by 106 and normalized to total counts (CPM), and log2 transformed after adding 1. Normalized mRNA abundance is referred to as \\u2018log2(CPM + 1)\\u2019 as specified in figure captions. For clustered heatmaps, Z-scores of log2(CPM + 1) were first calculated using the zscore function within the SciPy Stats module, after which the row and column clustering was calculated using the linkage function (metric = \\u2018Euclidean\\u2019, method = \\u2018average\\u2019) within fastcluster 1.2.3 and passed to Seaborn clustermap.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57465, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"title_1\"}, \"text\": \"Supplementary Material\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57488, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"Data and materials availability\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57520, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"There are restrictions to the availability of custom-order RNA Scope probes generated for this study due to limited production size. The exact new probe design IDs are listed in Antibodies and Reagents and are available for further production from ACD Biotechne.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57783, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"All bulk nuclear RNA-seq and snRNA-seq data from this study are publicly available on the Gene Expression Omnibus as GSE218132. Count matrices used in this study are in Supplementary File 3 and Supplementary File 5.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 57999, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"Python and Shell code used for processing of RNA-seq data is accessible at: https://github.com/simslab/DropSeqPipeline8, and Python code used for clustering and visualization of scRNA-seq data can be found at https://www.github.com/simslab/cluster_diffex2018.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58259, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"Appendices A and B. Supplementary data\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58298, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"Supplementary data to this article can be found online at https://doi.org/10.1016/j.bbi.2023.04.008.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58399, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"Declaration of Competing Interest\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58433, \"infons\": {\"section_type\": \"SUPPL\", \"type\": \"footnote\"}, \"text\": \"The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58604, \"infons\": {\"section_type\": \"REF\", \"type\": \"title\"}, \"text\": \"References\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58615, \"infons\": {\"fpage\": \"151\", \"lpage\": \"166\", \"name_0\": \"surname:Amor;given-names:S\", \"name_1\": \"surname:Peferoen;given-names:LAN\", \"name_2\": \"surname:Vogel;given-names:DYS\", \"name_3\": \"surname:Breur;given-names:M\", \"name_4\": \"surname:van der Valk;given-names:P\", \"name_5\": \"surname:Baker;given-names:D\", \"name_6\": \"surname:van Noort;given-names:JM\", \"pub-id_doi\": \"10.1111/imm.12233\", \"pub-id_pmid\": \"24329535\", \"section_type\": \"REF\", \"source\": \"Immunology\", \"type\": \"ref\", \"volume\": \"142\", \"year\": \"2014\"}, \"text\": \"Inflammation in neurodegenerative diseases\\u2013an update\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58670, \"infons\": {\"fpage\": \"25\", \"lpage\": \"29\", \"name_0\": \"surname:Ashburner;given-names:M\", \"name_1\": \"surname:Ball;given-names:CA\", \"name_10\": \"surname:Harris;given-names:MA\", \"name_11\": \"surname:Hill;given-names:DP\", \"name_12\": \"surname:Issel-Tarver;given-names:L\", \"name_13\": \"surname:Kasarskis;given-names:A\", \"name_14\": \"surname:Lewis;given-names:S\", \"name_15\": \"surname:Matese;given-names:JC\", \"name_16\": \"surname:Richardson;given-names:JE\", \"name_17\": \"surname:Ringwald;given-names:M\", \"name_18\": \"surname:Rubin;given-names:GM\", \"name_19\": \"surname:Sherlock;given-names:G\", \"name_2\": \"surname:Blake;given-names:JA\", \"name_3\": \"surname:Botstein;given-names:D\", \"name_4\": \"surname:Butler;given-names:H\", \"name_5\": \"surname:Cherry;given-names:JM\", \"name_6\": \"surname:Davis;given-names:AP\", \"name_7\": \"surname:Dolinski;given-names:K\", \"name_8\": \"surname:Dwight;given-names:SS\", \"name_9\": \"surname:Eppig;given-names:JT\", \"pub-id_doi\": \"10.1038/75556\", \"pub-id_pmid\": \"10802651\", \"section_type\": \"REF\", \"source\": \"Nat. Genet\", \"type\": \"ref\", \"volume\": \"25\", \"year\": \"2000\"}, \"text\": \"Gene Ontology: tool for the unification of biology\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58721, \"infons\": {\"fpage\": \"7922\", \"issue\": \"21\", \"lpage\": \"7930\", \"name_0\": \"surname:Babcock;given-names:AA\", \"name_1\": \"surname:Kuziel;given-names:WA\", \"name_2\": \"surname:Rivest;given-names:S\", \"name_3\": \"surname:Owens;given-names:T\", \"pub-id_pmid\": \"12944523\", \"section_type\": \"REF\", \"source\": \"J. Neurosci\", \"type\": \"ref\", \"volume\": \"23\", \"year\": \"2003\"}, \"text\": \"Chemokine expression by glial cells directs leukocytes to sites of axonal injury in the CNS\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58813, \"infons\": {\"fpage\": \"383\", \"lpage\": \"390\", \"name_0\": \"surname:B\\u00e4ckman;given-names:CM\", \"name_1\": \"surname:Malik;given-names:N\", \"name_2\": \"surname:Zhang;given-names:Y\", \"name_3\": \"surname:Shan;given-names:L\", \"name_4\": \"surname:Grinberg;given-names:A\", \"name_5\": \"surname:Hoffer;given-names:BJ\", \"name_6\": \"surname:Westphal;given-names:H\", \"name_7\": \"surname:Tomac;given-names:AC\", \"pub-id_doi\": \"10.1002/dvg.20228\", \"pub-id_pmid\": \"16865686\", \"section_type\": \"REF\", \"source\": \"Genesis\", \"type\": \"ref\", \"volume\": \"44\", \"year\": \"2006\"}, \"text\": \"Characterization of a mouse strain expressing Cre recombinase from the 3\\u2019 untranslated region of the dopamine transporter locus\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 58943, \"infons\": {\"fpage\": \"e142\", \"name_0\": \"surname:Barcia;given-names:C\", \"name_1\": \"surname:Ros;given-names:CM\", \"name_2\": \"surname:Annese;given-names:V\", \"name_3\": \"surname:G\\u00f3mez;given-names:A\", \"name_4\": \"surname:Ros-Bernal;given-names:F\", \"name_5\": \"surname:Aguado-Yera;given-names:D\", \"name_6\": \"surname:Mart\\u00ednez-Pag\\u00e1n;given-names:ME\", \"name_7\": \"surname:de Pablos;given-names:V\", \"name_8\": \"surname:Fernandez-Villalba;given-names:E\", \"name_9\": \"surname:Herrero;given-names:MT\", \"pub-id_doi\": \"10.1038/cddis.2011.17\", \"pub-id_pmid\": \"21472005\", \"section_type\": \"REF\", \"source\": \"Cell Death Dis\", \"type\": \"ref\", \"volume\": \"2\", \"year\": \"2011\"}, \"text\": \"IFN-\\u03b3 signaling, with the synergistic contribution of TNF-\\u03b1, mediates cell specific microglial and astroglial activation in experimental models of Parkinson\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59118, \"infons\": {\"fpage\": \"38\", \"issue\": \"1\", \"lpage\": \"44\", \"name_0\": \"surname:Becht;given-names:E\", \"name_1\": \"surname:McInnes;given-names:L\", \"name_2\": \"surname:Healy;given-names:J\", \"name_3\": \"surname:Dutertre;given-names:C-A\", \"name_4\": \"surname:Kwok;given-names:IWH\", \"name_5\": \"surname:Ng;given-names:LG\", \"name_6\": \"surname:Ginhoux;given-names:F\", \"name_7\": \"surname:Newell;given-names:EW\", \"section_type\": \"REF\", \"source\": \"Nat. Biotechnol\", \"type\": \"ref\", \"volume\": \"37\", \"year\": \"2019\"}, \"text\": \"Dimensionality reduction for visualizing single-cell data using UMAP\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59187, \"infons\": {\"fpage\": \"10956\", \"lpage\": \"10961\", \"name_0\": \"surname:Bernard-Valnet;given-names:R\", \"name_1\": \"surname:Yshii;given-names:L\", \"name_10\": \"surname:Pignolet;given-names:B\", \"name_11\": \"surname:Dauvilliers;given-names:Y\", \"name_12\": \"surname:Peyron;given-names:C\", \"name_13\": \"surname:Liblau;given-names:RS\", \"name_2\": \"surname:Qu\\u00e9riauit;given-names:C\", \"name_3\": \"surname:Nguyen;given-names:X-H\", \"name_4\": \"surname:Arthaud;given-names:S\", \"name_5\": \"surname:Rodrigues;given-names:M\", \"name_6\": \"surname:Canivet;given-names:A\", \"name_7\": \"surname:Morel;given-names:A-L\", \"name_8\": \"surname:Matthys;given-names:A\", \"name_9\": \"surname:Bauer;given-names:J\", \"pub-id_doi\": \"10.1073/pnas.1603325113\", \"pub-id_pmid\": \"27621438\", \"section_type\": \"REF\", \"source\": \"PNAS\", \"type\": \"ref\", \"volume\": \"113\", \"year\": \"2016\"}, \"text\": \"CD8 T cell-mediated killing of orexinergic neurons induces a narcolepsy-like phenotype in mice\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59282, \"infons\": {\"fpage\": \"182\", \"lpage\": \"192\", \"name_0\": \"surname:Brochard;given-names:V\", \"name_1\": \"surname:Combadi\\u00e8re;given-names:B\", \"name_10\": \"surname:Duyckaerts;given-names:C\", \"name_11\": \"surname:Flavell;given-names:RA\", \"name_12\": \"surname:Hirsch;given-names:EC\", \"name_13\": \"surname:Hunot;given-names:S\", \"name_2\": \"surname:Prigent;given-names:A\", \"name_3\": \"surname:Laouar;given-names:Y\", \"name_4\": \"surname:Perrin;given-names:A\", \"name_5\": \"surname:Beray-Berthat;given-names:V\", \"name_6\": \"surname:Bonduelle;given-names:O\", \"name_7\": \"surname:Alvarez-Fischer;given-names:D\", \"name_8\": \"surname:Callebert;given-names:J\", \"name_9\": \"surname:Launay;given-names:J-M\", \"pub-id_doi\": \"10.1172/JCI36470\", \"pub-id_pmid\": \"19104149\", \"section_type\": \"REF\", \"source\": \"J. clin. Invest\", \"type\": \"ref\", \"volume\": \"119\", \"year\": \"2009\"}, \"text\": \"Infiltration of CD4+ lymphocytes into the brain contributes to neurodegeneration in a mouse model of Parkinson disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59401, \"infons\": {\"fpage\": \"150\", \"lpage\": \"157\", \"name_0\": \"surname:Casta\\u00f1o;given-names:A\", \"name_1\": \"surname:Herrera;given-names:AJ\", \"name_2\": \"surname:Cano;given-names:J\", \"name_3\": \"surname:Machado;given-names:A\", \"pub-id_doi\": \"10.1046/j.1471-4159.2002.00799.x\", \"pub-id_pmid\": \"12067227\", \"section_type\": \"REF\", \"source\": \"J. Neurochem\", \"type\": \"ref\", \"volume\": \"81\", \"year\": \"2002\"}, \"text\": \"The degenerative effect of a single intranigral injection of LPS on the dopaminergic system is prevented by dexamethasone, and not mimicked by rh-TNF-alpha, IL-1beta and IFN-gamma\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59581, \"infons\": {\"fpage\": \"3633\", \"name_0\": \"surname:Cebri\\u00e1n;given-names:C\", \"name_1\": \"surname:Zucca;given-names:FA\", \"name_10\": \"surname:Zecca;given-names:L\", \"name_11\": \"surname:Loike;given-names:JD\", \"name_12\": \"surname:Sulzer;given-names:D\", \"name_2\": \"surname:Mauri;given-names:P\", \"name_3\": \"surname:Steinbeck;given-names:JA\", \"name_4\": \"surname:Studer;given-names:L\", \"name_5\": \"surname:Scherzer;given-names:CR\", \"name_6\": \"surname:Kanter;given-names:E\", \"name_7\": \"surname:Budhu;given-names:S\", \"name_8\": \"surname:Mandelbaum;given-names:J\", \"name_9\": \"surname:Vonsattel;given-names:JP\", \"pub-id_doi\": \"10.1038/ncomms4633\", \"pub-id_pmid\": \"24736453\", \"section_type\": \"REF\", \"source\": \"Nat. Commun\", \"type\": \"ref\", \"volume\": \"5\", \"year\": \"2014\"}, \"text\": \"MHC-I expression renders catecholaminergic neurons susceptible to T-cell-mediated degeneration\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59676, \"infons\": {\"fpage\": \"694\", \"lpage\": \"696\", \"name_0\": \"surname:Chakrabarty;given-names:P\", \"name_1\": \"surname:Ceballos-Diaz;given-names:C\", \"name_2\": \"surname:Lin;given-names:W-L\", \"name_3\": \"surname:Beccard;given-names:A\", \"name_4\": \"surname:Jansen-West;given-names:K\", \"name_5\": \"surname:McFarland;given-names:NR\", \"name_6\": \"surname:Janus;given-names:C\", \"name_7\": \"surname:Dickson;given-names:D\", \"name_8\": \"surname:Das;given-names:P\", \"name_9\": \"surname:Golde;given-names:TE\", \"pub-id_doi\": \"10.1038/nn.2829\", \"pub-id_pmid\": \"21572432\", \"section_type\": \"REF\", \"source\": \"Nat. Neurosci\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2011\"}, \"text\": \"Interferon-\\u03b3 induces progressive nigrostriatal degeneration and basal ganglia calcification\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59772, \"infons\": {\"fpage\": \"e0151569\", \"issue\": \"3\", \"name_0\": \"surname:Chien;given-names:C-H\", \"name_1\": \"surname:Lee;given-names:M-J\", \"name_2\": \"surname:Liou;given-names:H-C\", \"name_3\": \"surname:Liou;given-names:H-H\", \"name_4\": \"surname:Fu;given-names:W-M\", \"name_5\": \"surname:Block;given-names:ML\", \"pub-id_pmid\": \"26982707\", \"section_type\": \"REF\", \"source\": \"PLoS One\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2016\"}, \"text\": \"Microglia-derived cytokines/chemokines are involved in the enhancement of LPS-induced loss of nigrostriatal dopaminergic neurons in DJ-1 knockout mice\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 59923, \"infons\": {\"fpage\": \"1709\", \"lpage\": \"1725\", \"name_0\": \"surname:Chung;given-names:CY\", \"name_1\": \"surname:Seo;given-names:H\", \"name_2\": \"surname:Sonntag;given-names:KC\", \"name_3\": \"surname:Brooks;given-names:A\", \"name_4\": \"surname:Lin;given-names:L\", \"name_5\": \"surname:Isacson;given-names:O\", \"pub-id_doi\": \"10.1093/hmg/ddi178\", \"pub-id_pmid\": \"15888489\", \"section_type\": \"REF\", \"source\": \"Hum. Mol. Genet\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2005\"}, \"text\": \"Cell type-specific gene expression of midbrain dopaminergic neurons reveals molecules involved in their vulnerability and protection\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60056, \"infons\": {\"fpage\": \"172\", \"lpage\": \"185\", \"name_0\": \"surname:Clarkson;given-names:BDS\", \"name_1\": \"surname:Patel;given-names:MS\", \"name_2\": \"surname:LaFrance-Corey;given-names:RG\", \"name_3\": \"surname:Howe;given-names:CL\", \"pub-id_doi\": \"10.1002/acn3.516\", \"pub-id_pmid\": \"29468178\", \"section_type\": \"REF\", \"source\": \"Ann. Clin. Transl. Neurol\", \"type\": \"ref\", \"volume\": \"5\", \"year\": \"2018\"}, \"text\": \"Retrograde interferon-gamma signaling induces major histocompatibility class I expression in human-induced pluripotent stem cell-derived neurons\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60201, \"infons\": {\"fpage\": \"505\", \"lpage\": \"520\", \"name_0\": \"surname:Corriveau;given-names:RA\", \"name_1\": \"surname:Huh;given-names:GS\", \"name_2\": \"surname:Shatz;given-names:CJ\", \"pub-id_doi\": \"10.1016/S0896-6273(00)80562-0\", \"pub-id_pmid\": \"9768838\", \"section_type\": \"REF\", \"source\": \"Neuron\", \"type\": \"ref\", \"volume\": \"21\", \"year\": \"1998\"}, \"text\": \"Regulation of class I MHC gene expression in the developing and mature CNS by neural activity\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60295, \"infons\": {\"fpage\": \"1\", \"lpage\": \"10\", \"name_0\": \"surname:Dhanwani;given-names:R\", \"name_1\": \"surname:Lima-Junior;given-names:JR\", \"name_10\": \"surname:Litvan;given-names:I\", \"name_11\": \"surname:Alcalay;given-names:RN\", \"name_12\": \"surname:Peters;given-names:B\", \"name_13\": \"surname:Sulzer;given-names:D\", \"name_14\": \"surname:Arlehamn;given-names:CSL\", \"name_15\": \"surname:Sette;given-names:A\", \"name_2\": \"surname:Sethi;given-names:A\", \"name_3\": \"surname:Pham;given-names:J\", \"name_4\": \"surname:Williams;given-names:G\", \"name_5\": \"surname:Frazier;given-names:A\", \"name_6\": \"surname:Xu;given-names:Y\", \"name_7\": \"surname:Amara;given-names:AW\", \"name_8\": \"surname:Standaert;given-names:DG\", \"name_9\": \"surname:Goldman;given-names:JG\", \"pub-id_doi\": \"10.1038/s41531-022-00282-2\", \"pub-id_pmid\": \"35013321\", \"section_type\": \"REF\", \"source\": \"NPJ Parkinsons Dis\", \"type\": \"ref\", \"volume\": \"8\", \"year\": \"2022\"}, \"text\": \"Transcriptional analysis of peripheral memory T cells reveals Parkinson\\u2019s disease-specific gene signatures\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60404, \"infons\": {\"fpage\": \"458\", \"lpage\": \"471.e19\", \"name_0\": \"surname:Di Liberto;given-names:G\", \"name_1\": \"surname:Pantelyushin;given-names:S\", \"name_10\": \"surname:Salinas;given-names:G\", \"name_11\": \"surname:Lin-Marq;given-names:N\", \"name_12\": \"surname:Staszewski;given-names:O\", \"name_13\": \"surname:Costa Jord\\u00e3o;given-names:MJ\", \"name_14\": \"surname:Wagner;given-names:I\", \"name_15\": \"surname:Egervari;given-names:K\", \"name_16\": \"surname:Mack;given-names:M\", \"name_17\": \"surname:Bellone;given-names:C\", \"name_18\": \"surname:Bl\\u00fcmcke;given-names:I\", \"name_19\": \"surname:Prinz;given-names:M\", \"name_2\": \"surname:Kreutzfeldt;given-names:M\", \"name_20\": \"surname:Pinschewer;given-names:DD\", \"name_21\": \"surname:Merkler;given-names:D\", \"name_3\": \"surname:Page;given-names:N\", \"name_4\": \"surname:Musardo;given-names:S\", \"name_5\": \"surname:Coras;given-names:R\", \"name_6\": \"surname:Steinbach;given-names:K\", \"name_7\": \"surname:Vincenti;given-names:I\", \"name_8\": \"surname:Klimek;given-names:B\", \"name_9\": \"surname:Lingner;given-names:T\", \"pub-id_doi\": \"10.1016/j.cell.2018.07.049\", \"pub-id_pmid\": \"30173917\", \"section_type\": \"REF\", \"source\": \"Cell\", \"type\": \"ref\", \"volume\": \"175\", \"year\": \"2018\"}, \"text\": \"Neurons under T cell attack coordinate phagocyte-mediated synaptic stripping\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60481, \"infons\": {\"fpage\": \"205\", \"lpage\": \"210\", \"name_0\": \"surname:Dulken;given-names:BW\", \"name_1\": \"surname:Buckley;given-names:MT\", \"name_10\": \"surname:Wyss-Coray;given-names:T\", \"name_11\": \"surname:Weissman;given-names:IL\", \"name_12\": \"surname:Vogel;given-names:H\", \"name_13\": \"surname:Davis;given-names:MM\", \"name_14\": \"surname:Brunet;given-names:A\", \"name_2\": \"surname:Navarro Negredo;given-names:P\", \"name_3\": \"surname:Saligrama;given-names:N\", \"name_4\": \"surname:Cayrol;given-names:R\", \"name_5\": \"surname:Leeman;given-names:DS\", \"name_6\": \"surname:George;given-names:BM\", \"name_7\": \"surname:Boutet;given-names:SC\", \"name_8\": \"surname:Hebestreit;given-names:K\", \"name_9\": \"surname:Pluvinage;given-names:JV\", \"pub-id_doi\": \"10.1038/S41586-019-1362-5\", \"pub-id_pmid\": \"31270459\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"571\", \"year\": \"2019\"}, \"text\": \"Single-cell analysis reveals T cell infiltration in old neurogenic niches\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60555, \"infons\": {\"fpage\": \"1837\", \"issue\": \"12\", \"lpage\": \"1844\", \"name_0\": \"surname:Falc\\u00e3o;given-names:AM\", \"name_1\": \"surname:van Bruggen;given-names:D\", \"name_10\": \"surname:Williams;given-names:A\", \"name_11\": \"surname:Guerreiro-Cacais;given-names:AO\", \"name_12\": \"surname:Castelo-Branco;given-names:G\", \"name_2\": \"surname:Marques;given-names:S\", \"name_3\": \"surname:Meijer;given-names:M\", \"name_4\": \"surname:J\\u00e4kel;given-names:S\", \"name_5\": \"surname:Agirre;given-names:E\", \"name_6\": \"surname:Samudyata\", \"name_7\": \"surname:Floriddia;given-names:EM\", \"name_8\": \"surname:Vanichkina;given-names:DP\", \"name_9\": \"surname:ffrench-Constant;given-names:C\", \"pub-id_pmid\": \"30420755\", \"section_type\": \"REF\", \"source\": \"Nat. Med\", \"type\": \"ref\", \"volume\": \"24\", \"year\": \"2018\"}, \"text\": \"Disease-specific oligodendrocyte lineage cells arise in multiple sclerosis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60630, \"infons\": {\"fpage\": \"705\", \"lpage\": \"719\", \"name_0\": \"surname:Ferrari;given-names:DP\", \"name_1\": \"surname:Bortolanza;given-names:M\", \"name_2\": \"surname:Del Bel;given-names:EA\", \"pub-id_doi\": \"10.1007/s12640-021-00345-x\", \"pub-id_pmid\": \"33687725\", \"section_type\": \"REF\", \"source\": \"Neurotox. Res\", \"type\": \"ref\", \"volume\": \"39\", \"year\": \"2021\"}, \"text\": \"Interferon-\\u03b3 involvement in the neuroinflammation associated with Parkinson\\u2019s disease and L-DOPA-induced dyskinesia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60750, \"infons\": {\"fpage\": \"425\", \"lpage\": \"429\", \"name_0\": \"surname:Filiano;given-names:AJ\", \"name_1\": \"surname:Xu;given-names:Y\", \"name_10\": \"surname:Peerzade;given-names:SN\", \"name_11\": \"surname:Chen;given-names:H\", \"name_12\": \"surname:Lee;given-names:KS\", \"name_13\": \"surname:Scott;given-names:MM\", \"name_14\": \"surname:Beenhakker;given-names:MP\", \"name_15\": \"surname:Litvak;given-names:V\", \"name_16\": \"surname:Kipnis;given-names:J\", \"name_2\": \"surname:Tustison;given-names:NJ\", \"name_3\": \"surname:Marsh;given-names:RL\", \"name_4\": \"surname:Baker;given-names:W\", \"name_5\": \"surname:Smirnov;given-names:I\", \"name_6\": \"surname:Overall;given-names:CC\", \"name_7\": \"surname:Gadani;given-names:SP\", \"name_8\": \"surname:Turner;given-names:SD\", \"name_9\": \"surname:Weng;given-names:Z\", \"pub-id_doi\": \"10.1038/nature18626\", \"pub-id_pmid\": \"27409813\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"535\", \"year\": \"2016\"}, \"text\": \"Unexpected role of interferon-\\u03b3 in regulating neuronal connectivity and social behavior\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60842, \"infons\": {\"fpage\": \"577050\", \"name_0\": \"surname:Flood;given-names:L\", \"name_1\": \"surname:Korol;given-names:SV\", \"name_2\": \"surname:Ekselius;given-names:L\", \"name_3\": \"surname:Birnir;given-names:B\", \"name_4\": \"surname:Jin;given-names:Z\", \"pub-id_doi\": \"10.1016/j.jneuroim.2019.577050\", \"pub-id_pmid\": \"31505410\", \"section_type\": \"REF\", \"source\": \"J. Neuroimmunol\", \"type\": \"ref\", \"volume\": \"337\", \"year\": \"2019\"}, \"text\": \"Interferon-\\u03b3 potentiates GABAA receptor-mediated inhibitory currents in rat hippocampal CA1 pyramidal neurons\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 60956, \"infons\": {\"fpage\": \"3717\", \"lpage\": \"3733\", \"name_0\": \"surname:Galiano-Landeira;given-names:J\", \"name_1\": \"surname:Torra;given-names:A\", \"name_2\": \"surname:Vila;given-names:M\", \"name_3\": \"surname:Bov\\u00e9;given-names:J\", \"pub-id_doi\": \"10.1093/brain/awaa269\", \"pub-id_pmid\": \"33118032\", \"section_type\": \"REF\", \"source\": \"Brain\", \"type\": \"ref\", \"volume\": \"143\", \"year\": \"2020\"}, \"text\": \"CD8 T cell nigral infiltration precedes synucleinopathy in early stages of Parkinson\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61053, \"infons\": {\"fpage\": \"439\", \"lpage\": \"455\", \"name_0\": \"surname:Garretti;given-names:F\", \"name_1\": \"surname:Monahan;given-names:C\", \"name_2\": \"surname:Sette;given-names:A\", \"name_3\": \"surname:Agalliu;given-names:D\", \"name_4\": \"surname:Sulzer;given-names:D\", \"pub-id_doi\": \"10.1016/B978-0-12-819410-2.00023-0\", \"pub-id_pmid\": \"35034753\", \"section_type\": \"REF\", \"source\": \"Handb. Clin. Neurol\", \"type\": \"ref\", \"volume\": \"184\", \"year\": \"2022\"}, \"text\": \"T cells, \\u03b1-synuclein and Parkinson disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61100, \"infons\": {\"fpage\": \"399\", \"lpage\": \"404\", \"name_0\": \"surname:Gate;given-names:D\", \"name_1\": \"surname:Saligrama;given-names:N\", \"name_10\": \"surname:McBride;given-names:A\", \"name_11\": \"surname:Pluvinage;given-names:J\", \"name_12\": \"surname:Elahi;given-names:F\", \"name_13\": \"surname:Tam;given-names:G-K-Y\", \"name_14\": \"surname:Kim;given-names:Y\", \"name_15\": \"surname:Greicius;given-names:M\", \"name_16\": \"surname:Wagner;given-names:AD\", \"name_17\": \"surname:Aigner;given-names:L\", \"name_18\": \"surname:Galasko;given-names:DR\", \"name_19\": \"surname:Davis;given-names:MM\", \"name_2\": \"surname:Leventhal;given-names:O\", \"name_20\": \"surname:Wyss-Coray;given-names:T\", \"name_3\": \"surname:Yang;given-names:AC\", \"name_4\": \"surname:Unger;given-names:MS\", \"name_5\": \"surname:Middeldorp;given-names:J\", \"name_6\": \"surname:Chen;given-names:K\", \"name_7\": \"surname:Lehallier;given-names:B\", \"name_8\": \"surname:Channappa;given-names:D\", \"name_9\": \"surname:De Los Santos;given-names:MB\", \"pub-id_doi\": \"10.1038/S41586-019-1895-7\", \"pub-id_pmid\": \"31915375\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"577\", \"year\": \"2020\"}, \"text\": \"Clonally expanded CD8 T cells patrol the cerebrospinal fluid in Alzheimer\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61186, \"infons\": {\"fpage\": \"868\", \"lpage\": \"874\", \"name_0\": \"surname:Gate;given-names:D\", \"name_1\": \"surname:Tapp;given-names:E\", \"name_10\": \"surname:Channappa;given-names:D\", \"name_11\": \"surname:Henderson;given-names:VW\", \"name_12\": \"surname:Keller;given-names:A\", \"name_13\": \"surname:Aigner;given-names:L\", \"name_14\": \"surname:Galasko;given-names:DR\", \"name_15\": \"surname:Davis;given-names:MM\", \"name_16\": \"surname:Poston;given-names:KL\", \"name_17\": \"surname:Wyss-Coray;given-names:T\", \"name_2\": \"surname:Leventhal;given-names:O\", \"name_3\": \"surname:Shahid;given-names:M\", \"name_4\": \"surname:Nonninger;given-names:TJ\", \"name_5\": \"surname:Yang;given-names:AC\", \"name_6\": \"surname:Strempfl;given-names:K\", \"name_7\": \"surname:Unger;given-names:MS\", \"name_8\": \"surname:Fehlmann;given-names:T\", \"name_9\": \"surname:Oh;given-names:H\", \"pub-id_doi\": \"10.1126/science.abf7266\", \"pub-id_pmid\": \"34648304\", \"section_type\": \"REF\", \"source\": \"Science\", \"type\": \"ref\", \"volume\": \"374\", \"year\": \"2021\"}, \"text\": \"CD4+ T cells contribute to neurodegeneration in Lewy body dementia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61253, \"infons\": {\"fpage\": \"D325\", \"lpage\": \"D334\", \"pub-id_doi\": \"10.1093/nar/gkaa1113\", \"pub-id_pmid\": \"33290552\", \"section_type\": \"REF\", \"source\": \"Nucleic Acids Res\", \"type\": \"ref\", \"volume\": \"49\", \"year\": \"2021\"}, \"text\": \"The Gene Ontology resource: enriching a GOld mine\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61303, \"infons\": {\"fpage\": \"20918\", \"lpage\": \"20923\", \"name_0\": \"surname:Gottfried-Blackmore;given-names:A\", \"name_1\": \"surname:Kaunzner;given-names:UW\", \"name_2\": \"surname:Idoyaga;given-names:J\", \"name_3\": \"surname:Felger;given-names:JC\", \"name_4\": \"surname:McEwen;given-names:BS\", \"name_5\": \"surname:Bulloch;given-names:K\", \"pub-id_doi\": \"10.1073/pnas.0911509106\", \"pub-id_pmid\": \"19906988\", \"section_type\": \"REF\", \"source\": \"PNAS\", \"type\": \"ref\", \"volume\": \"106\", \"year\": \"2009\"}, \"text\": \"Acute in vivo exposure to interferon-gamma enables resident brain dendritic cells to become effective antigen presenting cells\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61430, \"infons\": {\"fpage\": \"82\", \"lpage\": \"90\", \"name_0\": \"surname:Gunsolly;given-names:C\", \"name_1\": \"surname:Nicholson;given-names:JD\", \"name_2\": \"surname:Listwak;given-names:SJ\", \"name_3\": \"surname:Ledee;given-names:D\", \"name_4\": \"surname:Zelenka;given-names:P\", \"name_5\": \"surname:Verthelyi;given-names:D\", \"name_6\": \"surname:Chapoval;given-names:S\", \"name_7\": \"surname:Keegan;given-names:A\", \"name_8\": \"surname:Tonelli;given-names:LH\", \"pub-id_doi\": \"10.1016/j.jneuroim.2010.04.019\", \"pub-id_pmid\": \"20605223\", \"section_type\": \"REF\", \"source\": \"J. Neuroimmunol\", \"type\": \"ref\", \"volume\": \"225\", \"year\": \"2010\"}, \"text\": \"Expression and regulation in the brain of the chemokine CCL27 gene locus\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61503, \"infons\": {\"fpage\": \"10896\", \"lpage\": \"10901\", \"name_0\": \"surname:Harrison;given-names:JK\", \"name_1\": \"surname:Jiang;given-names:Y\", \"name_10\": \"surname:Botti;given-names:P\", \"name_11\": \"surname:Bacon;given-names:KB\", \"name_12\": \"surname:Feng;given-names:L\", \"name_2\": \"surname:Chen;given-names:S\", \"name_3\": \"surname:Xia;given-names:Y\", \"name_4\": \"surname:Maciejewski;given-names:D\", \"name_5\": \"surname:McNamara;given-names:RK\", \"name_6\": \"surname:Streit;given-names:WJ\", \"name_7\": \"surname:Salafranca;given-names:MN\", \"name_8\": \"surname:Adhikari;given-names:S\", \"name_9\": \"surname:Thompson;given-names:DA\", \"pub-id_doi\": \"10.1073/pnas.95.18.10896\", \"pub-id_pmid\": \"9724801\", \"section_type\": \"REF\", \"source\": \"PNAS\", \"type\": \"ref\", \"volume\": \"95\", \"year\": \"1998\"}, \"text\": \"Role for neuronally derived fractalkine in mediating interactions between neurons and CX3CR1-expressing microglia\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61617, \"infons\": {\"fpage\": \"345\", \"lpage\": \"348\", \"name_0\": \"surname:Hirsch;given-names:E\", \"name_1\": \"surname:Graybiel;given-names:AM\", \"name_2\": \"surname:Agid;given-names:YA\", \"pub-id_doi\": \"10.1038/334345a0\", \"pub-id_pmid\": \"2899295\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"334\", \"year\": \"1988\"}, \"text\": \"Melanized dopaminergic neurons are differentially susceptible to degeneration in Parkinson\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61720, \"infons\": {\"fpage\": \"e70921\", \"name_0\": \"surname:Hobson;given-names:BD\", \"name_1\": \"surname:Choi;given-names:SJ\", \"name_2\": \"surname:Mosharov;given-names:EV\", \"name_3\": \"surname:Soni;given-names:RK\", \"name_4\": \"surname:Sulzer;given-names:D\", \"name_5\": \"surname:Sims;given-names:P\", \"pub-id_doi\": \"10.7554/eLife.70921\", \"pub-id_pmid\": \"35098924\", \"section_type\": \"REF\", \"source\": \"Elife\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2022a\"}, \"text\": \"Subcellular proteomics of dopamine neurons in the mouse brain\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61782, \"infons\": {\"name_0\": \"surname:Hobson;given-names:BD\", \"name_1\": \"surname:Sulzer;given-names:D\", \"pub-id_doi\": \"10.3233/JPD-223153\", \"section_type\": \"REF\", \"source\": \"J Parkinsons Dis\", \"type\": \"ref\", \"year\": \"2022\"}, \"text\": \"Neuronal Presentation of Antigen and Its Possible Role in Parkinson\\u2019s Disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61862, \"infons\": {\"fpage\": \"110208\", \"issue\": \"2\", \"name_0\": \"surname:Hobson;given-names:BD\", \"name_1\": \"surname:Kong;given-names:L\", \"name_2\": \"surname:Angelo;given-names:MF\", \"name_3\": \"surname:Lieberman;given-names:OJ\", \"name_4\": \"surname:Mosharov;given-names:EV\", \"name_5\": \"surname:Herzog;given-names:E\", \"name_6\": \"surname:Sulzer;given-names:D\", \"name_7\": \"surname:Sims;given-names:PA\", \"pub-id_pmid\": \"35021090\", \"section_type\": \"REF\", \"source\": \"Cell Rep\", \"type\": \"ref\", \"volume\": \"38\", \"year\": \"2022b\"}, \"text\": \"Subcellular and regional localization of mRNA translation in midbrain dopamine neurons\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 61949, \"infons\": {\"fpage\": \"43\", \"lpage\": \"50\", \"name_0\": \"surname:H\\u00f6ftberger;given-names:R\", \"name_1\": \"surname:Aboul-Enein;given-names:F\", \"name_2\": \"surname:Brueck;given-names:W\", \"name_3\": \"surname:Lucchinetti;given-names:C\", \"name_4\": \"surname:Rodriguez;given-names:M\", \"name_5\": \"surname:Schmidbauer;given-names:M\", \"name_6\": \"surname:Jellinger;given-names:K\", \"name_7\": \"surname:Lassmann;given-names:H\", \"pub-id_doi\": \"10.1111/j.1750-3639.2004.tb00496.x\", \"pub-id_pmid\": \"14997936\", \"section_type\": \"REF\", \"source\": \"Brain Pathol\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2004\"}, \"text\": \"Expression of major histocompatibility complex class I molecules on the different cell types in multiple sclerosis lesions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62072, \"infons\": {\"fpage\": \"1\", \"lpage\": \"30\", \"name_0\": \"surname:Howcroft;given-names:TK\", \"name_1\": \"surname:Singer;given-names:DS\", \"pub-id_doi\": \"10.1385/IR:27:1:1\", \"pub-id_pmid\": \"12637766\", \"section_type\": \"REF\", \"source\": \"Immunol. Res\", \"type\": \"ref\", \"volume\": \"27\", \"year\": \"2003\"}, \"text\": \"Expression of nonclassical MHC class Ib genes: comparison of regulatory elements\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62153, \"infons\": {\"fpage\": \"238\", \"name_0\": \"surname:Howe;given-names:CL\", \"name_1\": \"surname:LaFrance-Corey;given-names:RG\", \"name_2\": \"surname:Goddery;given-names:EN\", \"name_3\": \"surname:Johnson;given-names:RK\", \"name_4\": \"surname:Mirchia;given-names:K\", \"pub-id_doi\": \"10.1186/s12974-017-1015-2\", \"pub-id_pmid\": \"29202854\", \"section_type\": \"REF\", \"source\": \"J. Neuroinflammation\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2017\"}, \"text\": \"Neuronal CCL2 expression drives inflammatory monocyte infiltration into the brain during acute virus infection\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62264, \"infons\": {\"fpage\": \"258\", \"lpage\": \"267\", \"name_0\": \"surname:Isaacs;given-names:A\", \"name_1\": \"surname:Lindenmann;given-names:J\", \"name_2\": \"surname:Andrewes;given-names:CH\", \"pub-id_doi\": \"10.1098/rspb.1957.0048\", \"pub-id_pmid\": \"13465720\", \"section_type\": \"REF\", \"source\": \"Proceedings of the Royal Society of London. Series B - Biological Sciences\", \"type\": \"ref\", \"volume\": \"147\", \"year\": \"1957\"}, \"text\": \"Virus interference. I. The interferon\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62302, \"infons\": {\"fpage\": \"545\", \"lpage\": \"558\", \"name_0\": \"surname:Ivashkiv;given-names:LB\", \"pub-id_doi\": \"10.1038/S41577-018-0029-z\", \"pub-id_pmid\": \"29921905\", \"section_type\": \"REF\", \"source\": \"Nat. Rev. Immunol\", \"type\": \"ref\", \"volume\": \"18\", \"year\": \"2018\"}, \"text\": \"IFN\\u03b3: signalling, epigenetics and roles in immunity, metabolism, disease and cancer immunotherapy\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62404, \"infons\": {\"fpage\": \"12849\", \"name_0\": \"surname:Iwata;given-names:H\", \"name_1\": \"surname:Goettsch;given-names:C\", \"name_10\": \"surname:Inoue;given-names:N\", \"name_11\": \"surname:Fukuda;given-names:D\", \"name_12\": \"surname:Mojcher;given-names:A\", \"name_13\": \"surname:Mattson;given-names:PC\", \"name_14\": \"surname:Barab\\u00e1si;given-names:A-L\", \"name_15\": \"surname:Boothby;given-names:M\", \"name_16\": \"surname:Aikawa;given-names:E\", \"name_17\": \"surname:Singh;given-names:SA\", \"name_18\": \"surname:Aikawa;given-names:M\", \"name_2\": \"surname:Sharma;given-names:A\", \"name_3\": \"surname:Ricchiuto;given-names:P\", \"name_4\": \"surname:Goh;given-names:WWB\", \"name_5\": \"surname:Halu;given-names:A\", \"name_6\": \"surname:Yamada;given-names:I\", \"name_7\": \"surname:Yoshida;given-names:H\", \"name_8\": \"surname:Hara;given-names:T\", \"name_9\": \"surname:Wei;given-names:M\", \"pub-id_doi\": \"10.1038/ncomms12849\", \"pub-id_pmid\": \"27796300\", \"section_type\": \"REF\", \"source\": \"Nat. Commun\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2016\"}, \"text\": \"PARP9 and PARP14 cross-regulate macrophage activation via STAT1 ADP-ribosylation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62485, \"infons\": {\"fpage\": \"543\", \"issue\": \"7745\", \"lpage\": \"547\", \"name_0\": \"surname:J\\u00e4kel;given-names:S\", \"name_1\": \"surname:Agirre;given-names:E\", \"name_2\": \"surname:Mendanha Falc\\u00e3o;given-names:A\", \"name_3\": \"surname:van Bruggen;given-names:D\", \"name_4\": \"surname:Lee;given-names:KW\", \"name_5\": \"surname:Knuesel;given-names:I\", \"name_6\": \"surname:Malhotra;given-names:D\", \"name_7\": \"surname:ffrench-Constant;given-names:C\", \"name_8\": \"surname:Williams;given-names:A\", \"name_9\": \"surname:Castelo-Branco;given-names:G\", \"pub-id_pmid\": \"30747918\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"566\", \"year\": \"2019\"}, \"text\": \"Altered human oligodendrocyte heterogeneity in multiple sclerosis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62551, \"infons\": {\"fpage\": \"16\", \"lpage\": \"21\", \"name_0\": \"surname:Jongsma;given-names:MLM\", \"name_1\": \"surname:Guarda;given-names:G\", \"name_2\": \"surname:Spaapen;given-names:RM\", \"pub-id_doi\": \"10.1016/j.molimm.2017.12.005\", \"pub-id_pmid\": \"29224918\", \"section_type\": \"REF\", \"source\": \"Mol. Immunol\", \"type\": \"ref\", \"volume\": \"113\", \"year\": \"2019\"}, \"text\": \"The regulatory network behind MHC class I expression\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62604, \"infons\": {\"fpage\": \"1446\", \"issue\": \"11\", \"lpage\": \"1457\", \"name_0\": \"surname:Kaya;given-names:T\", \"name_1\": \"surname:Mattugini;given-names:N\", \"name_10\": \"surname:Kaji;given-names:S\", \"name_11\": \"surname:Liesz;given-names:A\", \"name_12\": \"surname:Gokce;given-names:O\", \"name_13\": \"surname:Simons;given-names:M\", \"name_2\": \"surname:Liu;given-names:L.u.\", \"name_3\": \"surname:Ji;given-names:H\", \"name_4\": \"surname:Cantuti-Castelvetri;given-names:L\", \"name_5\": \"surname:Wu;given-names:J\", \"name_6\": \"surname:Schifferer;given-names:M\", \"name_7\": \"surname:Groh;given-names:J\", \"name_8\": \"surname:Martini;given-names:R\", \"name_9\": \"surname:Besson-Girard;given-names:S\", \"pub-id_pmid\": \"36280798\", \"section_type\": \"REF\", \"source\": \"Nat. Neurosci\", \"type\": \"ref\", \"volume\": \"25\", \"year\": \"2022\"}, \"text\": \"CD8+ T cells induce interferon-responsive oligodendrocytes and microglia in white matter aging\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62699, \"infons\": {\"fpage\": \"1619\", \"lpage\": \"1636\", \"name_0\": \"surname:Klaus;given-names:C\", \"name_1\": \"surname:Liao;given-names:H\", \"name_2\": \"surname:Allendorf;given-names:DH\", \"name_3\": \"surname:Brown;given-names:GC\", \"name_4\": \"surname:Neumann;given-names:H\", \"pub-id_doi\": \"10.1002/glia.23945\", \"pub-id_pmid\": \"33340149\", \"section_type\": \"REF\", \"source\": \"Glia\", \"type\": \"ref\", \"volume\": \"69\", \"year\": \"2021\"}, \"text\": \"Sialylation acts as a checkpoint for innate immune responses in the central nervous system\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62790, \"infons\": {\"fpage\": \"2087\", \"lpage\": \"2103\", \"name_0\": \"surname:Kreutzfeldt;given-names:M\", \"name_1\": \"surname:Bergthaler;given-names:A\", \"name_10\": \"surname:Forman;given-names:R\", \"name_11\": \"surname:M\\u00fcller;given-names:W\", \"name_12\": \"surname:Becher;given-names:B\", \"name_13\": \"surname:Misgeld;given-names:T\", \"name_14\": \"surname:Kerschensteiner;given-names:M\", \"name_15\": \"surname:Pinschewer;given-names:DD\", \"name_16\": \"surname:Merkler;given-names:D\", \"name_2\": \"surname:Fernandez;given-names:M\", \"name_3\": \"surname:Br\\u00fcck;given-names:W\", \"name_4\": \"surname:Steinbach;given-names:K\", \"name_5\": \"surname:Vorm;given-names:M\", \"name_6\": \"surname:Coras;given-names:R\", \"name_7\": \"surname:Bl\\u00fcmcke;given-names:I\", \"name_8\": \"surname:Bonilla;given-names:WV\", \"name_9\": \"surname:Fleige;given-names:A\", \"pub-id_doi\": \"10.1084/jem.20122143\", \"pub-id_pmid\": \"23999498\", \"section_type\": \"REF\", \"source\": \"J. Exp. Med\", \"type\": \"ref\", \"volume\": \"210\", \"year\": \"2013\"}, \"text\": \"Neuroprotective intervention by interferon-\\u03b3 blockade prevents CD8+ T cell\\u2013mediated dendrite and synapse loss\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62904, \"infons\": {\"fpage\": \"499\", \"lpage\": \"524\", \"name_0\": \"surname:Krishnaswami;given-names:SR\", \"name_1\": \"surname:Grindberg;given-names:RV\", \"name_10\": \"surname:Hodge;given-names:R\", \"name_11\": \"surname:McCarthy;given-names:JK\", \"name_12\": \"surname:Kelder;given-names:M\", \"name_13\": \"surname:McCorrison;given-names:J\", \"name_14\": \"surname:Aevermann;given-names:BD\", \"name_15\": \"surname:Fuertes;given-names:FD\", \"name_16\": \"surname:Scheuermann;given-names:RH\", \"name_17\": \"surname:Lee;given-names:J\", \"name_18\": \"surname:Lein;given-names:ES\", \"name_19\": \"surname:Schork;given-names:N\", \"name_2\": \"surname:Novotny;given-names:M\", \"name_20\": \"surname:McConnell;given-names:MJ\", \"name_21\": \"surname:Gage;given-names:FH\", \"name_22\": \"surname:Lasken;given-names:RS\", \"name_3\": \"surname:Venepally;given-names:P\", \"name_4\": \"surname:Lacar;given-names:B\", \"name_5\": \"surname:Bhutani;given-names:K\", \"name_6\": \"surname:Linker;given-names:SB\", \"name_7\": \"surname:Pham;given-names:S\", \"name_8\": \"surname:Erwin;given-names:JA\", \"name_9\": \"surname:Miller;given-names:JA\", \"pub-id_doi\": \"10.1038/nprot.2016.015\", \"pub-id_pmid\": \"26890679\", \"section_type\": \"REF\", \"source\": \"Nat. Protoc\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2016\"}, \"text\": \"Using single nuclei for RNA-seq to capture the transcriptome of postmortem neurons\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 62987, \"infons\": {\"fpage\": \"205\", \"name_0\": \"surname:Kustrimovic;given-names:N\", \"name_1\": \"surname:Comi;given-names:C\", \"name_10\": \"surname:Sturchio;given-names:A\", \"name_11\": \"surname:Mauri;given-names:M\", \"name_12\": \"surname:Bono;given-names:G\", \"name_13\": \"surname:Marino;given-names:F\", \"name_14\": \"surname:Cosentino;given-names:M\", \"name_2\": \"surname:Magistrelli;given-names:L\", \"name_3\": \"surname:Rasini;given-names:E\", \"name_4\": \"surname:Legnaro;given-names:M\", \"name_5\": \"surname:Bombelli;given-names:R\", \"name_6\": \"surname:Aleksic;given-names:I\", \"name_7\": \"surname:Blandini;given-names:F\", \"name_8\": \"surname:Minafra;given-names:B\", \"name_9\": \"surname:Riboldazzi;given-names:G\", \"pub-id_doi\": \"10.1186/s12974-018-1248-8\", \"section_type\": \"REF\", \"source\": \"J. Neuroinflamm\", \"type\": \"ref\", \"volume\": \"15\", \"year\": \"2018\"}, \"text\": \"Parkinson\\u2019s disease patients have a complex phenotypic and functional Th1 bias: cross-sectional studies of CD4+ Th1/Th2/T17 and Treg in drug-na\\u00efve and drug-treated patients\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63163, \"infons\": {\"fpage\": \"4223\", \"lpage\": \"4234\", \"name_0\": \"surname:Lee;given-names:SH\", \"name_1\": \"surname:Carrero;given-names:JA\", \"name_2\": \"surname:Uppaluri;given-names:R\", \"name_3\": \"surname:White;given-names:JM\", \"name_4\": \"surname:Archambault;given-names:JM\", \"name_5\": \"surname:Lai;given-names:KS\", \"name_6\": \"surname:Chan;given-names:SR\", \"name_7\": \"surname:Sheehan;given-names:KCF\", \"name_8\": \"surname:Unanue;given-names:ER\", \"name_9\": \"surname:Schreiber;given-names:RD\", \"pub-id_doi\": \"10.4049/jimmunol.1300910\", \"pub-id_pmid\": \"24048899\", \"section_type\": \"REF\", \"source\": \"J. Immunol\", \"type\": \"ref\", \"volume\": \"191\", \"year\": \"2013\"}, \"text\": \"Identifying the initiating events of anti-Listeria responses using mice with conditional loss of IFN-\\u03b3 receptor subunit 1 (IFNGR1)\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63298, \"infons\": {\"fpage\": \"184\", \"lpage\": \"197\", \"name_0\": \"surname:Levine;given-names:JH\", \"name_1\": \"surname:Simonds;given-names:EF\", \"name_10\": \"surname:Finck;given-names:R\", \"name_11\": \"surname:Gedman;given-names:AL\", \"name_12\": \"surname:Radtke;given-names:I\", \"name_13\": \"surname:Downing;given-names:JR\", \"name_14\": \"surname:Pe\\u2019er;given-names:D\", \"name_15\": \"surname:Nolan;given-names:GP\", \"name_2\": \"surname:Bendall;given-names:SC\", \"name_3\": \"surname:Davis;given-names:KL\", \"name_4\": \"surname:Amir;given-names:ED\", \"name_5\": \"surname:Tadmor;given-names:MD\", \"name_6\": \"surname:Litvin;given-names:O\", \"name_7\": \"surname:Fienberg;given-names:HG\", \"name_8\": \"surname:Jager;given-names:A\", \"name_9\": \"surname:Zunder;given-names:ER\", \"pub-id_doi\": \"10.1016/j.cell.2015.05.047\", \"pub-id_pmid\": \"26095251\", \"section_type\": \"REF\", \"source\": \"Cell\", \"type\": \"ref\", \"volume\": \"162\", \"year\": \"2015\"}, \"text\": \"Data-Driven Phenotypic Dissection of AML Reveals Progenitor-like Cells that Correlate with Prognosis\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63399, \"infons\": {\"fpage\": \"e8557\", \"name_0\": \"surname:Levitin;given-names:HM\", \"name_1\": \"surname:Yuan;given-names:J\", \"name_10\": \"surname:Sims;given-names:PA\", \"name_2\": \"surname:Cheng;given-names:YL\", \"name_3\": \"surname:Ruiz;given-names:FJ\", \"name_4\": \"surname:Bush;given-names:EC\", \"name_5\": \"surname:Bruce;given-names:JN\", \"name_6\": \"surname:Canoll;given-names:P\", \"name_7\": \"surname:Iavarone;given-names:A\", \"name_8\": \"surname:Lasorella;given-names:A\", \"name_9\": \"surname:Blei;given-names:DM\", \"pub-id_pmid\": \"30796088\", \"section_type\": \"REF\", \"source\": \"Mol. Syst. Biol\", \"type\": \"ref\", \"volume\": \"15\", \"year\": \"2019\"}, \"text\": \"De novo gene signature identification from single-cell RNA-seq with hierarchical Poisson factorization\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63502, \"infons\": {\"fpage\": \"1875\", \"name_0\": \"surname:Lindestam Arlehamn;given-names:CS\", \"name_1\": \"surname:Dhanwani;given-names:R\", \"name_10\": \"surname:Amara;given-names:AW\", \"name_11\": \"surname:Standaert;given-names:DG\", \"name_12\": \"surname:Goldman;given-names:JG\", \"name_13\": \"surname:Litvan;given-names:I\", \"name_14\": \"surname:Peters;given-names:B\", \"name_15\": \"surname:Sulzer;given-names:D\", \"name_16\": \"surname:Sette;given-names:A\", \"name_2\": \"surname:Pham;given-names:J\", \"name_3\": \"surname:Kuan;given-names:R\", \"name_4\": \"surname:Frazier;given-names:A\", \"name_5\": \"surname:Rezende Dutra;given-names:J\", \"name_6\": \"surname:Phillips;given-names:E\", \"name_7\": \"surname:Mallal;given-names:S\", \"name_8\": \"surname:Roederer;given-names:M\", \"name_9\": \"surname:Marder;given-names:KS\", \"pub-id_doi\": \"10.1038/s41467-020-15626-w\", \"pub-id_pmid\": \"32313102\", \"section_type\": \"REF\", \"source\": \"Nat. Commun\", \"type\": \"ref\", \"volume\": \"11\", \"year\": \"2020\"}, \"text\": \"\\u03b1-Synuclein-specific T cell reactivity is associated with preclinical and early Parkinson\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63606, \"infons\": {\"fpage\": \"1191\", \"lpage\": \"1195\", \"name_0\": \"surname:Littman;given-names:SJ\", \"name_1\": \"surname:Faltynek;given-names:CR\", \"name_2\": \"surname:Baglioni;given-names:C\", \"pub-id_doi\": \"10.1016/S0021-9258(20)71226-3\", \"pub-id_pmid\": \"3155725\", \"section_type\": \"REF\", \"source\": \"J. Biol. Chem\", \"type\": \"ref\", \"volume\": \"260\", \"year\": \"1985\"}, \"text\": \"Binding of human recombinant 125I-interferon gamma to receptors on human cells\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63685, \"infons\": {\"fpage\": \"503\", \"lpage\": \"508\", \"name_0\": \"surname:Lodygin;given-names:D\", \"name_1\": \"surname:Hermann;given-names:M\", \"name_10\": \"surname:Reichardt;given-names:HM\", \"name_11\": \"surname:Zagrebelsky;given-names:M\", \"name_12\": \"surname:Mollenhauer;given-names:B\", \"name_13\": \"surname:K\\u00fcgler;given-names:S\", \"name_14\": \"surname:Fitzner;given-names:D\", \"name_15\": \"surname:Frahm;given-names:J\", \"name_16\": \"surname:Stadelmann;given-names:C\", \"name_17\": \"surname:Haberl;given-names:M\", \"name_18\": \"surname:Odoardi;given-names:F\", \"name_19\": \"surname:Fl\\u00fcgel;given-names:A\", \"name_2\": \"surname:Schweingruber;given-names:N\", \"name_3\": \"surname:Fl\\u00fcgel-Koch;given-names:C\", \"name_4\": \"surname:Watanabe;given-names:T\", \"name_5\": \"surname:Schlosser;given-names:C\", \"name_6\": \"surname:Merlini;given-names:A\", \"name_7\": \"surname:K\\u00f6rner;given-names:H\", \"name_8\": \"surname:Chang;given-names:H-F\", \"name_9\": \"surname:Fischer;given-names:HJ\", \"pub-id_doi\": \"10.1038/S41586-019-0964-2\", \"pub-id_pmid\": \"30787438\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"566\", \"year\": \"2019\"}, \"text\": \"\\u03b2-synuclein-reactive T cells induce autoimmune CNS grey matter degeneration\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63764, \"infons\": {\"fpage\": \"550\", \"name_0\": \"surname:Love;given-names:MI\", \"name_1\": \"surname:Huber;given-names:W\", \"name_2\": \"surname:Anders;given-names:S\", \"pub-id_doi\": \"10.1186/s13059-014-0550-8\", \"pub-id_pmid\": \"25516281\", \"section_type\": \"REF\", \"source\": \"Genome Biol\", \"type\": \"ref\", \"volume\": \"15\", \"year\": \"2014\"}, \"text\": \"Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63844, \"infons\": {\"fpage\": \"75\", \"name_0\": \"surname:Lun;given-names:ATL\", \"name_1\": \"surname:Bach;given-names:K\", \"name_2\": \"surname:Marioni;given-names:JC\", \"pub-id_doi\": \"10.1186/s13059-016-0947-7\", \"pub-id_pmid\": \"27122128\", \"section_type\": \"REF\", \"source\": \"Genome Biol\", \"type\": \"ref\", \"volume\": \"17\", \"year\": \"2016\"}, \"text\": \"Pooling across cells to normalize single-cell RNA sequencing data with many zero counts\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 63932, \"infons\": {\"fpage\": \"444\", \"lpage\": \"453\", \"name_0\": \"surname:Matsuda;given-names:W\", \"name_1\": \"surname:Furuta;given-names:T\", \"name_2\": \"surname:Nakamura;given-names:KC\", \"name_3\": \"surname:Hioki;given-names:H\", \"name_4\": \"surname:Fujiyama;given-names:F\", \"name_5\": \"surname:Arai;given-names:R\", \"name_6\": \"surname:Kaneko;given-names:T\", \"pub-id_doi\": \"10.1523/JNEUROSCI.4029-08.2009\", \"pub-id_pmid\": \"19144844\", \"section_type\": \"REF\", \"source\": \"J. Neurosci\", \"type\": \"ref\", \"volume\": \"29\", \"year\": \"2009\"}, \"text\": \"Single nigrostriatal dopaminergic neurons form widely spread and highly dense axonal arborizations in the neostriatum\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64050, \"infons\": {\"fpage\": \"1823\", \"lpage\": \"1833\", \"name_0\": \"surname:McDole;given-names:JR\", \"name_1\": \"surname:Danzer;given-names:SC\", \"name_2\": \"surname:Pun;given-names:RYK\", \"name_3\": \"surname:Chen;given-names:Y\", \"name_4\": \"surname:Johnson;given-names:HL\", \"name_5\": \"surname:Pirko;given-names:I\", \"name_6\": \"surname:Johnson;given-names:AJ\", \"pub-id_doi\": \"10.2353/ajpath.2010.100231\", \"pub-id_pmid\": \"20813972\", \"section_type\": \"REF\", \"source\": \"Am. J. Pathol\", \"type\": \"ref\", \"volume\": \"177\", \"year\": \"2010\"}, \"text\": \"Rapid formation of extended processes and engagement of Theiler\\u2019s virus-infected neurons by CNS-infiltrating CD8 T cells\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64173, \"infons\": {\"fpage\": \"1285\", \"lpage\": \"1291\", \"name_0\": \"surname:McGeer;given-names:PL\", \"name_1\": \"surname:Itagaki;given-names:S\", \"name_2\": \"surname:Boyes;given-names:BE\", \"name_3\": \"surname:McGeer;given-names:EG\", \"pub-id_doi\": \"10.1212/wnl.38.8.1285\", \"pub-id_pmid\": \"3399080\", \"section_type\": \"REF\", \"source\": \"Neurology\", \"type\": \"ref\", \"volume\": \"38\", \"year\": \"1988\"}, \"text\": \"Reactive microglia are positive for HLA-DR in the substantia nigra of Parkinson\\u2019s and Alzheimer\\u2019s disease brains\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64290, \"infons\": {\"fpage\": \"3174\", \"lpage\": \"3183\", \"name_0\": \"surname:Min;given-names:W\", \"name_1\": \"surname:Pober;given-names:JS\", \"name_2\": \"surname:Johnson;given-names:DR\", \"pub-id_pmid\": \"8617938\", \"section_type\": \"REF\", \"source\": \"J Immunol\", \"type\": \"ref\", \"volume\": \"156\", \"year\": \"1996\"}, \"text\": \"Kinetically coordinated induction of TAP1 and HLA class I by IFN-gamma: the rapid induction of TAP1 by IFN-gamma is mediated by Stat1 alpha\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64430, \"infons\": {\"fpage\": \"19\", \"name_0\": \"surname:Mo;given-names:M-S\", \"name_1\": \"surname:Li;given-names:G-H\", \"name_10\": \"surname:Chen;given-names:C-J\", \"name_11\": \"surname:Qu;given-names:S-G\", \"name_12\": \"surname:He;given-names:J-X\", \"name_13\": \"surname:Xu;given-names:P-Y\", \"name_2\": \"surname:Sun;given-names:C-C\", \"name_3\": \"surname:Huang;given-names:S-X\", \"name_4\": \"surname:Wei;given-names:L\", \"name_5\": \"surname:Zhang;given-names:L-M\", \"name_6\": \"surname:Zhou;given-names:M-M\", \"name_7\": \"surname:Wu;given-names:Z-H\", \"name_8\": \"surname:Guo;given-names:W-Y\", \"name_9\": \"surname:Yang;given-names:X-L\", \"pub-id_doi\": \"10.1186/s40035-018-0125-9\", \"section_type\": \"REF\", \"source\": \"Transl. Neurodegen\", \"type\": \"ref\", \"volume\": \"7\", \"year\": \"2018\"}, \"text\": \"Dopaminergic neurons show increased low-molecular-mass protein 7 activity induced by 6-hydroxydopamine in vitro and in vivo\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64554, \"infons\": {\"fpage\": \"94\", \"lpage\": \"97\", \"name_0\": \"surname:Mogi;given-names:M\", \"name_1\": \"surname:Kondo;given-names:T\", \"name_2\": \"surname:Mizuno;given-names:Y\", \"name_3\": \"surname:Nagatsu;given-names:T\", \"pub-id_doi\": \"10.1016/j.neulet.2006.12.003\", \"pub-id_pmid\": \"17196747\", \"section_type\": \"REF\", \"source\": \"Neurosci. Lett\", \"type\": \"ref\", \"volume\": \"414\", \"year\": \"2007\"}, \"text\": \"p53 protein, interferon-gamma, and NF-kappaB levels are elevated in the parkinsonian brain\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64645, \"infons\": {\"fpage\": \"14470\", \"lpage\": \"14475\", \"name_0\": \"surname:Morales;given-names:J\", \"name_1\": \"surname:Homey;given-names:B\", \"name_10\": \"surname:Zlotnik;given-names:A\", \"name_2\": \"surname:Vicari;given-names:AP\", \"name_3\": \"surname:Hudak;given-names:S\", \"name_4\": \"surname:Oldham;given-names:E\", \"name_5\": \"surname:Hedrick;given-names:J\", \"name_6\": \"surname:Orozco;given-names:R\", \"name_7\": \"surname:Copeland;given-names:NG\", \"name_8\": \"surname:Jenkins;given-names:NA\", \"name_9\": \"surname:McEvoy;given-names:LM\", \"pub-id_doi\": \"10.1073/pnas.96.25.14470\", \"pub-id_pmid\": \"10588729\", \"section_type\": \"REF\", \"source\": \"PNAS\", \"type\": \"ref\", \"volume\": \"96\", \"year\": \"1999\"}, \"text\": \"CTACK, a skin-associated chemokine that preferentially attracts skin-homing memory T cells\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64736, \"infons\": {\"fpage\": \"3328\", \"lpage\": \"3337\", \"name_0\": \"surname:Mount;given-names:MP\", \"name_1\": \"surname:Lira;given-names:A\", \"name_2\": \"surname:Grimes;given-names:D\", \"name_3\": \"surname:Smith;given-names:PD\", \"name_4\": \"surname:Faucher;given-names:S\", \"name_5\": \"surname:Slack;given-names:R\", \"name_6\": \"surname:Anisman;given-names:H\", \"name_7\": \"surname:Hayley;given-names:S\", \"name_8\": \"surname:Park;given-names:DS\", \"pub-id_doi\": \"10.1523/JNEUROSCI.5321-06.2007\", \"pub-id_pmid\": \"17376993\", \"section_type\": \"REF\", \"source\": \"J. Neurosci\", \"type\": \"ref\", \"volume\": \"27\", \"year\": \"2007\"}, \"text\": \"Involvement of interferon-gamma in microglial-mediated loss of dopaminergic neurons\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64820, \"infons\": {\"fpage\": \"1\", \"lpage\": \"18\", \"name_0\": \"surname:M\\u00fcllner;given-names:D\", \"pub-id_doi\": \"10.18637/jss.v053.i09\", \"section_type\": \"REF\", \"source\": \"J. Stat. Softw\", \"type\": \"ref\", \"volume\": \"53\", \"year\": \"2013\"}, \"text\": \"fastcluster: Fast hierarchical, agglomerative clustering routines for R and python\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64903, \"infons\": {\"fpage\": \"eaap7388\", \"name_0\": \"surname:Murakami;given-names:G\", \"name_1\": \"surname:Edamura;given-names:M\", \"name_2\": \"surname:Furukawa;given-names:T\", \"name_3\": \"surname:Kawasaki;given-names:H\", \"name_4\": \"surname:Kosugi;given-names:I\", \"name_5\": \"surname:Fukuda;given-names:A\", \"name_6\": \"surname:Iwashita;given-names:T\", \"name_7\": \"surname:Nakahara;given-names:D\", \"pub-id_doi\": \"10.1126/sciadv.aap7388\", \"pub-id_pmid\": \"29546241\", \"section_type\": \"REF\", \"source\": \"Sci. Adv\", \"type\": \"ref\", \"volume\": \"4\", \"year\": \"2018\"}, \"text\": \"MHC class I in dopaminergic neurons suppresses relapse to reward seeking\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 64976, \"infons\": {\"fpage\": \"e0213069\", \"issue\": \"4\", \"name_0\": \"surname:Naik;given-names:J\", \"name_1\": \"surname:Hau;given-names:CM\", \"name_10\": \"surname:Nieuwland;given-names:R\", \"name_11\": \"surname:Paulusma;given-names:CC\", \"name_12\": \"surname:Bosma;given-names:PJ\", \"name_13\": \"surname:Hakami;given-names:RM\", \"name_2\": \"surname:ten Bloemendaal;given-names:L\", \"name_3\": \"surname:Mok;given-names:KS\", \"name_4\": \"surname:Hajji;given-names:N\", \"name_5\": \"surname:Wehman;given-names:AM\", \"name_6\": \"surname:Meisner;given-names:S\", \"name_7\": \"surname:Muncan;given-names:V\", \"name_8\": \"surname:Paauw;given-names:NJ\", \"name_9\": \"surname:de Vries;given-names:HE\", \"pub-id_pmid\": \"30947313\", \"section_type\": \"REF\", \"source\": \"PLoS One\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2019\"}, \"text\": \"The P4-ATPase ATP9A is a novel determinant of exosome release\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65038, \"infons\": {\"fpage\": \"2781\", \"lpage\": \"2787\", \"name_0\": \"surname:Namiki;given-names:S\", \"name_1\": \"surname:Nakamura;given-names:T\", \"name_2\": \"surname:Oshima;given-names:S\", \"name_3\": \"surname:Yamazaki;given-names:M\", \"name_4\": \"surname:Sekine;given-names:Y\", \"name_5\": \"surname:Tsuchiya;given-names:K\", \"name_6\": \"surname:Okamoto;given-names:R\", \"name_7\": \"surname:Kanai;given-names:T\", \"name_8\": \"surname:Watanabe;given-names:M\", \"pub-id_doi\": \"10.1016/j.febslet.2005.04.012\", \"pub-id_pmid\": \"15907481\", \"section_type\": \"REF\", \"source\": \"FEBS Lett\", \"type\": \"ref\", \"volume\": \"579\", \"year\": \"2005\"}, \"text\": \"IRF-1 mediates upregulation of LMP7 by IFN-gamma and concerted expression of immunosubunits of the proteasome\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65148, \"infons\": {\"fpage\": \"305\", \"lpage\": \"316\", \"name_0\": \"surname:Neumann;given-names:H\", \"name_1\": \"surname:Schmidt;given-names:H\", \"name_2\": \"surname:Cavali\\u00e9;given-names:A\", \"name_3\": \"surname:Jenne;given-names:D\", \"name_4\": \"surname:Wekerle;given-names:H\", \"pub-id_doi\": \"10.1084/jem.185.2.305\", \"pub-id_pmid\": \"9016879\", \"section_type\": \"REF\", \"source\": \"J. Exp. Med\", \"type\": \"ref\", \"volume\": \"185\", \"year\": \"1997\"}, \"text\": \"Major histocompatibility complex (MHC) class I gene expression in single neurons of the central nervous system: differential regulation by interferon (IFN)-gamma and tumor necrosis factor (TNF)-alpha\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65348, \"infons\": {\"fpage\": \"19\", \"lpage\": \"35\", \"name_0\": \"surname:Popko;given-names:B\", \"name_1\": \"surname:Corbin;given-names:JG\", \"name_2\": \"surname:Baerwald;given-names:KD\", \"name_3\": \"surname:Dupree;given-names:J\", \"name_4\": \"surname:Garcia;given-names:AM\", \"pub-id_doi\": \"10.1007/BF02740619\", \"pub-id_pmid\": \"9170099\", \"section_type\": \"REF\", \"source\": \"Mol. Neurobiol\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"1997\"}, \"text\": \"The effects of interferon-gamma on the central nervous system\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65410, \"infons\": {\"fpage\": \"930\", \"lpage\": \"943\", \"name_0\": \"surname:Poulin;given-names:J-F\", \"name_1\": \"surname:Zou;given-names:J\", \"name_2\": \"surname:Drouin-Ouellet;given-names:J\", \"name_3\": \"surname:Kim;given-names:K-Y-A\", \"name_4\": \"surname:Cicchetti;given-names:F\", \"name_5\": \"surname:Awatramani;given-names:RB\", \"pub-id_doi\": \"10.1016/j.celrep.2014.10.008\", \"pub-id_pmid\": \"25437550\", \"section_type\": \"REF\", \"source\": \"Cell Rep\", \"type\": \"ref\", \"volume\": \"9\", \"year\": \"2014\"}, \"text\": \"Defining midbrain dopaminergic neuron diversity by single-cell gene expression profiling\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65499, \"infons\": {\"fpage\": \"115\", \"lpage\": \"134\", \"name_0\": \"surname:Rall;given-names:GF\", \"name_1\": \"surname:Whitton;given-names:JL\", \"pub-id_doi\": \"10.1007/978-3-642-72045-1_6\", \"section_type\": \"REF\", \"source\": \"Antigen Presentation, Current Topics in Microbiology and Immunology\", \"type\": \"ref\", \"year\": \"1998\"}, \"text\": \"\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65500, \"infons\": {\"fpage\": \"431\", \"lpage\": \"439\", \"name_0\": \"surname:Rico-Bautista;given-names:E\", \"name_1\": \"surname:Flores-Morales;given-names:A\", \"name_2\": \"surname:Fern\\u00e1ndez-P\\u00e9rez;given-names:L\", \"pub-id_doi\": \"10.1016/j.cytogfr.2006.09.008\", \"pub-id_pmid\": \"17070092\", \"section_type\": \"REF\", \"source\": \"Cytokine Growth Factor Rev\", \"type\": \"ref\", \"volume\": \"17\", \"year\": \"2006\"}, \"text\": \"Suppressor of cytokine signaling (SOCS) 2, a protein with multiple functions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65577, \"infons\": {\"fpage\": \"44\", \"lpage\": \"53\", \"name_0\": \"surname:Robertson;given-names:IB\", \"name_1\": \"surname:Horiguchi;given-names:M\", \"name_2\": \"surname:Zilberberg;given-names:L\", \"name_3\": \"surname:Dabovic;given-names:B\", \"name_4\": \"surname:Hadjiolova;given-names:K\", \"name_5\": \"surname:Rifkin;given-names:DB\", \"pub-id_doi\": \"10.1016/j.matbio.2015.05.005\", \"pub-id_pmid\": \"25960419\", \"section_type\": \"REF\", \"source\": \"Matrix Biol\", \"type\": \"ref\", \"volume\": \"47\", \"year\": \"2015\"}, \"text\": \"Latent TGF-\\u03b2-binding proteins\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65610, \"infons\": {\"fpage\": \"145\", \"lpage\": \"156\", \"name_0\": \"surname:Rose;given-names:RW\", \"name_1\": \"surname:Vorobyeva;given-names:AG\", \"name_2\": \"surname:Skipworth;given-names:JD\", \"name_3\": \"surname:Nicolas;given-names:E\", \"name_4\": \"surname:Rall;given-names:GF\", \"pub-id_doi\": \"10.1016/j.jneuroim.2007.10.007\", \"pub-id_pmid\": \"18006082\", \"section_type\": \"REF\", \"source\": \"J. Neuroimmunol\", \"type\": \"ref\", \"volume\": \"192\", \"year\": \"2007\"}, \"text\": \"Altered Levels of STAT1 and STAT3 influence the neuronal response to interferon Gamma\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65696, \"infons\": {\"fpage\": \"119\", \"name_0\": \"surname:Rostami;given-names:J\", \"name_1\": \"surname:Fotaki;given-names:G\", \"name_2\": \"surname:Sirois;given-names:J\", \"name_3\": \"surname:Mzezewa;given-names:R\", \"name_4\": \"surname:Bergstr\\u00f6m;given-names:J\", \"name_5\": \"surname:Essand;given-names:M\", \"name_6\": \"surname:Healy;given-names:L\", \"name_7\": \"surname:Erlandsson;given-names:A\", \"pub-id_doi\": \"10.1186/s12974-020-01776-7\", \"section_type\": \"REF\", \"source\": \"J. Neuroinflamm\", \"type\": \"ref\", \"volume\": \"17\", \"year\": \"2020\"}, \"text\": \"Astrocytes have the capacity to act as antigen-presenting cells in the Parkinson\\u2019s disease brain\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65795, \"infons\": {\"fpage\": \"43\", \"lpage\": \"52\", \"name_0\": \"surname:Sadakata;given-names:T\", \"name_1\": \"surname:Mizoguchi;given-names:A\", \"name_2\": \"surname:Sato;given-names:Y\", \"name_3\": \"surname:Katoh-Semba;given-names:R\", \"name_4\": \"surname:Fukuda;given-names:M\", \"name_5\": \"surname:Mikoshiba;given-names:K\", \"name_6\": \"surname:Furuichi;given-names:T\", \"pub-id_doi\": \"10.1523/JNEUROSCI.2528-03.2004\", \"pub-id_pmid\": \"14715936\", \"section_type\": \"REF\", \"source\": \"J. Neurosci\", \"type\": \"ref\", \"volume\": \"24\", \"year\": \"2004\"}, \"text\": \"The secretory granule-associated protein CAPS2 regulates neurotrophin release and cell survival\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65891, \"infons\": {\"fpage\": \"D852\", \"lpage\": \"857\", \"name_0\": \"surname:Samarajiwa;given-names:SA\", \"name_1\": \"surname:Forster;given-names:S\", \"name_2\": \"surname:Auchettl;given-names:K\", \"name_3\": \"surname:Hertzog;given-names:PJ\", \"pub-id_doi\": \"10.1093/nar/gkn732\", \"pub-id_pmid\": \"18996892\", \"section_type\": \"REF\", \"source\": \"Nucleic Acids Res\", \"type\": \"ref\", \"volume\": \"37\", \"year\": \"2009\"}, \"text\": \"INTERFEROME: the database of interferon regulated genes\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 65947, \"infons\": {\"fpage\": \"4622\", \"lpage\": \"4643\", \"name_0\": \"surname:Santos;given-names:SED\", \"name_1\": \"surname:Medeiros;given-names:M\", \"name_10\": \"surname:Noctor;given-names:SC\", \"name_11\": \"surname:Sherwood;given-names:CC\", \"name_12\": \"surname:Kaas;given-names:JH\", \"name_13\": \"surname:Manger;given-names:PR\", \"name_14\": \"surname:Herculano-Houzel;given-names:S\", \"name_2\": \"surname:Porfirio;given-names:J\", \"name_3\": \"surname:Tavares;given-names:W\", \"name_4\": \"surname:Pess\\u00f4a;given-names:L\", \"name_5\": \"surname:Grinberg;given-names:L\", \"name_6\": \"surname:Leite;given-names:REP\", \"name_7\": \"surname:Ferretti-Rebustini;given-names:REL\", \"name_8\": \"surname:Suemoto;given-names:CK\", \"name_9\": \"surname:Filho;given-names:WJ\", \"pub-id_doi\": \"10.1523/JNEUROSCI.2339-19.2020\", \"pub-id_pmid\": \"32253358\", \"section_type\": \"REF\", \"source\": \"J. Neurosci\", \"type\": \"ref\", \"volume\": \"40\", \"year\": \"2020\"}, \"text\": \"Similar microglial cell densities across brain structures and mammalian species: implications for brain tissue function\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66067, \"infons\": {\"fpage\": \"1015\", \"issue\": \"4\", \"lpage\": \"1030. e16\", \"name_0\": \"surname:Saunders;given-names:A\", \"name_1\": \"surname:Macosko;given-names:EZ\", \"name_10\": \"surname:Goeva;given-names:A\", \"name_11\": \"surname:Nemesh;given-names:J\", \"name_12\": \"surname:Kamitaki;given-names:N\", \"name_13\": \"surname:Brumbaugh;given-names:S\", \"name_14\": \"surname:Kulp;given-names:D\", \"name_15\": \"surname:McCarroll;given-names:SA\", \"name_2\": \"surname:Wysoker;given-names:A\", \"name_3\": \"surname:Goldman;given-names:M\", \"name_4\": \"surname:Krienen;given-names:FM\", \"name_5\": \"surname:de Rivera;given-names:H\", \"name_6\": \"surname:Bien;given-names:E\", \"name_7\": \"surname:Baum;given-names:M\", \"name_8\": \"surname:Bortolin;given-names:L\", \"name_9\": \"surname:Wang;given-names:S\", \"pub-id_pmid\": \"30096299\", \"section_type\": \"REF\", \"source\": \"Cell\", \"type\": \"ref\", \"volume\": \"174\", \"year\": \"2018\"}, \"text\": \"Molecular diversity and specializations among the cells of the adult mouse brain\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66148, \"infons\": {\"fpage\": \"163\", \"lpage\": \"189\", \"name_0\": \"surname:Schroder;given-names:K\", \"name_1\": \"surname:Hertzog;given-names:PJ\", \"name_2\": \"surname:Ravasi;given-names:T\", \"name_3\": \"surname:Hume;given-names:DA\", \"pub-id_doi\": \"10.1189/jlb.0603252\", \"pub-id_pmid\": \"14525967\", \"section_type\": \"REF\", \"source\": \"Journal of Leukocyte Biology\", \"type\": \"ref\", \"volume\": \"75\", \"year\": \"2004\"}, \"text\": \"Interferon-\\u03b3: an overview of signals, mechanisms and functions\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66215, \"infons\": {\"fpage\": \"1308\", \"lpage\": \"1323.e30\", \"name_0\": \"surname:Shekhar;given-names:K\", \"name_1\": \"surname:Lapan;given-names:SW\", \"name_10\": \"surname:McCarroll;given-names:SA\", \"name_11\": \"surname:Cepko;given-names:CL\", \"name_12\": \"surname:Regev;given-names:A\", \"name_13\": \"surname:Sanes;given-names:JR\", \"name_2\": \"surname:Whitney;given-names:IE\", \"name_3\": \"surname:Tran;given-names:NM\", \"name_4\": \"surname:Macosko;given-names:EZ\", \"name_5\": \"surname:Kowalczyk;given-names:M\", \"name_6\": \"surname:Adiconis;given-names:X\", \"name_7\": \"surname:Levin;given-names:JZ\", \"name_8\": \"surname:Nemesh;given-names:J\", \"name_9\": \"surname:Goldman;given-names:M\", \"pub-id_doi\": \"10.1016/j.cell.2016.07.054\", \"pub-id_pmid\": \"27565351\", \"section_type\": \"REF\", \"source\": \"Cell\", \"type\": \"ref\", \"volume\": \"166\", \"year\": \"2016\"}, \"text\": \"Comprehensive classification of retinal bipolar neurons by single-cell transcriptomics\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66302, \"infons\": {\"fpage\": \"1744\", \"lpage\": \"1746\", \"name_0\": \"surname:Shuai;given-names:K\", \"name_1\": \"surname:Stark;given-names:GR\", \"name_2\": \"surname:Kerr;given-names:IM\", \"name_3\": \"surname:Darnell;given-names:JE\", \"pub-id_doi\": \"10.1126/science.7690989\", \"pub-id_pmid\": \"7690989\", \"section_type\": \"REF\", \"source\": \"Science\", \"type\": \"ref\", \"volume\": \"261\", \"year\": \"1993\"}, \"text\": \"A single phosphotyrosine residue of Stat91 required for gene activation by interferon-gamma\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66394, \"infons\": {\"name_0\": \"surname:Snyder;given-names:ME\", \"name_1\": \"surname:Finlayson;given-names:MO\", \"name_10\": \"surname:Robbins;given-names:H\", \"name_11\": \"surname:Hook;given-names:JL\", \"name_12\": \"surname:Sykes;given-names:M\", \"name_13\": \"surname:D\\u2019Ovidio;given-names:F\", \"name_14\": \"surname:Bacchetta;given-names:M\", \"name_15\": \"surname:Sonett;given-names:JR\", \"name_16\": \"surname:Lederer;given-names:DJ\", \"name_17\": \"surname:Arcasoy;given-names:S\", \"name_18\": \"surname:Sims;given-names:PA\", \"name_19\": \"surname:Farber;given-names:DL\", \"name_2\": \"surname:Connors;given-names:TJ\", \"name_3\": \"surname:Dogra;given-names:P\", \"name_4\": \"surname:Senda;given-names:T\", \"name_5\": \"surname:Bush;given-names:E\", \"name_6\": \"surname:Carpenter;given-names:D\", \"name_7\": \"surname:Marboe;given-names:C\", \"name_8\": \"surname:Benvenuto;given-names:L\", \"name_9\": \"surname:Shah;given-names:L\", \"pub-id_doi\": \"10.1126/sciimmunol.aav5581\", \"section_type\": \"REF\", \"source\": \"Sci. Immunol\", \"type\": \"ref\", \"volume\": \"4\", \"year\": \"2019\"}, \"text\": \"Generation and persistence of human tissue-resident memory T cells in lung transplantation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66485, \"infons\": {\"fpage\": \"123\", \"lpage\": \"131.e6\", \"name_0\": \"surname:Sommer;given-names:A\", \"name_1\": \"surname:Marxreiter;given-names:F\", \"name_10\": \"surname:Kohl;given-names:Z\", \"name_11\": \"surname:Xiang;given-names:W\", \"name_12\": \"surname:Gage;given-names:FH\", \"name_13\": \"surname:Winkler;given-names:J\", \"name_14\": \"surname:Prots;given-names:I\", \"name_15\": \"surname:Winner;given-names:B\", \"name_2\": \"surname:Krach;given-names:F\", \"name_3\": \"surname:Fadler;given-names:T\", \"name_4\": \"surname:Grosch;given-names:J\", \"name_5\": \"surname:Maroni;given-names:M\", \"name_6\": \"surname:Graef;given-names:D\", \"name_7\": \"surname:Eberhardt;given-names:E\", \"name_8\": \"surname:Riemenschneider;given-names:MJ\", \"name_9\": \"surname:Yeo;given-names:GW\", \"pub-id_doi\": \"10.1016/j.stem.2018.06.015\", \"pub-id_pmid\": \"29979986\", \"section_type\": \"REF\", \"source\": \"Cell Stem Cell\", \"type\": \"ref\", \"volume\": \"23\", \"year\": \"2018\"}, \"text\": \"Th17 lymphocytes induce neuronal cell death in a human iPSC-based model of parkinson\\u2019s disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66582, \"infons\": {\"fpage\": \"133\", \"lpage\": \"141\", \"name_0\": \"surname:Strickland;given-names:MR\", \"name_1\": \"surname:Koller;given-names:EJ\", \"name_2\": \"surname:Deng;given-names:DZ\", \"name_3\": \"surname:Ceballos-Diaz;given-names:C\", \"name_4\": \"surname:Golde;given-names:TE\", \"name_5\": \"surname:Chakrabarty;given-names:P\", \"pub-id_doi\": \"10.1016/j.nbd.2017.11.007\", \"pub-id_pmid\": \"29196213\", \"section_type\": \"REF\", \"source\": \"Neurobiol. Dis\", \"type\": \"ref\", \"volume\": \"110\", \"year\": \"2018\"}, \"text\": \"Ifngr1 and Stat1 mediated canonical Ifn-\\u03b3 signaling drives nigrostriatal degeneration\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66672, \"infons\": {\"fpage\": \"6720\", \"lpage\": \"6732\", \"name_0\": \"surname:St\\u00fcve;given-names:O\", \"name_1\": \"surname:Youssef;given-names:S\", \"name_10\": \"surname:Zamvil;given-names:SS\", \"name_2\": \"surname:Slavin;given-names:AJ\", \"name_3\": \"surname:King;given-names:CL\", \"name_4\": \"surname:Patarroyo;given-names:JC\", \"name_5\": \"surname:Hirschberg;given-names:DL\", \"name_6\": \"surname:Brickey;given-names:WJ\", \"name_7\": \"surname:Soos;given-names:JM\", \"name_8\": \"surname:Piskurich;given-names:JF\", \"name_9\": \"surname:Chapman;given-names:HA\", \"pub-id_doi\": \"10.4049/jimmunol.169.12.6720\", \"pub-id_pmid\": \"12471103\", \"section_type\": \"REF\", \"source\": \"The Journal of Immunology\", \"type\": \"ref\", \"volume\": \"169\", \"year\": \"2002\"}, \"text\": \"The Role of the MHC Class II Transactivator in Class II Expression and Antigen Presentation by Astrocytes and in Susceptibility to Central Nervous System Autoimmune Disease\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66845, \"infons\": {\"fpage\": \"656\", \"lpage\": \"661\", \"name_0\": \"surname:Sulzer;given-names:D\", \"name_1\": \"surname:Alcalay;given-names:RN\", \"name_10\": \"surname:Dawson;given-names:VL\", \"name_11\": \"surname:Dawson;given-names:TM\", \"name_12\": \"surname:Oseroff;given-names:C\", \"name_13\": \"surname:Pham;given-names:J\", \"name_14\": \"surname:Sidney;given-names:J\", \"name_15\": \"surname:Dillon;given-names:MB\", \"name_16\": \"surname:Carpenter;given-names:C\", \"name_17\": \"surname:Weiskopf;given-names:D\", \"name_18\": \"surname:Phillips;given-names:E\", \"name_19\": \"surname:Mallal;given-names:S\", \"name_2\": \"surname:Garretti;given-names:F\", \"name_20\": \"surname:Peters;given-names:B\", \"name_21\": \"surname:Frazier;given-names:A\", \"name_22\": \"surname:Lindestam Arlehamn;given-names:CS\", \"name_23\": \"surname:Sette;given-names:A\", \"name_3\": \"surname:Cote;given-names:L\", \"name_4\": \"surname:Kanter;given-names:E\", \"name_5\": \"surname:Agin-Liebes;given-names:J\", \"name_6\": \"surname:Liong;given-names:C\", \"name_7\": \"surname:McMurtrey;given-names:C\", \"name_8\": \"surname:Hildebrand;given-names:WH\", \"name_9\": \"surname:Mao;given-names:X\", \"pub-id_doi\": \"10.1038/nature22815\", \"pub-id_pmid\": \"28636593\", \"section_type\": \"REF\", \"source\": \"Nature\", \"type\": \"ref\", \"volume\": \"546\", \"year\": \"2017\"}, \"text\": \"T cells from patients with Parkinson\\u2019s disease recognize \\u03b1-synuclein peptides\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 66927, \"infons\": {\"fpage\": \"1113\", \"lpage\": \"1121\", \"name_0\": \"surname:Svensson;given-names:M\", \"name_1\": \"surname:Marsal;given-names:J\", \"name_2\": \"surname:Ericsson;given-names:A\", \"name_3\": \"surname:Carramolino;given-names:L\", \"name_4\": \"surname:Brod\\u00e9n;given-names:T\", \"name_5\": \"surname:M\\u00e1rquez;given-names:G\", \"name_6\": \"surname:Agace;given-names:WW\", \"pub-id_doi\": \"10.1172/JCI15988\", \"pub-id_pmid\": \"12393847\", \"section_type\": \"REF\", \"source\": \"J. Clin. Invest\", \"type\": \"ref\", \"volume\": \"110\", \"year\": \"2002\"}, \"text\": \"CCL25 mediates the localization of recently activated CD8alphabeta(+) lymphocytes to the small-intestinal mucosa\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67040, \"infons\": {\"fpage\": \"3883\", \"issue\": \"24\", \"lpage\": \"3893\", \"name_0\": \"surname:Tanaka;given-names:Y\", \"name_1\": \"surname:Ono;given-names:N\", \"name_2\": \"surname:Shima;given-names:T\", \"name_3\": \"surname:Tanaka;given-names:G\", \"name_4\": \"surname:Katoh;given-names:Y\", \"name_5\": \"surname:Nakayama;given-names:K\", \"name_6\": \"surname:Takatsu;given-names:H\", \"name_7\": \"surname:Shin;given-names:H-W\", \"name_8\": \"surname:Nakano;given-names:A\", \"pub-id_pmid\": \"27733620\", \"section_type\": \"REF\", \"source\": \"Mol. Biol. Cell\", \"type\": \"ref\", \"volume\": \"27\", \"year\": \"2016\"}, \"text\": \"The phospholipid flippase ATP9A is required for the recycling pathway from the endosomes to the plasma membrane\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67152, \"infons\": {\"fpage\": \"1099\", \"lpage\": \"1107\", \"name_0\": \"surname:Taylor;given-names:GA\", \"pub-id_doi\": \"10.1111/j.1462-5822.2007.00916.X\", \"pub-id_pmid\": \"17359233\", \"section_type\": \"REF\", \"source\": \"Cell. Microbiol\", \"type\": \"ref\", \"volume\": \"9\", \"year\": \"2007\"}, \"text\": \"IRG proteins: key mediators of interferon-regulated host resistance to intracellular pathogens\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67247, \"infons\": {\"fpage\": \"307\", \"lpage\": \"319\", \"name_0\": \"surname:Ugras;given-names:S\", \"name_1\": \"surname:Daniels;given-names:MJ\", \"name_10\": \"surname:Martinez;given-names:D\", \"name_11\": \"surname:Evans;given-names:P\", \"name_12\": \"surname:Brown;given-names:D\", \"name_13\": \"surname:Duda;given-names:JE\", \"name_14\": \"surname:Ischiropoulos;given-names:H\", \"name_2\": \"surname:Fazelinia;given-names:H\", \"name_3\": \"surname:Gould;given-names:NS\", \"name_4\": \"surname:Yocum;given-names:AK\", \"name_5\": \"surname:Luk;given-names:KC\", \"name_6\": \"surname:Luna;given-names:E\", \"name_7\": \"surname:Ding;given-names:H\", \"name_8\": \"surname:McKennan;given-names:C\", \"name_9\": \"surname:Seeholzer;given-names:S\", \"pub-id_doi\": \"10.1016/j.ebiom.2018.05.007\", \"pub-id_pmid\": \"29759483\", \"section_type\": \"REF\", \"source\": \"EBioMedicine\", \"type\": \"ref\", \"volume\": \"31\", \"year\": \"2018\"}, \"text\": \"Induction of the immunoproteasome subunit Lmp7 links proteostasis and immunity in \\u03b1-synuclein aggregation disorders\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67367, \"infons\": {\"fpage\": \"67\", \"lpage\": \"75\", \"name_0\": \"surname:van den Elsen;given-names:PJ\", \"name_1\": \"surname:Holling;given-names:TM\", \"name_2\": \"surname:Kuipers;given-names:HF\", \"name_3\": \"surname:van der Stoep;given-names:N\", \"pub-id_doi\": \"10.1016/j.coi.2003.11.015\", \"pub-id_pmid\": \"14734112\", \"section_type\": \"REF\", \"source\": \"Curr. Opin. Immunol\", \"type\": \"ref\", \"volume\": \"16\", \"year\": \"2004\"}, \"text\": \"Transcriptional regulation of antigen presentation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67418, \"infons\": {\"fpage\": \"789\", \"lpage\": \"800\", \"name_0\": \"surname:Vass;given-names:K\", \"name_1\": \"surname:Lassmann;given-names:H\", \"pub-id_pmid\": \"2121041\", \"section_type\": \"REF\", \"source\": \"Am. J. Pathol\", \"type\": \"ref\", \"volume\": \"137\", \"year\": \"1990\"}, \"text\": \"Intrathecal application of interferon gamma. Progressive appearance of MHC antigens within the rat nervous system\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67532, \"infons\": {\"fpage\": \"22\", \"lpage\": \"29\", \"name_0\": \"surname:Wang;given-names:F\", \"name_1\": \"surname:Flanagan;given-names:J\", \"name_2\": \"surname:Su;given-names:N\", \"name_3\": \"surname:Wang;given-names:L-C\", \"name_4\": \"surname:Bui;given-names:S\", \"name_5\": \"surname:Nielson;given-names:A\", \"name_6\": \"surname:Wu;given-names:X\", \"name_7\": \"surname:Vo;given-names:H-T\", \"name_8\": \"surname:Ma;given-names:X-J\", \"name_9\": \"surname:Luo;given-names:Y\", \"pub-id_doi\": \"10.1016/j.jmoldx.2011.08.002\", \"pub-id_pmid\": \"22166544\", \"section_type\": \"REF\", \"source\": \"J. Mol. Diagn\", \"type\": \"ref\", \"volume\": \"14\", \"year\": \"2012\"}, \"text\": \"RNAscope: A novel in Situ RNA analysis platform for formalin-fixed, paraffin-embedded tissues\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67626, \"infons\": {\"fpage\": \"e90\", \"name_0\": \"surname:Xie;given-names:Z\", \"name_1\": \"surname:Bailey;given-names:A\", \"name_10\": \"surname:Jeon;given-names:M\", \"name_11\": \"surname:Ma\\u2019ayan;given-names:A\", \"name_2\": \"surname:Kuleshov;given-names:MV\", \"name_3\": \"surname:Clarke;given-names:DJB\", \"name_4\": \"surname:Evangelista;given-names:JE\", \"name_5\": \"surname:Jenkins;given-names:SL\", \"name_6\": \"surname:Lachmann;given-names:A\", \"name_7\": \"surname:Wojciechowicz;given-names:ML\", \"name_8\": \"surname:Kropiwnicki;given-names:E\", \"name_9\": \"surname:Jagodnik;given-names:KM\", \"pub-id_doi\": \"10.1002/cpz1.90\", \"pub-id_pmid\": \"33780170\", \"section_type\": \"REF\", \"source\": \"Current Protocols\", \"type\": \"ref\", \"volume\": \"1\", \"year\": \"2021\"}, \"text\": \"Gene Set Knowledge Discovery with Enrichr\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67668, \"infons\": {\"fpage\": \"2923\", \"lpage\": \"2934\", \"name_0\": \"surname:Yshii;given-names:LM\", \"name_1\": \"surname:Gebauer;given-names:CM\", \"name_10\": \"surname:Liblau;given-names:R\", \"name_2\": \"surname:Pignolet;given-names:B\", \"name_3\": \"surname:Maur\\u00e9;given-names:E\", \"name_4\": \"surname:Qu\\u00e9riault;given-names:C\", \"name_5\": \"surname:Pierau;given-names:M\", \"name_6\": \"surname:Saito;given-names:H\", \"name_7\": \"surname:Suzuki;given-names:N\", \"name_8\": \"surname:Brunner-Weinzierl;given-names:M\", \"name_9\": \"surname:Bauer;given-names:J\", \"pub-id_doi\": \"10.1093/brain/aww225\", \"pub-id_pmid\": \"27604307\", \"section_type\": \"REF\", \"source\": \"Brain\", \"type\": \"ref\", \"volume\": \"139\", \"year\": \"2016\"}, \"text\": \"CTLA4 blockade elicits paraneoplastic neurological disease in a mouse model\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67744, \"infons\": {\"fpage\": \"57\", \"name_0\": \"surname:Yuan;given-names:J\", \"name_1\": \"surname:Levitin;given-names:HM\", \"name_10\": \"surname:Bruce;given-names:JN\", \"name_11\": \"surname:Lasorella;given-names:A\", \"name_12\": \"surname:Iavarone;given-names:A\", \"name_13\": \"surname:Sims;given-names:PA\", \"name_2\": \"surname:Frattini;given-names:V\", \"name_3\": \"surname:Bush;given-names:EC\", \"name_4\": \"surname:Boyett;given-names:DM\", \"name_5\": \"surname:Samanamud;given-names:J\", \"name_6\": \"surname:Ceccarelli;given-names:M\", \"name_7\": \"surname:Dovas;given-names:A\", \"name_8\": \"surname:Zanazzi;given-names:G\", \"name_9\": \"surname:Canoll;given-names:P\", \"pub-id_doi\": \"10.1186/s13073-018-0567-9\", \"pub-id_pmid\": \"30041684\", \"section_type\": \"REF\", \"source\": \"Genome Med\", \"type\": \"ref\", \"volume\": \"10\", \"year\": \"2018\"}, \"text\": \"Single-cell transcriptome analysis of lineage diversity in high-grade glioma\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67821, \"infons\": {\"fpage\": \"82\", \"name_0\": \"surname:Zhao;given-names:W\", \"name_1\": \"surname:Dovas;given-names:A\", \"name_10\": \"surname:Bruce;given-names:JN\", \"name_11\": \"surname:Canoll;given-names:P\", \"name_12\": \"surname:Sims;given-names:PA\", \"name_2\": \"surname:Spinazzi;given-names:EF\", \"name_3\": \"surname:Levitin;given-names:HM\", \"name_4\": \"surname:Banu;given-names:MA\", \"name_5\": \"surname:Upadhyayula;given-names:P\", \"name_6\": \"surname:Sudhakar;given-names:T\", \"name_7\": \"surname:Marie;given-names:T\", \"name_8\": \"surname:Otten;given-names:ML\", \"name_9\": \"surname:Sisti;given-names:MB\", \"pub-id_doi\": \"10.1186/s13073-021-00894-y\", \"pub-id_pmid\": \"33975634\", \"section_type\": \"REF\", \"source\": \"Genome Med\", \"type\": \"ref\", \"volume\": \"13\", \"year\": \"2021\"}, \"text\": \"Deconvolution of cell type-specific drug responses in human tumor tissue with single-cell RNA-seq\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 67919, \"infons\": {\"fpage\": \"239\", \"lpage\": \"260\", \"name_0\": \"surname:Zhou;given-names:F\", \"pub-id_doi\": \"10.1080/08830180902978120\", \"pub-id_pmid\": \"19811323\", \"section_type\": \"REF\", \"source\": \"Int. Rev. Immunol\", \"type\": \"ref\", \"volume\": \"28\", \"year\": \"2009\"}, \"text\": \"Molecular mechanisms of IFN-gamma to up-regulate MHC class I antigen processing and presentation\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 68016, \"infons\": {\"file\": \"nihms-1900921-f0001.jpg\", \"id\": \"F1\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Core transcriptional responses to IFN-\\u03b3 are conserved in neurons but are lower in magnitude compared to glia (a) Experimental schematic for NeuN\\u2212/NeuN + RNA-seq after injection of saline or IFN-\\u03b3 in the midbrain of WT C57BL/6J mice. (b) Principal component analysis (PCA) of RNA-seq UMI counts normalized with DESeq2 variance stabilizing transformation. Each point represents a bulk RNA-seq replicate of ~ 1000 sorted nuclei (n = 4\\u20136 replicates each of NeuN\\u2212 or NeuN + nuclei from 2 to 3 mice for each treatment/timepoint). (c) Number of differentially expressed genes (DEG) for IFN-\\u03b3 vs. Sal comparisons in NeuN\\u2212 or NeuN + samples, or both (overlap), at each timepoint (DESeq2, \\u2223log2FC\\u2223 > 1 and pAdj < 0.01). (d) Mean \\u00b1 SEM for mRNA abundance in NeuN\\u2212/NeuN + samples, normalized as log2 (CPM + 1). Samples as in (b), (n = 4\\u20136 replicates each of NeuN\\u2212/+ nuclei from 2 to 3 mice for each treatment/timepoint). Saline samples from all timepoints are together at t = 0. (e) Heatmap of z-scored, normalized mRNA abundance (log2[CPM + 1]) for 28 mRNAs upregulated at \\u2265 3 timepoints in both NeuN\\u2212 and NeuN + samples (see Supp. Fig. 1d). Samples as in (b); each column is a replicate. (f) Scatter plots comparing the effect of IFN-\\u03b3 (log2 fold change) in NeuN + vs. NeuN\\u2212 nuclei for all IFN-\\u03b3-induced genes at each timepoint (DESeq2, log2FC > 1 and pAdj < 0.01). Points are colored by the DESeq2 log2FC comparing IFN-\\u03b3-treated NeuN + vs. NeuN\\u2212 samples: values \\u22121 to \\u22123 correspond correspond to 2- and 8-fold higher relative expression in NeuN\\u2212 nuclei, respectively.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 69614, \"infons\": {\"file\": \"nihms-1900921-f0002.jpg\", \"id\": \"F2\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Divergent transcriptional responses to IFN-\\u03b3 in glia (a) Summary of DEGs from interaction analysis (DESeq2: ~NeuN + IFN + NeuN:IFN), broken down by baseline expression and effect of IFN-\\u03b3 in NeuN + and NeuN\\u2212 samples. (b) Heatmap of z-scored mRNA abundances for genes upregulated in NeuN\\u2212 samples and significant in NeuN \\u00d7 IFN interaction analysis in (a), normalized as log2(CPM + 1). Each column represents a bulk RNA-seq replicate of ~ 1000 sorted nuclei (n = 4\\u20136 replicates from 2 to 3 mice for each treatment/timepoint). (c) Mean \\u00b1 SEM for mRNA abundance in NeuN\\u2212/NeuN + samples, normalized as log2(CPM + 1). Samples as in (b), (n = 4\\u20136 replicates each of NeuN\\u2212/+ nuclei from 2 to 3 mice for each treatment/timepoint). All saline samples are together at t = 0. (d) Heatmap of z-scored mRNA abundances for genes downregulated in NeuN\\u2212 samples and significant in NeuN \\u00d7 IFN interaction analysis in (a), normalized as log2(CPM + 1). Each column represents a bulk RNA-seq replicate of ~ 1000 sorted nuclei (n = 4\\u20136 replicates from 2 to 3 mice for each treatment/timepoint).\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 70708, \"infons\": {\"file\": \"nihms-1900921-f0003.jpg\", \"id\": \"F3\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Single nucleus RNA-seq analysis of responses to IFN-\\u03b3. (a) UMAP embedding of snRNA-seq profiles from the ventral midbrain used for downstream analysis (4,699 nuclei after collapse to major cell types and filtering, see Methods). Left: major cell types for downstream analysis, see Supp. Fig. 4b for clustering with marker genes. Right: mouse sample origin (n = 1 for saline, n = 2 for IFN-\\u03b3). (b) Box and whiskers plots depicting the sum of mRNA abundance (log2[CPM + 1]) in each major cell group for the indicated sets of MHC-I or MHC-II genes. (c) Clustered heatmap of z-scored mRNA abundance (log2[CPM + 1]) for the union of highly differentially expressed genes across all major cell groups (modified Mann-Whitney U test comparing IFN vs. Sal, \\u2223log2FC\\u2223 > 4 and q < 0.05). (d) Heatmap of average mRNA abundance (log2[CPM + 1]) for genes in MGI GO:0060335: \\u2018positive regulation of interferon-gamma-mediated signaling pathway\\u2019. (e) Upper: UMAP embedding of dopamine neuronal snRNA-seq profiles with subcluster IDs and mouse sample origin. Lower: Heatmap of average mRNA abundance (log2[CPM + 1]) for select IFN response genes within dopamine neuronal subclusters from each mouse sample.\", \"sentences\": [], \"annotations\": [], \"relations\": []}, {\"offset\": 71906, \"infons\": {\"file\": \"nihms-1900921-f0004.jpg\", \"id\": \"F4\", \"section_type\": \"FIG\", \"type\": \"fig_caption\"}, \"text\": \"Ifngr1 is required for pSTAT1 induction and MHC-I upregulation in mDA neurons after in vivo exposure to IFN-\\u03b3 (a) TH and pSTAT1-Y701 immunofluorescence in the SNc of WT:Ifngr1fl/fl or DAT-Cre:Ifngr1fl/fl mice after saline or IFN-\\u03b3. White arrowheads indicate TH+ mDA neurons, blue arrows indicate NeuN\\u2212 nuclei, and yellow arrows indicate TH\\u2212 neurons. Scale bar: 50 \\u03bcm. (b) pSTAT1-Y701 in TH+/NeuN+ mDA neurons, TH\\u2212/NeuN+ non-DA neurons, and TH\\u2212/NeuN\\u2212 glial nuclei in the SNc of WT:Ifngr1fl/fl or DAT-Cre:Ifngr1fl/fl mice after exposure to IFN-\\u03b3. Scale bar: 10 \\u03bcm. (c) Average nuclear pSTAT1 intensity (arbitrary units, median background subtracted) in each of the indicated cell types, genotypes, and treatment groups, related to (a-b). 10\\u201320 neurons were quantified per region per mouse; n = 2 saline, n = 3 IFN-\\u03b3. **** p < 0.001, Mann-Whitney U test. (d-e) MHC-I and Tap1 RNA FISH in the SNc of WT:Ifngr1fl/fl or DAT-Cre:Ifngr1fl/fl mice after saline or IFN-\\u03b3. White arrowheads indicate TH+ mDA neurons, blue arrows indicate other cells with intact Ifngr1. Scale bars (d): 20 \\u03bcm, (e): 15 \\u03bcm. (f) Average MHC-I or Tap1 RNA intensity (arbitrary units, median background subtracted) in TH+ mDA neurons of the indicated genotype and treatment groups, related to (d-e). 10\\u201320 neurons were quantified per region per mouse. WT; n = 3 Sal, n = 4 IFN-\\u03b3. DAT-Cre:Ifngr1fl/fl (DA-cKO); n = 3 Sal, n = 4 IFN-\\u03b3. **** p < 0.001, ** p < 0.01, Mann-Whitney U test.\", \"sentences\": [], \"annotations\": [], \"relations\": []}], \"annotations\": [], \"relations\": []}]}]\n",
      "No record found for article 3039181.\\Didn't find 23 articles\n",
      "No record found for article 2789353.\\Didn't find 24 articles\n",
      "No record found for article 1626585.\\Didn't find 25 articles\n",
      "No record found for article 10852970.\\Didn't find 26 articles\n",
      "No record found for article 11107502.\\Didn't find 27 articles\n",
      "No record found for article 4758380.\\Didn't find 28 articles\n",
      "No record found for article 8673439.\\Didn't find 29 articles\n",
      "No record found for article 5806540.\\Didn't find 30 articles\n",
      "No record found for article 3932103.\\Didn't find 31 articles\n",
      "No record found for article 6073070.\\Didn't find 32 articles\n",
      "No record found for article 7932415.\\Didn't find 33 articles\n",
      "No record found for article 7744901.\\Didn't find 34 articles\n",
      "No record found for article 10563980.\\Didn't find 35 articles\n",
      "No record found for article 3575938.\\Didn't find 36 articles\n",
      "No record found for article 2877585.\\Didn't find 37 articles\n",
      "No record found for article 3229195.\\Didn't find 38 articles\n",
      "No record found for article 4086382.\\Didn't find 39 articles\n",
      "No record found for article 6329580.\\Didn't find 40 articles\n",
      "No record found for article 9052569.\\Didn't find 41 articles\n",
      "No record found for article 11286219.\\Didn't find 42 articles\n",
      "No record found for article 1647775.\\Didn't find 43 articles\n",
      "No record found for article 11069116.\\Didn't find 44 articles\n",
      "No record found for article 4202965.\\Didn't find 45 articles\n",
      "No record found for article 9924670.\\Didn't find 46 articles\n",
      "No record found for article 6436161.\\Didn't find 47 articles\n",
      "No record found for article 3763765.\\Didn't find 48 articles\n",
      "No record found for article 3629749.\\Didn't find 49 articles\n",
      "No record found for article 6515605.\\Didn't find 50 articles\n",
      "No record found for article 2682533.\\Didn't find 51 articles\n",
      "No record found for article 4068009.\\Didn't find 52 articles\n",
      "No record found for article 6422153.\\Didn't find 53 articles\n",
      "No record found for article 2635052.\\Didn't find 54 articles\n",
      "No record found for article 10019130.\\Didn't find 55 articles\n",
      "No record found for article 10880442.\\Didn't find 56 articles\n",
      "No record found for article 10563922.\\Didn't find 57 articles\n",
      "No record found for article 6550717.\\Didn't find 58 articles\n",
      "No record found for article 3527884.\\Didn't find 59 articles\n",
      "No record found for article 3782085.\\Didn't find 60 articles\n",
      "No record found for article 5942003.\\Didn't find 61 articles\n",
      "No record found for article 5494459.\\Didn't find 62 articles\n",
      "No record found for article 2406850.\\Didn't find 63 articles\n",
      "No record found for article 10852746.\\Didn't find 64 articles\n",
      "No record found for article 8861729.\\Didn't find 65 articles\n",
      "No record found for article 3562078.\\Didn't find 66 articles\n",
      "No record found for article 474736.\\Didn't find 67 articles\n",
      "No record found for article 6491139.\\Didn't find 68 articles\n",
      "No record found for article 9642834.\\Didn't find 69 articles\n",
      "No record found for article 1468137.\\Didn't find 70 articles\n",
      "No record found for article 131040.\\Didn't find 71 articles\n"
     ]
    },
    {
     "ename": "ConnectionError",
     "evalue": "('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mRemoteDisconnected\u001b[0m                        Traceback (most recent call last)",
      "File \u001b[0;32m/usr/lib/python3/dist-packages/urllib3/connectionpool.py:700\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[0;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[1;32m    699\u001b[0m \u001b[38;5;66;03m# Make the request on the httplib connection object.\u001b[39;00m\n\u001b[0;32m--> 700\u001b[0m httplib_response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_make_request\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    701\u001b[0m \u001b[43m    \u001b[49m\u001b[43mconn\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    702\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    703\u001b[0m \u001b[43m    \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    704\u001b[0m \u001b[43m    \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout_obj\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    705\u001b[0m \u001b[43m    \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    706\u001b[0m \u001b[43m    \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    707\u001b[0m \u001b[43m    \u001b[49m\u001b[43mchunked\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mchunked\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    708\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    710\u001b[0m \u001b[38;5;66;03m# If we're going to release the connection in ``finally:``, then\u001b[39;00m\n\u001b[1;32m    711\u001b[0m \u001b[38;5;66;03m# the response doesn't need to know about the connection. Otherwise\u001b[39;00m\n\u001b[1;32m    712\u001b[0m \u001b[38;5;66;03m# it will also try to release it and we'll have a double-release\u001b[39;00m\n\u001b[1;32m    713\u001b[0m \u001b[38;5;66;03m# mess.\u001b[39;00m\n",
      "File \u001b[0;32m/usr/lib/python3/dist-packages/urllib3/connectionpool.py:446\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[0;34m(self, conn, method, url, timeout, chunked, **httplib_request_kw)\u001b[0m\n\u001b[1;32m    442\u001b[0m         \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mBaseException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m    443\u001b[0m             \u001b[38;5;66;03m# Remove the TypeError from the exception chain in\u001b[39;00m\n\u001b[1;32m    444\u001b[0m             \u001b[38;5;66;03m# Python 3 (including for exceptions like SystemExit).\u001b[39;00m\n\u001b[1;32m    445\u001b[0m             \u001b[38;5;66;03m# Otherwise it looks like a bug in the code.\u001b[39;00m\n\u001b[0;32m--> 446\u001b[0m             \u001b[43msix\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mraise_from\u001b[49m\u001b[43m(\u001b[49m\u001b[43me\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43;01mNone\u001b[39;49;00m\u001b[43m)\u001b[49m\n\u001b[1;32m    447\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m (SocketTimeout, BaseSSLError, SocketError) \u001b[38;5;28;01mas\u001b[39;00m e:\n",
      "File \u001b[0;32m<string>:3\u001b[0m, in \u001b[0;36mraise_from\u001b[0;34m(value, from_value)\u001b[0m\n",
      "File \u001b[0;32m/usr/lib/python3/dist-packages/urllib3/connectionpool.py:441\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[0;34m(self, conn, method, url, timeout, chunked, **httplib_request_kw)\u001b[0m\n\u001b[1;32m    440\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 441\u001b[0m     httplib_response \u001b[38;5;241m=\u001b[39m \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mgetresponse\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    442\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mBaseException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m    443\u001b[0m     \u001b[38;5;66;03m# Remove the TypeError from the exception chain in\u001b[39;00m\n\u001b[1;32m    444\u001b[0m     \u001b[38;5;66;03m# Python 3 (including for exceptions like SystemExit).\u001b[39;00m\n\u001b[1;32m    445\u001b[0m     \u001b[38;5;66;03m# Otherwise it looks like a bug in the code.\u001b[39;00m\n",
      "File \u001b[0;32m/usr/lib/python3.10/http/client.py:1375\u001b[0m, in \u001b[0;36mHTTPConnection.getresponse\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m   1374\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m-> 1375\u001b[0m     \u001b[43mresponse\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbegin\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   1376\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m:\n",
      "File \u001b[0;32m/usr/lib/python3.10/http/client.py:318\u001b[0m, in \u001b[0;36mHTTPResponse.begin\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    317\u001b[0m \u001b[38;5;28;01mwhile\u001b[39;00m \u001b[38;5;28;01mTrue\u001b[39;00m:\n\u001b[0;32m--> 318\u001b[0m     version, status, reason \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_read_status\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    319\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m status \u001b[38;5;241m!=\u001b[39m CONTINUE:\n",
      "File \u001b[0;32m/usr/lib/python3.10/http/client.py:287\u001b[0m, in \u001b[0;36mHTTPResponse._read_status\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    284\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m line:\n\u001b[1;32m    285\u001b[0m     \u001b[38;5;66;03m# Presumably, the server closed the connection before\u001b[39;00m\n\u001b[1;32m    286\u001b[0m     \u001b[38;5;66;03m# sending a valid response.\u001b[39;00m\n\u001b[0;32m--> 287\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m RemoteDisconnected(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mRemote end closed connection without\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m    288\u001b[0m                              \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m response\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    289\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n",
      "\u001b[0;31mRemoteDisconnected\u001b[0m: Remote end closed connection without response",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[0;31mProtocolError\u001b[0m                             Traceback (most recent call last)",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/requests/adapters.py:667\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[0;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[1;32m    666\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 667\u001b[0m     resp \u001b[38;5;241m=\u001b[39m \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43murlopen\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    668\u001b[0m \u001b[43m        \u001b[49m\u001b[43mmethod\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    669\u001b[0m \u001b[43m        \u001b[49m\u001b[43murl\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    670\u001b[0m \u001b[43m        \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    671\u001b[0m \u001b[43m        \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    672\u001b[0m \u001b[43m        \u001b[49m\u001b[43mredirect\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[1;32m    673\u001b[0m \u001b[43m        \u001b[49m\u001b[43massert_same_host\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[1;32m    674\u001b[0m \u001b[43m        \u001b[49m\u001b[43mpreload_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[1;32m    675\u001b[0m \u001b[43m        \u001b[49m\u001b[43mdecode_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[1;32m    676\u001b[0m \u001b[43m        \u001b[49m\u001b[43mretries\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmax_retries\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    677\u001b[0m \u001b[43m        \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    678\u001b[0m \u001b[43m        \u001b[49m\u001b[43mchunked\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mchunked\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    679\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    681\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m (ProtocolError, \u001b[38;5;167;01mOSError\u001b[39;00m) \u001b[38;5;28;01mas\u001b[39;00m err:\n",
      "File \u001b[0;32m/usr/lib/python3/dist-packages/urllib3/connectionpool.py:756\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[0;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[1;32m    754\u001b[0m     e \u001b[38;5;241m=\u001b[39m ProtocolError(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mConnection aborted.\u001b[39m\u001b[38;5;124m\"\u001b[39m, e)\n\u001b[0;32m--> 756\u001b[0m retries \u001b[38;5;241m=\u001b[39m \u001b[43mretries\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mincrement\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    757\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43merror\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43me\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_pool\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_stacktrace\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43msys\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mexc_info\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;241;43m2\u001b[39;49m\u001b[43m]\u001b[49m\n\u001b[1;32m    758\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    759\u001b[0m retries\u001b[38;5;241m.\u001b[39msleep()\n",
      "File \u001b[0;32m/usr/lib/python3/dist-packages/urllib3/util/retry.py:532\u001b[0m, in \u001b[0;36mRetry.increment\u001b[0;34m(self, method, url, response, error, _pool, _stacktrace)\u001b[0m\n\u001b[1;32m    531\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m read \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;28;01mFalse\u001b[39;00m \u001b[38;5;129;01mor\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_is_method_retryable(method):\n\u001b[0;32m--> 532\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[43msix\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mreraise\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;28;43mtype\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43merror\u001b[49m\u001b[43m)\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43merror\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_stacktrace\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    533\u001b[0m \u001b[38;5;28;01melif\u001b[39;00m read \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n",
      "File \u001b[0;32m/usr/lib/python3/dist-packages/six.py:718\u001b[0m, in \u001b[0;36mreraise\u001b[0;34m(tp, value, tb)\u001b[0m\n\u001b[1;32m    717\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m value\u001b[38;5;241m.\u001b[39m__traceback__ \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m tb:\n\u001b[0;32m--> 718\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m value\u001b[38;5;241m.\u001b[39mwith_traceback(tb)\n\u001b[1;32m    719\u001b[0m \u001b[38;5;28;01mraise\u001b[39;00m value\n",
      "File \u001b[0;32m/usr/lib/python3/dist-packages/urllib3/connectionpool.py:700\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[0;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[1;32m    699\u001b[0m \u001b[38;5;66;03m# Make the request on the httplib connection object.\u001b[39;00m\n\u001b[0;32m--> 700\u001b[0m httplib_response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_make_request\u001b[49m\u001b[43m(\u001b[49m\n\u001b[1;32m    701\u001b[0m \u001b[43m    \u001b[49m\u001b[43mconn\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    702\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    703\u001b[0m \u001b[43m    \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    704\u001b[0m \u001b[43m    \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout_obj\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    705\u001b[0m \u001b[43m    \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    706\u001b[0m \u001b[43m    \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    707\u001b[0m \u001b[43m    \u001b[49m\u001b[43mchunked\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mchunked\u001b[49m\u001b[43m,\u001b[49m\n\u001b[1;32m    708\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    710\u001b[0m \u001b[38;5;66;03m# If we're going to release the connection in ``finally:``, then\u001b[39;00m\n\u001b[1;32m    711\u001b[0m \u001b[38;5;66;03m# the response doesn't need to know about the connection. Otherwise\u001b[39;00m\n\u001b[1;32m    712\u001b[0m \u001b[38;5;66;03m# it will also try to release it and we'll have a double-release\u001b[39;00m\n\u001b[1;32m    713\u001b[0m \u001b[38;5;66;03m# mess.\u001b[39;00m\n",
      "File \u001b[0;32m/usr/lib/python3/dist-packages/urllib3/connectionpool.py:446\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[0;34m(self, conn, method, url, timeout, chunked, **httplib_request_kw)\u001b[0m\n\u001b[1;32m    442\u001b[0m         \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mBaseException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m    443\u001b[0m             \u001b[38;5;66;03m# Remove the TypeError from the exception chain in\u001b[39;00m\n\u001b[1;32m    444\u001b[0m             \u001b[38;5;66;03m# Python 3 (including for exceptions like SystemExit).\u001b[39;00m\n\u001b[1;32m    445\u001b[0m             \u001b[38;5;66;03m# Otherwise it looks like a bug in the code.\u001b[39;00m\n\u001b[0;32m--> 446\u001b[0m             \u001b[43msix\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mraise_from\u001b[49m\u001b[43m(\u001b[49m\u001b[43me\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43;01mNone\u001b[39;49;00m\u001b[43m)\u001b[49m\n\u001b[1;32m    447\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m (SocketTimeout, BaseSSLError, SocketError) \u001b[38;5;28;01mas\u001b[39;00m e:\n",
      "File \u001b[0;32m<string>:3\u001b[0m, in \u001b[0;36mraise_from\u001b[0;34m(value, from_value)\u001b[0m\n",
      "File \u001b[0;32m/usr/lib/python3/dist-packages/urllib3/connectionpool.py:441\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[0;34m(self, conn, method, url, timeout, chunked, **httplib_request_kw)\u001b[0m\n\u001b[1;32m    440\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m--> 441\u001b[0m     httplib_response \u001b[38;5;241m=\u001b[39m \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mgetresponse\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    442\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mBaseException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m    443\u001b[0m     \u001b[38;5;66;03m# Remove the TypeError from the exception chain in\u001b[39;00m\n\u001b[1;32m    444\u001b[0m     \u001b[38;5;66;03m# Python 3 (including for exceptions like SystemExit).\u001b[39;00m\n\u001b[1;32m    445\u001b[0m     \u001b[38;5;66;03m# Otherwise it looks like a bug in the code.\u001b[39;00m\n",
      "File \u001b[0;32m/usr/lib/python3.10/http/client.py:1375\u001b[0m, in \u001b[0;36mHTTPConnection.getresponse\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m   1374\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m-> 1375\u001b[0m     \u001b[43mresponse\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbegin\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   1376\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m:\n",
      "File \u001b[0;32m/usr/lib/python3.10/http/client.py:318\u001b[0m, in \u001b[0;36mHTTPResponse.begin\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    317\u001b[0m \u001b[38;5;28;01mwhile\u001b[39;00m \u001b[38;5;28;01mTrue\u001b[39;00m:\n\u001b[0;32m--> 318\u001b[0m     version, status, reason \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_read_status\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    319\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m status \u001b[38;5;241m!=\u001b[39m CONTINUE:\n",
      "File \u001b[0;32m/usr/lib/python3.10/http/client.py:287\u001b[0m, in \u001b[0;36mHTTPResponse._read_status\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    284\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m line:\n\u001b[1;32m    285\u001b[0m     \u001b[38;5;66;03m# Presumably, the server closed the connection before\u001b[39;00m\n\u001b[1;32m    286\u001b[0m     \u001b[38;5;66;03m# sending a valid response.\u001b[39;00m\n\u001b[0;32m--> 287\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m RemoteDisconnected(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mRemote end closed connection without\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m    288\u001b[0m                              \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124m response\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[1;32m    289\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n",
      "\u001b[0;31mProtocolError\u001b[0m: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[0;31mConnectionError\u001b[0m                           Traceback (most recent call last)",
      "Cell \u001b[0;32mIn[28], line 7\u001b[0m\n\u001b[1;32m      5\u001b[0m over_context_window \u001b[38;5;241m=\u001b[39m \u001b[38;5;241m0\u001b[39m\n\u001b[1;32m      6\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m \u001b[38;5;28mid\u001b[39m \u001b[38;5;129;01min\u001b[39;00m uid_list:\n\u001b[0;32m----> 7\u001b[0m         response \u001b[38;5;241m=\u001b[39m \u001b[43mrequests\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43mf\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mhttps://www.ncbi.nlm.nih.gov/research/bionlp/RESTful/pmcoa.cgi/BioC_json/PMC\u001b[39;49m\u001b[38;5;132;43;01m{\u001b[39;49;00m\u001b[38;5;28;43mid\u001b[39;49m\u001b[38;5;132;43;01m}\u001b[39;49;00m\u001b[38;5;124;43m/unicode\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m)\u001b[49m\n\u001b[1;32m      8\u001b[0m         \u001b[38;5;66;03m# print(response, id)\u001b[39;00m\n\u001b[1;32m      9\u001b[0m         \u001b[38;5;28;01mif\u001b[39;00m response\u001b[38;5;241m.\u001b[39mtext \u001b[38;5;241m==\u001b[39m \u001b[38;5;124mf\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mNo record can be found for the input: pmc\u001b[39m\u001b[38;5;132;01m{\u001b[39;00m\u001b[38;5;28mid\u001b[39m\u001b[38;5;132;01m}\u001b[39;00m\u001b[38;5;124m'\u001b[39m:\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/requests/api.py:73\u001b[0m, in \u001b[0;36mget\u001b[0;34m(url, params, **kwargs)\u001b[0m\n\u001b[1;32m     62\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mget\u001b[39m(url, params\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs):\n\u001b[1;32m     63\u001b[0m \u001b[38;5;250m    \u001b[39m\u001b[38;5;124mr\u001b[39m\u001b[38;5;124;03m\"\"\"Sends a GET request.\u001b[39;00m\n\u001b[1;32m     64\u001b[0m \n\u001b[1;32m     65\u001b[0m \u001b[38;5;124;03m    :param url: URL for the new :class:`Request` object.\u001b[39;00m\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m     70\u001b[0m \u001b[38;5;124;03m    :rtype: requests.Response\u001b[39;00m\n\u001b[1;32m     71\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[0;32m---> 73\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mrequest\u001b[49m\u001b[43m(\u001b[49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[38;5;124;43mget\u001b[39;49m\u001b[38;5;124;43m\"\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mparams\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mparams\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/requests/api.py:59\u001b[0m, in \u001b[0;36mrequest\u001b[0;34m(method, url, **kwargs)\u001b[0m\n\u001b[1;32m     55\u001b[0m \u001b[38;5;66;03m# By using the 'with' statement we are sure the session is closed, thus we\u001b[39;00m\n\u001b[1;32m     56\u001b[0m \u001b[38;5;66;03m# avoid leaving sockets open which can trigger a ResourceWarning in some\u001b[39;00m\n\u001b[1;32m     57\u001b[0m \u001b[38;5;66;03m# cases, and look like a memory leak in others.\u001b[39;00m\n\u001b[1;32m     58\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m sessions\u001b[38;5;241m.\u001b[39mSession() \u001b[38;5;28;01mas\u001b[39;00m session:\n\u001b[0;32m---> 59\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43msession\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[43m(\u001b[49m\u001b[43mmethod\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43murl\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/requests/sessions.py:589\u001b[0m, in \u001b[0;36mSession.request\u001b[0;34m(self, method, url, params, data, headers, cookies, files, auth, timeout, allow_redirects, proxies, hooks, stream, verify, cert, json)\u001b[0m\n\u001b[1;32m    584\u001b[0m send_kwargs \u001b[38;5;241m=\u001b[39m {\n\u001b[1;32m    585\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mtimeout\u001b[39m\u001b[38;5;124m\"\u001b[39m: timeout,\n\u001b[1;32m    586\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mallow_redirects\u001b[39m\u001b[38;5;124m\"\u001b[39m: allow_redirects,\n\u001b[1;32m    587\u001b[0m }\n\u001b[1;32m    588\u001b[0m send_kwargs\u001b[38;5;241m.\u001b[39mupdate(settings)\n\u001b[0;32m--> 589\u001b[0m resp \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msend\u001b[49m\u001b[43m(\u001b[49m\u001b[43mprep\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43msend_kwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    591\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m resp\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/requests/sessions.py:703\u001b[0m, in \u001b[0;36mSession.send\u001b[0;34m(self, request, **kwargs)\u001b[0m\n\u001b[1;32m    700\u001b[0m start \u001b[38;5;241m=\u001b[39m preferred_clock()\n\u001b[1;32m    702\u001b[0m \u001b[38;5;66;03m# Send the request\u001b[39;00m\n\u001b[0;32m--> 703\u001b[0m r \u001b[38;5;241m=\u001b[39m \u001b[43madapter\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43msend\u001b[49m\u001b[43m(\u001b[49m\u001b[43mrequest\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[38;5;241;43m*\u001b[39;49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m    705\u001b[0m \u001b[38;5;66;03m# Total elapsed time of the request (approximately)\u001b[39;00m\n\u001b[1;32m    706\u001b[0m elapsed \u001b[38;5;241m=\u001b[39m preferred_clock() \u001b[38;5;241m-\u001b[39m start\n",
      "File \u001b[0;32m~/.local/lib/python3.10/site-packages/requests/adapters.py:682\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[0;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[1;32m    667\u001b[0m     resp \u001b[38;5;241m=\u001b[39m conn\u001b[38;5;241m.\u001b[39murlopen(\n\u001b[1;32m    668\u001b[0m         method\u001b[38;5;241m=\u001b[39mrequest\u001b[38;5;241m.\u001b[39mmethod,\n\u001b[1;32m    669\u001b[0m         url\u001b[38;5;241m=\u001b[39murl,\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m    678\u001b[0m         chunked\u001b[38;5;241m=\u001b[39mchunked,\n\u001b[1;32m    679\u001b[0m     )\n\u001b[1;32m    681\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m (ProtocolError, \u001b[38;5;167;01mOSError\u001b[39;00m) \u001b[38;5;28;01mas\u001b[39;00m err:\n\u001b[0;32m--> 682\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m(err, request\u001b[38;5;241m=\u001b[39mrequest)\n\u001b[1;32m    684\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m MaxRetryError \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m    685\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(e\u001b[38;5;241m.\u001b[39mreason, ConnectTimeoutError):\n\u001b[1;32m    686\u001b[0m         \u001b[38;5;66;03m# TODO: Remove this in 3.0.0: see #2811\u001b[39;00m\n",
      "\u001b[0;31mConnectionError\u001b[0m: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
     ]
    }
   ],
   "source": [
    "import json\n",
    "\n",
    "not_found_counter = 0\n",
    "full_texts = []\n",
    "over_context_window = 0\n",
    "for id in uid_list:\n",
    "        response = requests.get(f\"https://www.ncbi.nlm.nih.gov/research/bionlp/RESTful/pmcoa.cgi/BioC_json/PMC{id}/unicode\")\n",
    "        # print(response, id)\n",
    "        if response.text == f'No record can be found for the input: pmc{id}':\n",
    "            not_found_counter += 1\n",
    "            print(f\"No record found for article {id}.\\Didn't find {not_found_counter} articles\")\n",
    "            continue\n",
    "        else:\n",
    "            try:\n",
    "                data = json.loads(response.text[1:-1].strip())\n",
    "                # print(data['documents'][0]['passages'][1:][1]['infons']['section_type'])\n",
    "                total_len = 0\n",
    "                total_len_tokenized = 0\n",
    "                for text in data['documents'][0]['passages'][:]:\n",
    "                    if text != None:\n",
    "                        # print(text['text'])\n",
    "                        total_len += len(text['text']) \n",
    "                        total_len_tokenized += len(tokenizer.encode(text['text']))\n",
    "                        # print(tokenizer.encode(text['text']))\n",
    "                # if data['documents'][0]['passages'][1:][1]['infons']['section_type'] == 'ABSTRACT' or data['documents'][0]['passages'][1:][1]['infons']['section_type'] == 'METHOD':\n",
    "                # print(total_len)\n",
    "\n",
    "                # print(total_len_tokenized)\n",
    "                if total_len_tokenized > 120000:\n",
    "                     over_context_window += 1\n",
    "                     print(total_len_tokenized, ' ', id, ' --- ', over_context_window)\n",
    "                # break\n",
    "            except json.JSONDecodeError as e:\n",
    "                 print(\"Json error for string: \", response.text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "data['documents'][0]['passages'][:]['infons']['section_type']"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
